{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# TO DO\n",
    "1. extract all text untill next example\n",
    "2. clean the collected urls to only include lates version ( removes duplication)\n",
    "https://ppubs.uspto.gov/pubwebapp/static/pages/ppubsbasic.html"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from collection import collect_urls,download_files,unzip_files\n",
    "from utils import extract_examples_start_w_word,find_doc_number,process_siblings,extract_num_dot_examples,extract_examples_w_word,extract_experiments_w_heading,save_as_json,remove_duplicate_docs\n",
    "\n",
    "download_path = \"E:\\\\patents_data_23_24\"\n",
    "unzip_path = \"E:\\\\unzipped_patents_23_24\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data Collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n"
     ]
    }
   ],
   "source": [
    "from lxml import etree\n",
    "import requests\n",
    "\n",
    "# url =\"https://bulkdata.uspto.gov/data/patent/application/redbook/fulltext/2023/\"\n",
    "# rp = requests.get(url, timeout=10)\n",
    "# parser = etree.XMLParser(recover=True)\n",
    "# root = etree.fromstring(rp.text.encode(), parser)\n",
    "# href_values = root.findall(\".//a[@href]\") \n",
    "# urls = [href.get('href') for href in href_values if href.get('href').endswith('.zip')]\n",
    "# for ur in urls[:10]:\n",
    "#     print(url + ur)\n",
    "\n",
    "start_year = int(input(\"Enter the year you want to start from: \"))\n",
    "end_year = int(input(\"Enter the year you want to end at: \"))\n",
    "\n",
    "\n",
    "def fetch_urls_from_pto(start_year,end_year):\n",
    "    urls = {}\n",
    "    for year in range(start_year, end_year + 1):\n",
    "        url = f\"https://bulkdata.uspto.gov/data/patent/application/redbook/fulltext/{year}/\"\n",
    "        rp = requests.get(url, timeout=10)\n",
    "        parser = etree.XMLParser(recover=True)\n",
    "        root = etree.fromstring(rp.text.encode(), parser)\n",
    "        href_values = root.findall(\".//a[@href]\")\n",
    "        urls[year] = [href.get('href') for href in href_values if href.get('href').endswith('.zip')]\n",
    "    return urls\n",
    "\n",
    "urls = fetch_urls_from_pto(start_year,end_year)\n",
    "print(len(urls))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "import re\n",
    "from collections import defaultdict\n",
    "latest_versions = {}\n",
    "latest_files={}\n",
    "\n",
    "# Regex to extract date and optional revision number\n",
    "pattern = re.compile(r\"(ipa\\d{6})(?:_r(\\d+))?\\.zip\")\n",
    "for year in urls.keys():\n",
    "    latest_versions = {}\n",
    "    for file in urls[year]:\n",
    "        match = pattern.match(file)\n",
    "        if match:\n",
    "            base_name, revision = match.groups()\n",
    "            revision = int(revision) if revision else 0  # Default to 0 if no revision\n",
    "            \n",
    "            # Extract the current highest revision number for the base_name\n",
    "            current_revision_match = re.search(r'_r(\\d+)', latest_versions.get(base_name, \"\"))\n",
    "            current_revision = int(current_revision_match.group(1)) if current_revision_match else 0\n",
    "            \n",
    "            # Update if the new file has a higher revision\n",
    "            if base_name not in latest_versions or revision > current_revision:\n",
    "                latest_versions[base_name] = file\n",
    "\n",
    "# Get the final list of unique latest versions\n",
    "    latest_files[year] = sorted(latest_versions.values())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloaded ipa230105.zip ------- 1 / 52\n",
      "Downloaded ipa230112_r1.zip ------- 2 / 52\n",
      "Downloaded ipa230119_r1.zip ------- 3 / 52\n",
      "Downloaded ipa230126_r2.zip ------- 4 / 52\n",
      "Downloaded ipa230202_r2.zip ------- 5 / 52\n",
      "Downloaded ipa230209_r1.zip ------- 6 / 52\n",
      "Downloaded ipa230216_r2.zip ------- 7 / 52\n",
      "Downloaded ipa230223.zip ------- 8 / 52\n",
      "Downloaded ipa230302.zip ------- 9 / 52\n",
      "Downloaded ipa230309.zip ------- 10 / 52\n",
      "Downloaded ipa230316.zip ------- 11 / 52\n",
      "Downloaded ipa230323.zip ------- 12 / 52\n",
      "Downloaded ipa230330.zip ------- 13 / 52\n",
      "Downloaded ipa230406.zip ------- 14 / 52\n",
      "Downloaded ipa230413.zip ------- 15 / 52\n",
      "Downloaded ipa230420.zip ------- 16 / 52\n",
      "Downloaded ipa230427.zip ------- 17 / 52\n",
      "Downloaded ipa230504_r1.zip ------- 18 / 52\n",
      "Downloaded ipa230511_r1.zip ------- 19 / 52\n",
      "Downloaded ipa230518.zip ------- 20 / 52\n",
      "Downloaded ipa230525.zip ------- 21 / 52\n",
      "Downloaded ipa230601.zip ------- 22 / 52\n",
      "Downloaded ipa230608.zip ------- 23 / 52\n",
      "Downloaded ipa230615.zip ------- 24 / 52\n",
      "Downloaded ipa230622.zip ------- 25 / 52\n",
      "Downloaded ipa230629.zip ------- 26 / 52\n",
      "Downloaded ipa230706.zip ------- 27 / 52\n",
      "Downloaded ipa230713.zip ------- 28 / 52\n",
      "Downloaded ipa230720.zip ------- 29 / 52\n",
      "Downloaded ipa230727_r2.zip ------- 30 / 52\n",
      "Downloaded ipa230803_r2.zip ------- 31 / 52\n",
      "Downloaded ipa230810.zip ------- 32 / 52\n",
      "Downloaded ipa230817.zip ------- 33 / 52\n",
      "Downloaded ipa230824.zip ------- 34 / 52\n",
      "Downloaded ipa230831.zip ------- 35 / 52\n",
      "Downloaded ipa230907.zip ------- 36 / 52\n",
      "Downloaded ipa230914.zip ------- 37 / 52\n",
      "Downloaded ipa230921_r1.zip ------- 38 / 52\n",
      "Downloaded ipa230928.zip ------- 39 / 52\n",
      "Downloaded ipa231005_r2.zip ------- 40 / 52\n",
      "Downloaded ipa231012_r2.zip ------- 41 / 52\n",
      "Downloaded ipa231019.zip ------- 42 / 52\n",
      "Downloaded ipa231026.zip ------- 43 / 52\n",
      "Downloaded ipa231102.zip ------- 44 / 52\n",
      "Downloaded ipa231109.zip ------- 45 / 52\n",
      "Downloaded ipa231116.zip ------- 46 / 52\n",
      "Downloaded ipa231123.zip ------- 47 / 52\n",
      "Downloaded ipa231130_r1.zip ------- 48 / 52\n",
      "Downloaded ipa231207_r1.zip ------- 49 / 52\n",
      "Downloaded ipa231214_r1.zip ------- 50 / 52\n",
      "Downloaded ipa231221_r1.zip ------- 51 / 52\n",
      "Downloaded ipa231228_r1.zip ------- 52 / 52\n",
      "Downloaded ipa240104_r1.zip ------- 1 / 52\n",
      "Downloaded ipa240111_r1.zip ------- 2 / 52\n",
      "Downloaded ipa240118_r1.zip ------- 3 / 52\n",
      "Downloaded ipa240125_r1.zip ------- 4 / 52\n",
      "Downloaded ipa240201_r1.zip ------- 5 / 52\n",
      "Downloaded ipa240208_r1.zip ------- 6 / 52\n",
      "Downloaded ipa240215.zip ------- 7 / 52\n",
      "Downloaded ipa240222.zip ------- 8 / 52\n",
      "Downloaded ipa240229.zip ------- 9 / 52\n",
      "Downloaded ipa240307.zip ------- 10 / 52\n",
      "Downloaded ipa240314.zip ------- 11 / 52\n",
      "Downloaded ipa240321.zip ------- 12 / 52\n",
      "Downloaded ipa240328.zip ------- 13 / 52\n",
      "Downloaded ipa240404.zip ------- 14 / 52\n",
      "Downloaded ipa240411.zip ------- 15 / 52\n",
      "Downloaded ipa240418.zip ------- 16 / 52\n",
      "Downloaded ipa240425.zip ------- 17 / 52\n",
      "Downloaded ipa240502.zip ------- 18 / 52\n",
      "Downloaded ipa240509.zip ------- 19 / 52\n",
      "Downloaded ipa240516.zip ------- 20 / 52\n",
      "Downloaded ipa240523.zip ------- 21 / 52\n",
      "Downloaded ipa240530.zip ------- 22 / 52\n",
      "Downloaded ipa240606.zip ------- 23 / 52\n",
      "Downloaded ipa240613.zip ------- 24 / 52\n",
      "Downloaded ipa240620.zip ------- 25 / 52\n",
      "Downloaded ipa240627.zip ------- 26 / 52\n",
      "Downloaded ipa240704.zip ------- 27 / 52\n",
      "Downloaded ipa240711.zip ------- 28 / 52\n",
      "Downloaded ipa240718.zip ------- 29 / 52\n",
      "Downloaded ipa240725.zip ------- 30 / 52\n",
      "Downloaded ipa240801.zip ------- 31 / 52\n",
      "Downloaded ipa240808.zip ------- 32 / 52\n",
      "Downloaded ipa240815.zip ------- 33 / 52\n",
      "Downloaded ipa240822.zip ------- 34 / 52\n",
      "Downloaded ipa240829.zip ------- 35 / 52\n",
      "Downloaded ipa240905.zip ------- 36 / 52\n",
      "Downloaded ipa240912.zip ------- 37 / 52\n",
      "Downloaded ipa240919.zip ------- 38 / 52\n"
     ]
    },
    {
     "ename": "ConnectTimeout",
     "evalue": "HTTPSConnectionPool(host='bulkdata.uspto.gov', port=443): Max retries exceeded with url: /data/patent/application/redbook/fulltext/2024/ipa240926.zip (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002392B8C8CA0>, 'Connection to bulkdata.uspto.gov timed out. (connect timeout=10)'))",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mtimeout\u001b[0m                                   Traceback (most recent call last)",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    173\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 174\u001b[0m     conn \u001b[38;5;241m=\u001b[39m connection\u001b[38;5;241m.\u001b[39mcreate_connection(\n\u001b[0;32m    175\u001b[0m         (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_dns_host, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mport), \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mextra_kw\n\u001b[0;32m    176\u001b[0m     )\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:95\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     94\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m err \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m---> 95\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m err\n\u001b[0;32m     97\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m socket\u001b[38;5;241m.\u001b[39merror(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mgetaddrinfo returns an empty list\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     84\u001b[0m     sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m sock\n",
      "\u001b[1;31mtimeout\u001b[0m: timed out",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mConnectTimeoutError\u001b[0m                       Traceback (most recent call last)",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[1;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[0;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[0;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[0;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:386\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[1;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[0;32m    385\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 386\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_validate_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43mconn\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    387\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    388\u001b[0m     \u001b[38;5;66;03m# Py2 raises this as a BaseSSLError, Py3 raises it as socket timeout.\u001b[39;00m\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:1040\u001b[0m, in \u001b[0;36mHTTPSConnectionPool._validate_conn\u001b[1;34m(self, conn)\u001b[0m\n\u001b[0;32m   1039\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mgetattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msock\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):  \u001b[38;5;66;03m# AppEngine might not have  `.sock`\u001b[39;00m\n\u001b[1;32m-> 1040\u001b[0m     \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m   1042\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m conn\u001b[38;5;241m.\u001b[39mis_verified:\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:358\u001b[0m, in \u001b[0;36mHTTPSConnection.connect\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    356\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mconnect\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m    357\u001b[0m     \u001b[38;5;66;03m# Add certificate verification\u001b[39;00m\n\u001b[1;32m--> 358\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msock \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_new_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    359\u001b[0m     hostname \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:179\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n\u001b[1;32m--> 179\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m ConnectTimeoutError(\n\u001b[0;32m    180\u001b[0m         \u001b[38;5;28mself\u001b[39m,\n\u001b[0;32m    181\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection to \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m timed out. (connect timeout=\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m)\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[0;32m    182\u001b[0m         \u001b[38;5;241m%\u001b[39m (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout),\n\u001b[0;32m    183\u001b[0m     )\n\u001b[0;32m    185\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketError \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "\u001b[1;31mConnectTimeoutError\u001b[0m: (<urllib3.connection.HTTPSConnection object at 0x000002392B8C8CA0>, 'Connection to bulkdata.uspto.gov timed out. (connect timeout=10)')",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mMaxRetryError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\requests\\adapters.py:440\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    439\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[1;32m--> 440\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    441\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    442\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    443\u001b[0m \u001b[43m        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    444\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    445\u001b[0m \u001b[43m        \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    446\u001b[0m \u001b[43m        \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    447\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    448\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    449\u001b[0m \u001b[43m        \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    450\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\n\u001b[0;32m    451\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    453\u001b[0m \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[0;32m    454\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:785\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    783\u001b[0m     e \u001b[38;5;241m=\u001b[39m ProtocolError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection aborted.\u001b[39m\u001b[38;5;124m\"\u001b[39m, e)\n\u001b[1;32m--> 785\u001b[0m retries \u001b[38;5;241m=\u001b[39m \u001b[43mretries\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mincrement\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    786\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msys\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mexc_info\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\n\u001b[0;32m    787\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    788\u001b[0m retries\u001b[38;5;241m.\u001b[39msleep()\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\urllib3\\util\\retry.py:592\u001b[0m, in \u001b[0;36mRetry.increment\u001b[1;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[0;32m    591\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m new_retry\u001b[38;5;241m.\u001b[39mis_exhausted():\n\u001b[1;32m--> 592\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m MaxRetryError(_pool, url, error \u001b[38;5;129;01mor\u001b[39;00m ResponseError(cause))\n\u001b[0;32m    594\u001b[0m log\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mIncremented Retry for (url=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m): \u001b[39m\u001b[38;5;132;01m%r\u001b[39;00m\u001b[38;5;124m\"\u001b[39m, url, new_retry)\n",
      "\u001b[1;31mMaxRetryError\u001b[0m: HTTPSConnectionPool(host='bulkdata.uspto.gov', port=443): Max retries exceeded with url: /data/patent/application/redbook/fulltext/2024/ipa240926.zip (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002392B8C8CA0>, 'Connection to bulkdata.uspto.gov timed out. (connect timeout=10)'))",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mConnectTimeout\u001b[0m                            Traceback (most recent call last)",
      "Input \u001b[1;32mIn [29]\u001b[0m, in \u001b[0;36m<cell line: 17>\u001b[1;34m()\u001b[0m\n\u001b[0;32m     17\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m year \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mrange\u001b[39m(start_year, end_year \u001b[38;5;241m+\u001b[39m \u001b[38;5;241m1\u001b[39m):\n\u001b[0;32m     18\u001b[0m         url \u001b[38;5;241m=\u001b[39m \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhttps://bulkdata.uspto.gov/data/patent/application/redbook/fulltext/\u001b[39m\u001b[38;5;132;01m{\u001b[39;00myear\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m/\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m---> 19\u001b[0m         files\u001b[38;5;241m.\u001b[39mappend(\u001b[43mdownload_files\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mdownload_path\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mlatest_files\u001b[49m\u001b[43m[\u001b[49m\u001b[43myear\u001b[49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m)\n",
      "Input \u001b[1;32mIn [29]\u001b[0m, in \u001b[0;36mdownload_files\u001b[1;34m(main_url, download_path, files)\u001b[0m\n\u001b[0;32m      6\u001b[0m url \u001b[38;5;241m=\u001b[39m main_url \u001b[38;5;241m+\u001b[39m file_name\n\u001b[0;32m      9\u001b[0m zip_file_path \u001b[38;5;241m=\u001b[39m os\u001b[38;5;241m.\u001b[39mpath\u001b[38;5;241m.\u001b[39mjoin(download_path, file_name) \u001b[38;5;66;03m#.split(\".\")[-1]\u001b[39;00m\n\u001b[1;32m---> 10\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mstream\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mTrue\u001b[39;49;00m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;241;43m10\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[0;32m     11\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m \u001b[38;5;28mopen\u001b[39m(zip_file_path, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mwb\u001b[39m\u001b[38;5;124m\"\u001b[39m) \u001b[38;5;28;01mas\u001b[39;00m f:\n\u001b[0;32m     12\u001b[0m     \u001b[38;5;28;01mfor\u001b[39;00m chunk \u001b[38;5;129;01min\u001b[39;00m response\u001b[38;5;241m.\u001b[39miter_content(chunk_size\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m8192\u001b[39m):\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\requests\\api.py:75\u001b[0m, in \u001b[0;36mget\u001b[1;34m(url, params, **kwargs)\u001b[0m\n\u001b[0;32m     64\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[0;32m     65\u001b[0m     \u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[0;32m     66\u001b[0m \n\u001b[0;32m     67\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     72\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[0;32m     73\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[1;32m---> 75\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m'\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\requests\\api.py:61\u001b[0m, in \u001b[0;36mrequest\u001b[1;34m(method, url, **kwargs)\u001b[0m\n\u001b[0;32m     57\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[0;32m     58\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[0;32m     59\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[0;32m     60\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[1;32m---> 61\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\requests\\sessions.py:529\u001b[0m, in \u001b[0;36mSession.request\u001b[1;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[0;32m    524\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[0;32m    525\u001b[0m     \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m'\u001b[39m: timeout,\n\u001b[0;32m    526\u001b[0m     \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m'\u001b[39m: allow_redirects,\n\u001b[0;32m    527\u001b[0m }\n\u001b[0;32m    528\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[1;32m--> 529\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[0;32m    531\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\requests\\sessions.py:645\u001b[0m, in \u001b[0;36mSession.send\u001b[1;34m(self, request, **kwargs)\u001b[0m\n\u001b[0;32m    642\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[0;32m    644\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[1;32m--> 645\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    647\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[0;32m    648\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\site-packages\\requests\\adapters.py:507\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    504\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, ConnectTimeoutError):\n\u001b[0;32m    505\u001b[0m     \u001b[38;5;66;03m# TODO: Remove this in 3.0.0: see #2811\u001b[39;00m\n\u001b[0;32m    506\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, NewConnectionError):\n\u001b[1;32m--> 507\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m ConnectTimeout(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[0;32m    509\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, ResponseError):\n\u001b[0;32m    510\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m RetryError(e, request\u001b[38;5;241m=\u001b[39mrequest)\n",
      "\u001b[1;31mConnectTimeout\u001b[0m: HTTPSConnectionPool(host='bulkdata.uspto.gov', port=443): Max retries exceeded with url: /data/patent/application/redbook/fulltext/2024/ipa240926.zip (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002392B8C8CA0>, 'Connection to bulkdata.uspto.gov timed out. (connect timeout=10)'))"
     ]
    }
   ],
   "source": [
    "import os\n",
    "def download_files(main_url, download_path,files):\n",
    "    if not os.path.exists(download_path):\n",
    "        os.makedirs(download_path)\n",
    "    for index,file_name in enumerate(files):\n",
    "        url = main_url + file_name\n",
    "\n",
    "\n",
    "        zip_file_path = os.path.join(download_path, file_name) #.split(\".\")[-1]\n",
    "        response = requests.get(url, stream=True, timeout=10)\n",
    "        with open(zip_file_path, \"wb\") as f:\n",
    "            for chunk in response.iter_content(chunk_size=8192):\n",
    "                f.write(chunk)\n",
    "        print(f\"Downloaded {file_name} ------- {index + 1} / {len(files)}\")\n",
    "download_path = \"E:\\\\patents_data_23_24\"\n",
    "files = []\n",
    "for year in range(start_year, end_year + 1):\n",
    "        url = f\"https://bulkdata.uspto.gov/data/patent/application/redbook/fulltext/{year}/\"\n",
    "        files.append(download_files(url, download_path, latest_files[year]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Download path set to: patents_data\n",
      "https://bulkdata.uspto.gov/data/patent/application/redbook/fulltext/2023/\n"
     ]
    },
    {
     "ename": "JSONDecodeError",
     "evalue": "Expecting value: line 3 column 1 (char 6)",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mJSONDecodeError\u001b[0m                           Traceback (most recent call last)",
      "File \u001b[1;32m~\\AppData\\Roaming\\Python\\Python312\\site-packages\\requests\\models.py:974\u001b[0m, in \u001b[0;36mResponse.json\u001b[1;34m(self, **kwargs)\u001b[0m\n\u001b[0;32m    973\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 974\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mcomplexjson\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mloads\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mtext\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    975\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m JSONDecodeError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    976\u001b[0m     \u001b[38;5;66;03m# Catch JSON-related errors and raise as requests.JSONDecodeError\u001b[39;00m\n\u001b[0;32m    977\u001b[0m     \u001b[38;5;66;03m# This aliases json.JSONDecodeError and simplejson.JSONDecodeError\u001b[39;00m\n",
      "File \u001b[1;32mc:\\Users\\amoha\\anaconda3\\envs\\cuda\\Lib\\json\\__init__.py:346\u001b[0m, in \u001b[0;36mloads\u001b[1;34m(s, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)\u001b[0m\n\u001b[0;32m    343\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m (\u001b[38;5;28mcls\u001b[39m \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m object_hook \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m\n\u001b[0;32m    344\u001b[0m         parse_int \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m parse_float \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m\n\u001b[0;32m    345\u001b[0m         parse_constant \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m object_pairs_hook \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m kw):\n\u001b[1;32m--> 346\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_default_decoder\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mdecode\u001b[49m\u001b[43m(\u001b[49m\u001b[43ms\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    347\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mcls\u001b[39m \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[1;32mc:\\Users\\amoha\\anaconda3\\envs\\cuda\\Lib\\json\\decoder.py:337\u001b[0m, in \u001b[0;36mJSONDecoder.decode\u001b[1;34m(self, s, _w)\u001b[0m\n\u001b[0;32m    333\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\"Return the Python representation of ``s`` (a ``str`` instance\u001b[39;00m\n\u001b[0;32m    334\u001b[0m \u001b[38;5;124;03mcontaining a JSON document).\u001b[39;00m\n\u001b[0;32m    335\u001b[0m \n\u001b[0;32m    336\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m--> 337\u001b[0m obj, end \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mraw_decode\u001b[49m\u001b[43m(\u001b[49m\u001b[43ms\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43midx\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_w\u001b[49m\u001b[43m(\u001b[49m\u001b[43ms\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m0\u001b[39;49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mend\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    338\u001b[0m end \u001b[38;5;241m=\u001b[39m _w(s, end)\u001b[38;5;241m.\u001b[39mend()\n",
      "File \u001b[1;32mc:\\Users\\amoha\\anaconda3\\envs\\cuda\\Lib\\json\\decoder.py:355\u001b[0m, in \u001b[0;36mJSONDecoder.raw_decode\u001b[1;34m(self, s, idx)\u001b[0m\n\u001b[0;32m    354\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mStopIteration\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m err:\n\u001b[1;32m--> 355\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m JSONDecodeError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mExpecting value\u001b[39m\u001b[38;5;124m\"\u001b[39m, s, err\u001b[38;5;241m.\u001b[39mvalue) \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n\u001b[0;32m    356\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m obj, end\n",
      "\u001b[1;31mJSONDecodeError\u001b[0m: Expecting value: line 3 column 1 (char 6)",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mJSONDecodeError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[10], line 34\u001b[0m\n\u001b[0;32m     31\u001b[0m     os\u001b[38;5;241m.\u001b[39mmakedirs(download_path, exist_ok\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n\u001b[0;32m     32\u001b[0m     \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFallback: Download path set to: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mdownload_path\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m---> 34\u001b[0m files \u001b[38;5;241m=\u001b[39m \u001b[43mcollect_urls\u001b[49m\u001b[43m(\u001b[49m\u001b[43myear\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     35\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m files:\n\u001b[0;32m     36\u001b[0m     total_size \u001b[38;5;241m=\u001b[39m \u001b[38;5;28msum\u001b[39m(file[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mfileSize\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m/\u001b[39m \u001b[38;5;241m1024\u001b[39m \u001b[38;5;241m/\u001b[39m \u001b[38;5;241m1024\u001b[39m \u001b[38;5;28;01mfor\u001b[39;00m file \u001b[38;5;129;01min\u001b[39;00m files)\n",
      "Cell \u001b[1;32mIn[10], line 16\u001b[0m, in \u001b[0;36mcollect_urls\u001b[1;34m(year)\u001b[0m\n\u001b[0;32m     14\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m []\n\u001b[0;32m     15\u001b[0m \u001b[38;5;66;03m#print(f\"Urls collected for the period from {start_date} to {end_date} successfully\")\u001b[39;00m\n\u001b[1;32m---> 16\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mjson\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mproductFiles\u001b[39m\u001b[38;5;124m\"\u001b[39m]\n",
      "File \u001b[1;32m~\\AppData\\Roaming\\Python\\Python312\\site-packages\\requests\\models.py:978\u001b[0m, in \u001b[0;36mResponse.json\u001b[1;34m(self, **kwargs)\u001b[0m\n\u001b[0;32m    974\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m complexjson\u001b[38;5;241m.\u001b[39mloads(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtext, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    975\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m JSONDecodeError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    976\u001b[0m     \u001b[38;5;66;03m# Catch JSON-related errors and raise as requests.JSONDecodeError\u001b[39;00m\n\u001b[0;32m    977\u001b[0m     \u001b[38;5;66;03m# This aliases json.JSONDecodeError and simplejson.JSONDecodeError\u001b[39;00m\n\u001b[1;32m--> 978\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m RequestsJSONDecodeError(e\u001b[38;5;241m.\u001b[39mmsg, e\u001b[38;5;241m.\u001b[39mdoc, e\u001b[38;5;241m.\u001b[39mpos)\n",
      "\u001b[1;31mJSONDecodeError\u001b[0m: Expecting value: line 3 column 1 (char 6)"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "import os\n",
    "\n",
    "def collect_urls(year):\n",
    "    url = f\"https://bulkdata.uspto.gov/data/patent/application/redbook/fulltext/{year}/\"\n",
    "    # params = {\n",
    "    #     \"data\": f'{{\"name\":\"APPXML\",\"fromDate\":\"{start_date}\",\"toDate\":\"{end_date}\"}}'\n",
    "    # }\n",
    "    print(url)\n",
    "    response = requests.get(url, timeout=10)\n",
    "    if response.status_code != 200:\n",
    "        print(f\"Error: {response.status_code}\")\n",
    "        return []\n",
    "    #print(f\"Urls collected for the period from {start_date} to {end_date} successfully\")\n",
    "    return response.json()[\"productFiles\"]\n",
    "\n",
    "# start_date = input(\"Enter start date YYYY-MM: \")\n",
    "# end_date = input(\"Enter end date YYYY-MM: \")\n",
    "year = input(\"Enter year: \")\n",
    "download_path = \"patents_data\"\n",
    "unzip_path = \"unzipped_patents_data\"\n",
    "\n",
    "try:\n",
    "    os.makedirs(download_path, exist_ok=True)\n",
    "    print(f\"Download path set to: {download_path}\")\n",
    "except PermissionError:\n",
    "    print(\"Error: No permission to create directory on D drive\")\n",
    "    # Fallback to current directory\n",
    "    download_path = \"patents_data\"\n",
    "    os.makedirs(download_path, exist_ok=True)\n",
    "    print(f\"Fallback: Download path set to: {download_path}\")\n",
    "\n",
    "files = collect_urls(year)\n",
    "if files:\n",
    "    total_size = sum(file[\"fileSize\"] / 1024 / 1024 for file in files)\n",
    "    print(\n",
    "        f\"\\nTotal file size: {round(total_size / 1024, 2)} GB for  {year}\\n\"\n",
    "    )\n",
    "    download_files(files, download_path)\n",
    "    unzip_files(download_path, unzip_path)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   0%|          | 0/90 [00:00<?, ?it/s]"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   1%|          | 1/90 [00:04<06:05,  4.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230105.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   2%|▏         | 2/90 [00:06<04:19,  2.95s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230112_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   3%|▎         | 3/90 [00:10<05:19,  3.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230119_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   4%|▍         | 4/90 [00:15<05:41,  3.97s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230126_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   6%|▌         | 5/90 [00:18<05:29,  3.88s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230202_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   7%|▋         | 6/90 [00:23<05:44,  4.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230209_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   8%|▊         | 7/90 [00:27<05:38,  4.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230216_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:   9%|▉         | 8/90 [00:31<05:26,  3.98s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230223.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  10%|█         | 9/90 [00:36<05:44,  4.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230302.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  11%|█         | 10/90 [00:40<05:32,  4.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230309.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  12%|█▏        | 11/90 [00:44<05:35,  4.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230316.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  13%|█▎        | 12/90 [00:49<05:38,  4.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230323.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  14%|█▍        | 13/90 [00:53<05:46,  4.49s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230330.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  16%|█▌        | 14/90 [00:57<05:21,  4.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230406.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  17%|█▋        | 15/90 [01:01<05:18,  4.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230413.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  18%|█▊        | 16/90 [01:06<05:19,  4.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230420.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  19%|█▉        | 17/90 [01:10<05:07,  4.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230427.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  20%|██        | 18/90 [01:14<05:07,  4.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230504_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  21%|██        | 19/90 [01:18<05:01,  4.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230511_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  22%|██▏       | 20/90 [01:23<05:03,  4.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230518.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  23%|██▎       | 21/90 [01:27<04:58,  4.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230525.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  24%|██▍       | 22/90 [01:31<04:36,  4.06s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230601.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  26%|██▌       | 23/90 [01:35<04:37,  4.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230608.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  27%|██▋       | 24/90 [01:40<04:44,  4.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230615.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  28%|██▊       | 25/90 [01:45<05:03,  4.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230622.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  29%|██▉       | 26/90 [01:50<05:07,  4.81s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230629.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  30%|███       | 27/90 [01:54<04:49,  4.59s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230706.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  31%|███       | 28/90 [01:58<04:29,  4.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230713.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  32%|███▏      | 29/90 [02:02<04:13,  4.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230720.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  33%|███▎      | 30/90 [02:06<04:04,  4.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230727_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  34%|███▍      | 31/90 [02:10<04:02,  4.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230803_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  36%|███▌      | 32/90 [02:14<03:52,  4.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230810.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  37%|███▋      | 33/90 [02:18<03:54,  4.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230817.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  38%|███▊      | 34/90 [02:22<03:47,  4.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230824.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  39%|███▉      | 35/90 [02:26<03:47,  4.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230831.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  40%|████      | 36/90 [02:31<03:55,  4.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230907.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  41%|████      | 37/90 [02:36<03:56,  4.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230914.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  42%|████▏     | 38/90 [02:41<04:07,  4.76s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230921_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  43%|████▎     | 39/90 [02:46<04:03,  4.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa230928.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  44%|████▍     | 40/90 [02:52<04:20,  5.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231005_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  46%|████▌     | 41/90 [03:01<05:10,  6.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231012_r2.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  47%|████▋     | 42/90 [03:09<05:22,  6.73s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231019.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  48%|████▊     | 43/90 [03:17<05:32,  7.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231026.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  49%|████▉     | 44/90 [03:26<05:55,  7.73s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231102.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  50%|█████     | 45/90 [03:36<06:09,  8.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231109.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  51%|█████     | 46/90 [03:45<06:11,  8.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231116.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  52%|█████▏    | 47/90 [03:51<05:37,  7.85s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231123.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  53%|█████▎    | 48/90 [04:00<05:46,  8.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231130_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  54%|█████▍    | 49/90 [04:07<05:22,  7.86s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231207_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  56%|█████▌    | 50/90 [04:14<05:01,  7.53s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231214_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  57%|█████▋    | 51/90 [04:23<05:08,  7.91s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231221_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  58%|█████▊    | 52/90 [04:32<05:13,  8.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa231228_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  59%|█████▉    | 53/90 [04:42<05:33,  9.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240104_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  60%|██████    | 54/90 [04:50<05:05,  8.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240111_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  61%|██████    | 55/90 [04:58<04:52,  8.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240118_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  62%|██████▏   | 56/90 [05:08<04:58,  8.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240125_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  63%|██████▎   | 57/90 [05:17<04:51,  8.85s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240201_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  64%|██████▍   | 58/90 [05:28<05:02,  9.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240208_r1.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  66%|██████▌   | 59/90 [05:35<04:34,  8.87s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240215.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  67%|██████▋   | 60/90 [05:44<04:25,  8.84s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240222.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  68%|██████▊   | 61/90 [05:53<04:19,  8.96s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240229.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  69%|██████▉   | 62/90 [06:00<03:50,  8.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240307.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  70%|███████   | 63/90 [06:11<04:08,  9.22s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240314.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  71%|███████   | 64/90 [06:21<04:07,  9.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240321.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  72%|███████▏  | 65/90 [06:32<04:09,  9.99s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240328.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  73%|███████▎  | 66/90 [06:42<03:57,  9.89s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240404.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  74%|███████▍  | 67/90 [06:52<03:48,  9.92s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240411.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  76%|███████▌  | 68/90 [07:02<03:41, 10.05s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240418.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  77%|███████▋  | 69/90 [07:13<03:34, 10.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240425.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  78%|███████▊  | 70/90 [07:24<03:30, 10.51s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240502.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  79%|███████▉  | 71/90 [07:34<03:16, 10.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240509.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  80%|████████  | 72/90 [07:45<03:11, 10.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240516.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  81%|████████  | 73/90 [07:53<02:42,  9.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240523.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  82%|████████▏ | 74/90 [08:03<02:37,  9.83s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240530.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  83%|████████▎ | 75/90 [08:14<02:33, 10.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240606.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  84%|████████▍ | 76/90 [08:24<02:22, 10.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240613.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  86%|████████▌ | 77/90 [08:36<02:17, 10.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240620.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  87%|████████▋ | 78/90 [08:47<02:11, 10.92s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240627.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  88%|████████▊ | 79/90 [08:58<01:58, 10.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240704.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  89%|████████▉ | 80/90 [09:14<02:03, 12.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240711.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  90%|█████████ | 81/90 [09:23<01:42, 11.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240718.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  91%|█████████ | 82/90 [09:31<01:22, 10.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240725.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  92%|█████████▏| 83/90 [09:44<01:17, 11.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240801.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  93%|█████████▎| 84/90 [09:54<01:05, 10.95s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240808.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  94%|█████████▍| 85/90 [10:05<00:54, 10.87s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240815.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  96%|█████████▌| 86/90 [10:15<00:42, 10.71s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240822.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  97%|█████████▋| 87/90 [10:27<00:32, 10.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240829.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  98%|█████████▊| 88/90 [10:34<00:19,  9.85s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240905.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files:  99%|█████████▉| 89/90 [10:43<00:09,  9.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240912.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Unzipping files: 100%|██████████| 90/90 [10:55<00:00,  7.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unzipped ipa240919.zip to E:\\unzipped_patents_23_24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "download_path = \"E:\\\\patents_data_23_24\"\n",
    "unzip_path = \"E:\\\\unzipped_patents_23_24\"\n",
    "unzip_files(download_path, unzip_path)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Processing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "def process_xml_files(directory_path):\n",
    "    \"\"\"\n",
    "    Process all XML files in the given directory and split them into parts.\n",
    "    \n",
    "    Args:\n",
    "        directory_path (str): Path to directory containing XML files\n",
    "        \n",
    "    Returns:\n",
    "        list: Combined list of XML parts from all files\n",
    "    \"\"\"\n",
    "    all_xml_parts = []\n",
    "    \n",
    "    # Get all XML files in directory\n",
    "    xml_files = [f for f in os.listdir(directory_path) if f.endswith('.xml')]\n",
    "    \n",
    "    # Process each file\n",
    "    for xml_file in xml_files:\n",
    "        file_path = os.path.join(directory_path, xml_file)\n",
    "        try:\n",
    "            with open(file_path, \"r\", encoding=\"utf-8\") as file:\n",
    "                content = file.read()\n",
    "                parts = content.split('<?xml version=\"1.0\" encoding=\"UTF-8\"?>')\n",
    "                # Remove empty parts and extend master list\n",
    "                parts = [p for p in parts if p.strip()]\n",
    "                all_xml_parts.extend(parts)\n",
    "        except Exception as e:\n",
    "            print(f\"Error processing {xml_file}: {str(e)}\")\n",
    "    \n",
    "    return all_xml_parts\n",
    "xml_parts = process_xml_files(unzip_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "30085"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(xml_parts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "ename": "MemoryError",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mMemoryError\u001b[0m                               Traceback (most recent call last)",
      "Input \u001b[1;32mIn [2]\u001b[0m, in \u001b[0;36m<cell line: 14>\u001b[1;34m()\u001b[0m\n\u001b[0;32m     12\u001b[0m         content \u001b[38;5;241m=\u001b[39m file\u001b[38;5;241m.\u001b[39mread()\n\u001b[0;32m     13\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m content\n\u001b[1;32m---> 14\u001b[0m xml_parts \u001b[38;5;241m=\u001b[39m \u001b[43mread_xml_file\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mE:\u001b[39;49m\u001b[38;5;130;43;01m\\\\\u001b[39;49;00m\u001b[38;5;124;43m2023-092024.xml\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n",
      "Input \u001b[1;32mIn [2]\u001b[0m, in \u001b[0;36mread_xml_file\u001b[1;34m(file_path)\u001b[0m\n\u001b[0;32m      2\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[0;32m      3\u001b[0m \u001b[38;5;124;03mRead the content of an XML file.\u001b[39;00m\n\u001b[0;32m      4\u001b[0m \u001b[38;5;124;03m\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m      9\u001b[0m \u001b[38;5;124;03m    str: Content of the XML file\u001b[39;00m\n\u001b[0;32m     10\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[0;32m     11\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m \u001b[38;5;28mopen\u001b[39m(file_path, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mr\u001b[39m\u001b[38;5;124m\"\u001b[39m, encoding\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mutf-8\u001b[39m\u001b[38;5;124m\"\u001b[39m) \u001b[38;5;28;01mas\u001b[39;00m file:\n\u001b[1;32m---> 12\u001b[0m     content \u001b[38;5;241m=\u001b[39m \u001b[43mfile\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mread\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     13\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m content\n",
      "File \u001b[1;32mc:\\ProgramData\\anaconda3\\lib\\codecs.py:322\u001b[0m, in \u001b[0;36mBufferedIncrementalDecoder.decode\u001b[1;34m(self, input, final)\u001b[0m\n\u001b[0;32m    319\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mdecode\u001b[39m(\u001b[38;5;28mself\u001b[39m, \u001b[38;5;28minput\u001b[39m, final\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mFalse\u001b[39;00m):\n\u001b[0;32m    320\u001b[0m     \u001b[38;5;66;03m# decode input (taking the buffer into account)\u001b[39;00m\n\u001b[0;32m    321\u001b[0m     data \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mbuffer \u001b[38;5;241m+\u001b[39m \u001b[38;5;28minput\u001b[39m\n\u001b[1;32m--> 322\u001b[0m     (result, consumed) \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_buffer_decode\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43merrors\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mfinal\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    323\u001b[0m     \u001b[38;5;66;03m# keep undecoded input until the next call\u001b[39;00m\n\u001b[0;32m    324\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mbuffer \u001b[38;5;241m=\u001b[39m data[consumed:]\n",
      "\u001b[1;31mMemoryError\u001b[0m: "
     ]
    }
   ],
   "source": [
    "def read_xml_file(file_path):\n",
    "    \"\"\"\n",
    "    Read the content of an XML file.\n",
    "    \n",
    "    Args:\n",
    "        file_path (str): Path to the XML file\n",
    "    \n",
    "    Returns:\n",
    "        str: Content of the XML file\n",
    "    \"\"\"\n",
    "    with open(file_path, \"r\", encoding=\"utf-8\") as file:\n",
    "        content = file.read()\n",
    "    return content\n",
    "xml_parts = read_xml_file(\"E:\\\\2023-092024.xml\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# with open(\"2023-092024.xml\", \"w\", encoding=\"utf-8\") as file:\n",
    "#         for part in xml_parts:\n",
    "#             file.write(part)\n",
    "#             file.write(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# with open(\"unzipped_patents_data\\\\ipa230105.xml\", \"r\", encoding=\"utf-8\") as file:\n",
    "#     content = file.read()\n",
    "# xml_parts = content.split('<?xml version=\"1.0\" encoding=\"UTF-8\"?>')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of duplicates 680 out of 30085\n"
     ]
    }
   ],
   "source": [
    "xml_no_dup = remove_duplicate_docs(xml_parts)\n",
    "print(f\"Number of duplicates {len(xml_parts) - len(xml_no_dup)} out of {len(xml_parts)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# from bs4 import BeautifulSoup\n",
    "\n",
    "# def extract_experiments_w_heading(text):\n",
    "#     \"\"\"Extracts the 'Examples' section and its experiments from a patent text.\"\"\"\n",
    "\n",
    "#     # Use BeautifulSoup to parse the structure (for HTML-like patents)\n",
    "#     soup = BeautifulSoup(text, \"html.parser\")\n",
    "\n",
    "#     # Find the \"EXAMPLES\" section heading\n",
    "#     examples_heading = soup.find(\n",
    "#         lambda tag: tag.name == \"heading\"\n",
    "#         and (\n",
    "#             \"EXAMPLES\" in tag.text.upper()\n",
    "#             or \"EXAMPLE\" == tag.text.upper()\n",
    "#             or \"EXPERIMENT\" == tag.text.upper()\n",
    "#             or \"EXPERIMENTS\" in tag.text.upper()\n",
    "#         )\n",
    "#     )\n",
    "#     if not examples_heading:\n",
    "#         # print(\"No 'Examples' section found.\")\n",
    "#         return None\n",
    "\n",
    "\n",
    "#     return examples_heading "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'28168'"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "find_doc_number(xml_parts[28760])[0][-5:] #28168"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'number': 'Example 1',\n",
       "  'title': '',\n",
       "  'content': ['This example describes one of the many potential sets of criteria for the selection of sites of incorporation of non-naturally encoded amino acids into relaxin.',\n",
       "   'FIGS. 1-4 show the structure and the sequence of relaxin and the table below includes sequences with the A chain, B chain, relaxin and prorelaxin. Relaxin polypeptides were generated by substituting a naturally encoded amino acid with a non-naturally encoded amino acid. Each polypeptide had one of the amino acids substituted with para-acetylphenylalanine (pAcF or pAF). The polypeptides generated lacked the leader sequence and were A/B chain relaxin polypeptides (SEQ ID NO. 1-3). Each of the polypeptides generated had a non-naturally encoded amino acid substitution at one of the following positions 1, 5, 18, 13, 2 of SEQ ID NO: 4 or in those positions of the A chain of any of the known relaxin sequences or 5, 7, 18, 28 of SEQ ID NO: 5 or 6 in those same positions of the B chain of any of the known relaxin sequences. FIG. 2 shows the structure of human relaxin that was labeled using the PyMOL software (DeLano Scientific; Palo Alto, Calif.) and some amino acids corresponding to those substituted with para-acetylphenylalanine in relaxin polypeptides of the invention.',\n",
       "   'Another set of criteria for the selection of preferred sites of incorporation of non-naturally encoded amino acids includes using and comparing crystal structures from the Protein Data Bank, or other data banks, are used to model the structure of relaxin and residues are identified that 1) would not interfere with binding to their receptor, and 2) would not be present in the interior of the protein. In some embodiments, one or more non-naturally encoded encoded amino acids are incorporated at, but not limited to, one or more of the following positions of relaxin: 1, 5, 18, 13, 2 of SEQ ID NO: 4 or in those positions of the A chain of any of the known relaxin sequences or 5, 7, 18, 28 of SEQ ID NO: 5 or 6 in those same positions of the B chain of any of the known relaxin sequences.',\n",
       "   'The following criteria were used to evaluate each position of relaxin and relaxin analogs for the introduction of a non-naturally encoded amino acid: the residue (a) should not interfere with binding of the receptor based on structural analysis, b) should not be affected by alanine or homolog scanning mutagenesis (c) should be surface exposed and exhibit minimal van der Waals or hydrogen bonding interactions with surrounding residues, (d) should be either deleted or variable in relaxin variants, (e) would result in conservative changes upon substitution with a non-naturally encoded amino acid and (f) could be found in either highly flexible regions or structurally rigid regions. In addition, further calculations can be performed on the relaxin molecule, utilizing the Cx program (Pintar et al. (2002) Bioinformatics, 18, pp 980) to evaluate the extent of protrusion for each protein atom.',\n",
       "   'In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in the A chain of relaxin: before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25 (i.e., at the carboxyl terminus of the protein) (SEQ ID NO: 4 or the corresponding amino acids in SEQ ID NOs: 1-3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in the B chain of relaxin: before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30 (i.e., at the carboxyl terminus of the protein) (SEQ ID NO: 5 or 6 or the corresponding amino acids in SEQ ID NOs: 1-3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in relaxin: 1, 5, 31, 2, 13, 29, 18, 52 (SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in relaxin: 5, 31, 2, 13, 29, 18, 52 (SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in relaxin: 1, 5, 31, 2, 13, 29 (SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in relaxin: 5, 31, 2, 13, 29 (SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in relaxin: 1, 5, 31, 2, 13 (SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in relaxin: 5, 31, 2, 13 (SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3).']},\n",
       " {'number': 'Example 2',\n",
       "  'title': '',\n",
       "  'content': ['This example details cloning and expression of a relaxin polypeptide including a non-naturally encoded amino acid in E. coli.',\n",
       "   'Methods for cloning relaxin are known to those of ordinary skill in the art. Polypeptide and polynucleotide sequences for relaxin and cloning of relaxin into host cells are detailed in U.S. Pat. Nos. 4,758,516; 5,166,191; 5,179,195, 5,945,402; and 5,759,807; all of which patents are herein incorporated by reference.',\n",
       "   'cDNA encoding relaxin is shown as SEQ TD NOs: 12 and the mature polypeptide amino acid sequence is shown as SEQ TD NO: 1.',\n",
       "   'TABLE\\u20031\\n\\n\\n\\n\\n\\n\\n\\nRelaxin\\u2003Sequences\\u2003Cited\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSEQ\\n\\n\\n\\n\\nID\\nSequence\\n\\n\\n\\nNO:\\nName\\nSequence\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n1\\nRelaxin\\nQLYSALANKCCHVGCTKRSLARFC\\n\\n\\n\\namino\\nDSWMEEVIKLCGRELVRAQIAICGMSTWS\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n2\\nRelaxin\\nQLYSALANKCCHVGCTKRSLARFC\\n\\n\\n\\namino\\nASWMEEVIKLCGRELVRAQIAICGMSTWS\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n\\nBl\\u2003Ala\\n\\n\\n\\n \\n\\n\\n3\\nPro-\\nDSWMEEVIKLCGRELVRAQIAICGMSTWSRREA\\n\\n\\n\\nrelaxin\\nEDLQVGQVELGGGPGAGSLQPLALEGSLQKRQL\\n\\n\\n\\namino\\nYSALANKCCHVGCTKRSLARFC\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n4\\nRelaxin\\nQLYSALANKCCHVGCTKRSLARFC\\n\\n\\n\\nA\\u2003chain,\\n\\n\\n\\n\\namino\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n5\\nRelaxin\\nDSWMEEVIKLCGRELVRAQIAICGMSTWS\\n\\n\\n\\nB\\u2003chain,\\n\\n\\n\\n\\namino\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n6\\nRelaxin\\nASWMEEVIKLCGRELVRAQIAICGMSTWS\\n\\n\\n\\nB\\u2003chain,\\n\\n\\n\\n\\namino\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n\\nwith\\u2003B1\\n\\n\\n\\n\\nAla\\n\\n\\n\\n \\n\\n\\n7\\nC\\u2003peptide\\nRREAEDLQVGQVELGGGPGAGSLQPLALEG\\n\\n\\n\\namino\\nSLQKR\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n8\\nRelaxin\\nMKKNIAFLLKR\\n\\n\\n\\nleader\\n\\n\\n\\n\\namino\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n9\\nInsulin\\nMIEGGR\\n\\n\\n\\nleader\\n\\n\\n\\n\\namino\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n10\\nRelaxin\\ncaactctacagtgcattggctaataaatgttg\\n\\n\\n\\nA\\u2003chain,\\nccatgttggttgtaccaaaagatctcttgcta\\n\\n\\n\\nnucleic\\ngattttgc\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n11\\nRelaxin\\ngactcatggatggaggaagttattaaattatg\\n\\n\\n\\nB\\u2003chain,\\ncggccgcgaattagttcgcgcgcagattgcca\\n\\n\\n\\nnucleic\\ntttgcggcatgagcacctggagc\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\ncaactctacagtgcattggctaataaatg\\n\\n\\n \\n\\n\\n12\\nRelaxin,\\nttgccatgttggttgtaccaaaagatctc\\n\\n\\n\\nA\\u2003and\\u2003B\\nttgctagattttgcgactcatggatggag\\n\\n\\n\\nchains,\\ngaagttattaaattatgcggccgcgaatt\\n\\n\\n\\nnucleic\\nagttcgcgcgcagattgccatttgcggca\\n\\n\\n\\nacid\\ntgagcacctggagc\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n13\\nRelaxin\\natgaaaaagaatatcgcatttcttcttaaa\\n\\n\\n\\nleader\\ncgg\\n\\n\\n\\nnucleic\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n14\\nInsulin\\natgattgaaggtggtcgt\\n\\n\\n\\nleader\\n\\n\\n\\n\\nnucleic\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence\\n\\n\\n\\n \\n\\n\\n15\\nExample\\nM\\u2003I\\u2003E\\u2003G\\u2003G\\u2003R\\u2003D\\u2003S\\u2003W\\u2003M\\u2003E\\u2003E\\u2003V\\u2003I\\u2003K\\u2003L\\u2003C\\u2003G\\u2003R\\u2003E\\u2003L\\u2003V\\u2003R\\u2003A\\u2003Q\\u2003I\\n\\n\\n\\nof\\u2003a\\nA\\u2003\\u2003\\u2003\\u2003\\u2003I\\u2003\\u2003\\u2003\\u2003\\u2003C\\u2003\\u2003\\u2003\\u2003G\\u2003\\u2003\\u2003\\u2003\\u2003\\u2003M\\u2003\\u2003\\u2003\\u2003\\u2003\\u2003S\\u2003\\u2003\\u2003\\u2003\\u2003T\\u2003\\u2003\\u2003\\u2003\\u2003W\\u2003\\u2003\\u2003\\u2003\\u2003\\u2003S\\n\\n\\n\\nrelaxin\\nR\\u2003\\u2003R\\u2003\\u2003E\\u2003\\u2003A\\u2003\\u2003E\\u2003\\u2003D\\u2003\\u2003L\\u2003\\u2003Q\\u2003\\u2003V\\u2003\\u2003G\\u2003\\u2003Q\\u2003\\u2003V\\u2003\\u2003E\\u2003\\u2003L\\u2003\\u2003G\\u2003\\u2003G\\u2003\\u2003G\\u2003\\u2003P\\n\\n\\n\\nexpression\\nG\\u2003\\u2003\\u2003A\\u2003\\u2003G\\u2003\\u2003S\\u2003\\u2003L\\u2003\\u2003Q\\u2003\\u2003P\\u2003\\u2003L\\u2003\\u2003A\\u2003\\u2003L\\u2003\\u2003E\\u2003\\u2003G\\u2003\\u2003S\\u2003\\u2003L\\u2003\\u2003\\u2003Q\\u2003\\u2003K\\u2003\\u2003\\u2003R\\n\\n\\n\\nconstruct\\nQ\\u2003L\\u2003Y\\u2003S\\u2003A\\u2003L\\u2003A\\u2003N\\u2003K\\u2003C\\u2003C\\u2003H\\u2003V\\u2003G\\u2003C\\u2003T\\u2003K\\u2003R\\u2003S\\u2003L\\u2003A\\u2003R\\u2003F\\u2003C\\n\\n\\n\\namino\\n\\n\\n\\n\\nacid\\n\\n\\n\\n\\nsequence',\n",
       "   'An introduced translation system that comprises an orthogonal tRNA (O-tRNA) and an orthogonal aminoacyl tRNA synthetase (O—RS) is used to express relaxin or relaxin analogs containing a non-naturally encoded amino acid. The O—RS preferentially aminoacylates the O-tRNA with a non-naturally encoded amino acid. In turn the translation system inserts the non-naturally encoded amino acid into the relaxin or relaxin analog, in response to an encoded selector codon. Suitable O—RS and O-tRNA sequences are described in WO 2006/068802 entitled “Compositions of Aminoacyl-tRNA Synthetase and Uses Thereof” (E9; SEQ TD NO: 16) and WO 2007/021297 entitled “Compositions of tRNA and Uses Thereof” (F13; SEQ ID NO: 17), which are incorporated by reference in their entirety herein.',\n",
       "   'TABLE 2\\n\\n\\n\\n\\n\\nSequences Cited\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSEQ ID NO:18\\nM. jannaschii mtRNACUATyr\\ntRNA\\n\\n\\nSEQ ID NO:19\\nHLAD03; an optimized amber supressor tRNA\\ntRNA\\n\\n\\nSEQ ID NO:20\\nHL325A; an optimizedAGGA frameshift supressor tRNA\\ntRNA\\n\\n\\nSEQ ID NO:21\\nAminoacyl tRNA synthetase for the incorporation of p-azido-L-phenylalanine\\nRS\\n\\n\\n\\np-Az-PheRS(6)\\n\\n\\n\\nSEQ ID NO:22\\nAminoacyl tRNA synthetase for the incorporation of p-benzoyl-L-phenylalanine\\nRS\\n\\n\\n\\np-BpaRS(1)\\n\\n\\n\\nSEQ ID NO:23\\nAminoacyl tRNA synthetase for the incorporation of propargyl-phenylalanine\\nRS\\n\\n\\n\\nPropargyl-PheRS\\n\\n\\n\\nSEQ ID NO:24\\nAminoacyl tRNA synthetase for the incorporation of propargyl-phenylalanine\\nRS\\n\\n\\n\\nPropargyl-PheRS\\n\\n\\n\\nSEQ ID NO:25\\nAminoacyl tRNA synthetase for the incorporation of propargyl-phenylalanine\\nRS\\n\\n\\n\\nPropargyl-PheRS\\n\\n\\n\\nSEQ ID NO:26\\nAminoacyl tRNA synthetase for the incorporation of p-azido-phenylalanine\\nRS\\n\\n\\n\\np-Az-PheRS(1)\\n\\n\\n\\nSEQ ID NO:27\\nAminoacyl tRNA synthetase for the incorporation of p-azido-phenylalanine\\nRS\\n\\n\\n\\np-Az-PheRS(3)\\n\\n\\n\\nSEQ ID NO:28\\nAminoacyl tRNA synthetase for the incorporation of p-azido-phenylalanine\\nRS\\n\\n\\n\\np-Az-PheRS(4)\\n\\n\\n\\nSEQ ID NO:29\\nAminoacyl tRNA synthetase for the incorporation of p-azido-phenylalanine\\nRS\\n\\n\\n\\np-Az-PheRS(2)\\n\\n\\n\\nSEQ ID NO:30\\nAminoacyl tRNA synthetase for the incorporation ofp-acetyl-phenylalanine (LWI)\\nRS\\n\\n\\nSEQ ID NO:31\\nAminoacyl tRNA synthetase for the incorporation ofp-acetyl -phenylalanine (LW5)\\nRS\\n\\n\\nSEQ ID NO:32\\nAminoacyl tRNA synthetase for the incorporation ofp-acetyl-phenylalanine (LW6)\\nRS\\n\\n\\nSEQ ID NO:33\\nAminoacyl tRNA synthetase for the incorporation ofp-azido-phenylalanine (AzPheRS-5)\\nRS\\n\\n\\nSEQ ID NO:34\\nAminoacyl tRNA synthetase for the incorporation of p-azido-phenylalanine (AzPheRS-6)\\nRS',\n",
       "   'The transformation of E. coli with plasmids containing the modified relaxin or relaxin analog gene and the orthogonal aminoacyl tRNA synthetase/tRNA pair (specific for the desired non-naturally encoded amino acid) allows the site-specific incorporation of non-naturally encoded amino acid into the relaxin polypeptide.',\n",
       "   'Wild type mature relaxin is amplified by PCR from a cDNA synthesis reaction using standard protocols and cloned into pET30 (NcoI-BamHI). Prior to or alternatively following sequence confirmation, relaxin including an N-terminal HIHIHH SGG sequence is subcloned into a suppression vector containing an amber suppressor tyrosyl tRNATyr/CUA from Methanococcus jannaschii (Mj tRNATyr/CUA) under constitutive control of a synthetic promoter derived from the E. coli lipoprotein promoter sequence (Miller, J. H., Gene, 1986), as well as well as the orthogonal tyrosyl-tRNA-synthetase (MjTyrRS) under control of the E. coli GlnRS promoter. Expression of relaxin is under control of the T7 promoter. Amber mutations are introduced using standard quick change mutation protocols (Stratagene; La Jolla, Calif.). Constructs are sequence verified.',\n",
       "   'Testing of long-acting relaxin compounds may be done using the STZ diabetic rat model (PCO 08-400-209).',\n",
       "   'Suppression with Para-Acetyl-Phenylalanine (pAcF)',\n",
       "   'Plasmids (e.g. pt_RLX_BA1_AV13am_p1395 (AXID2381)) were used to transform into the Escherichia coli strain W3110B57 to produce RLX-BA1-AV13pAF W3110 B2 strain of E. coli in which expression of the T7 polymerase was under control of an arabinose-inducible promoter. Overnight bacterial cultures were diluted 1:100 into shake flasks containing 2×YT culture media and grown at 37° C. to an OD600 of ˜0.8. Protein expression was induced by the addition of arabinose (0.2% final), and para-acetyl-phenylalanine (pAcF) to a final concentration of 4 mM. Cultures were incubated at 37° C. for 5 hours. Cells were pelleted and resuspended in B-PER lysis buffer (Pierce) 100 ul/OD/ml+10 ug/ml DNase and incubated at 37° C. for 30 min. Cellular material was removed by centrifugation and the supernatant removed. The pellet was re-suspended in an equal amount of SDS-PAGE protein loading buffer. All samples were loaded on a 4-12% PAGE gel with MES and DTT. Methods for purification of relaxin are known to those of ordinary skill in the art and are confirmed by SDS-PAGE, Western Blot analyses, or electrospray-ionization ion trap mass spectrometry and the like.',\n",
       "   'His-tagged mutant relaxin proteins can be purified using methods known to those of ordinary skill in the art. The ProBond Nickel-Chelating Resin (Invitrogen, Carlsbad, Calif.) may be used via the standard His-tagged protein purification procedures provided by the manufacturer. Functional measurements of the proteins may be done through methods known in the art, methods provided within this application and incorporated references, and alternatively an ELISA on live cells can be developed to assess relaxin polypeptides of the invention.',\n",
       "   'TABLE 3\\n\\n\\n\\n\\n\\n\\n\\nAnalyses of Relaxin Variants\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nLAL\\n\\nRP-\\nSE-\\n\\n\\nBatch/\\n\\n(EU/\\n\\nHPLC\\nHPLC\\n\\n\\nDescription\\nSDS-PAGE\\nmg)\\none. at A280\\npurity\\npurity\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nWT Relaxin3\\nNR Major\\nNT\\n2.4\\n91.2%\\nNT\\n\\n\\n\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates near\\n\\n\\n\\n\\n\\n\\n\\nthe 6 kDa\\n\\n\\n\\n\\n\\n\\n\\nMW standard\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\nNR Major\\n7.7\\n3.4\\n\\u2009100%\\nNT\\n\\n\\nQ1PAF1,3,4\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates\\n\\n\\n\\n\\n\\n\\n\\nbetween the\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n49 and 38\\n3.6\\n2.4\\n\\u2009100%\\nNT\\n\\n\\nA5pAF1,3,4\\nkDa MW\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\nstandards\\n16.1\\n2.4\\n\\u2009100%\\nNT\\n\\n\\nR18pAF1,3,4\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\nNT\\n0.6\\n99.8%\\nNT\\n\\n\\nE5PAF2,3\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\n10.5\\n2.3\\n99.4%\\nNT\\n\\n\\nV7pAF2,3,4\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\nNT\\n2.0\\n99.0%\\nNT\\n\\n\\nA18pAF2,3\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\n9.1\\n2.2\\n99.4%\\nNT\\n\\n\\nW28pAF2,3,4\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\n5.1\\n3.7\\n99.5%\\nNT\\n\\n\\nV13pAF1,3,4\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\nNT\\n0.6\\n99.6%\\nNT\\n\\n\\nE5pAF2\\n\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\n\\n0.0\\n1.6\\n99.5%\\nNT\\n\\n\\nL2pAF1\\n\\n\\n\\n\\n\\n\\n\\nWT Relaxin3\\nNR Major\\n0.1\\n2.5\\n85.6%\\nNT\\n\\n\\n\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates near\\n\\n\\n\\n\\n\\n\\n\\nthe 6 kDa\\n\\n\\n\\n\\n\\n\\n\\nMW standard\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\nNR Major\\n<0.4\\n2.3\\n99.3%\\n99.3%\\n\\n\\nQ1pAF1,3\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates\\n\\n\\n\\n\\n\\n\\n\\nbetween the\\n\\n\\n\\n\\n\\n\\n\\n49 and 38\\n\\n\\n\\n\\n\\n\\n\\nkDa MW\\n\\n\\n\\n\\n\\n\\n\\nstandards\\n\\n\\n\\n\\n\\n\\n5K PEGylated\\nNR Major\\n5.4\\n1.5\\n99.4%\\n99.8%\\n\\n\\nV13pAF1,3\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates near\\n\\n\\n\\n\\n\\n\\n\\nthe 14 kDa\\n\\n\\n\\n\\n\\n\\n\\nMW standard\\n\\n\\n\\n\\n\\n\\n10K PEGylated\\nNR Major\\n9.8\\n1.8\\n99.4%\\n99.5%\\n\\n\\nV13pAF13\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates\\n\\n\\n\\n\\n\\n\\n\\nbetween the\\n\\n\\n\\n\\n\\n\\n\\n28 and 17\\n\\n\\n\\n\\n\\n\\n\\nkDaMW\\n\\n\\n\\n\\n\\n\\n\\nstandards\\n\\n\\n\\n\\n\\n\\n20K PEGylated\\nNR Major\\n4.7\\n1.9\\n99.0%\\n99.5%\\n\\n\\nV13pAF1,3\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates\\n\\n\\n\\n\\n\\n\\n\\nbetween the\\n\\n\\n\\n\\n\\n\\n\\n49 and 38\\n\\n\\n\\n\\n\\n\\n\\nkDaMW\\n\\n\\n\\n\\n\\n\\n\\nstandards\\n\\n\\n\\n\\n\\n\\n3 OK PEGylated\\nNR Major\\n8.0\\n2.1\\n99.5%\\n99.8%\\n\\n\\nV13pAF1,3\\nband\\n\\n\\n\\n\\n\\n\\n\\nmigrates\\n\\n\\n\\n\\n\\n\\n\\nbetween the\\n\\n\\n\\n\\n\\n\\n\\n62 and 49\\n\\n\\n\\n\\n\\n\\n\\nkDaMW\\n\\n\\n\\n\\n\\n\\n\\nstandards\\n\\n\\n\\n\\n\\nSamples not reduced (NR); NT = Not Tested\\n\\n\\n1pAcF substitution located on A-chain\\n\\n\\n2pAcF substitution located on B-chain\\n\\n\\n3Asp to Ala substitution on B-chain (BA1)\\n\\n\\n4used for initial PK',\n",
       "   'TABLE 4\\n\\n\\n\\n\\n\\n\\n\\nRelaxin Variant Loss of Activity\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nIn vitro\\n\\n\\n\\n\\nFermentor\\n\\n\\n\\n\\nActivity\\nPEG\\nIn vivo PK Term\\nIn vivo\\nShake\\nTiter\\nAnalytical\\n\\n\\nVariant\\nFold Loss\\nsize\\nHL (hr)\\npharm\\nflask\\n(cell paste)\\ncharacterization\\n\\n\\n\\n\\n\\nRLX-A-\\n17\\n5K,\\n2.6(5K), 8.7(10K),\\n\\nyes\\n\\u20031 gm/L\\nSDS-PAGE, conc.,\\n\\n\\nAV13\\n\\n10K,\\n13.8(20K), 26.8(30K)\\n\\n\\n\\nLAL, RP-HPLC, SE-\\n\\n\\n\\n\\n20K,\\n\\n\\n\\n\\nHPLC\\n\\n\\n\\n\\n30K\\n\\n\\nRLX-A-\\n12\\n20K\\n10.7\\n20K\\nyes\\n720 mg/L\\nSDS-PAGE, conc.,\\n\\n\\nAQ1\\n\\n\\n\\nshows\\n\\n\\nRP-HPLC, LAL, SE-\\n\\n\\n\\n\\n\\n\\nefficacy\\n\\n\\nHPLC\\n\\n\\nRLX-A-\\n12\\n20K\\n12.2\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nAA5\\n\\n\\n\\n\\n\\n\\nRP-HPLC, LAL\\n\\n\\nRLX-A-\\n15\\n20K\\n13.1\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nBV7\\n\\n\\n\\n\\n\\n\\nRP-HPLC, LAL\\n\\n\\nRLX-A-\\n17\\n20K\\n\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nAL2\\n\\n\\n\\n\\n\\n\\nRP-HPLC\\n\\n\\nRLX-A-\\n17\\n20K\\n\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nBE5\\n\\n\\n\\n\\n\\n\\nRP-HPLC\\n\\n\\nRLX-A-\\n21\\n20K\\n12.5\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nAR18\\n\\n\\n\\n\\n\\n\\nRP-HPLC, LAL\\n\\n\\nRLX-\\n22\\n20K\\n\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nBE5\\n\\n\\n\\n\\n\\n\\nRP-HPLC\\n\\n\\nRLX-A-\\n48\\n20K\\n\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nBA18\\n\\n\\n\\n\\n\\n\\nRP-HPLC\\n\\n\\nRLX-A-\\n48\\n20K\\n13.9\\n\\nyes\\n\\nSDS-PAGE, conc.,\\n\\n\\nBW28\\n\\n\\n\\n\\n\\n\\nRP-HPLC, LAL']},\n",
       " {'number': 'Example 3',\n",
       "  'title': '',\n",
       "  'content': ['This example details expression of Pro-relaxin polypeptides by E. coli.',\n",
       "   'E. coli expressed Pro-relaxin as a single chain protein composed of 88 amino acids. Upon digestion with trypsin and carboxypeptidase, a connecting peptide and leader sequence are removed. The resulting peptide is a small 6 kDa two-chain peptide member of the insulin superfamily which consists of a 24 residue A-chain and a 29 residue B-chain. The structural fold is characterized by two peptide chains which are held together by two interchain (Cys11-Cys36, and Cys24-Cys48) and one intrachain (Cys10-Cys15) disulfide bonds. The tertiary structure based on a crystal structure of human relaxin-2 revealed a compact fold comprising three helical segments and a short extended region that enclose a hydrophobic core.',\n",
       "   'Relaxin with one or more non-naturally encoded amino acid(s) provides a unique chemistry and enables a specific PEGylated recombinant variant containing a biosynthetically incorporated, chemically reactive, carbonyl group, by replacement of a natural amino acid with para-acetylphenylalanine (pAcF), providing a unique covalent site of attachment for a poly(ethylene) glycol (PEG).']},\n",
       " {'number': 'Example 4',\n",
       "  'title': '',\n",
       "  'content': ['This example details expression of Pro-relaxin polypeptides by E. coli.',\n",
       "   'This example describes the scale up of relaxin polypeptide production using a five (5) liter fermentor. These methods and scale up may also be used for 10L, 30L, 150L and 1000L batches. In some embodiments of the present invention, at least 2 g of relaxin protein is produced for each liter of cell culture. In another embodiment of the present invention, at least 4 g of relaxin protein is produced for each liter of cell culture. In another embodiment of the present invention, at least 6 g of relaxin protein is produced for each liter of cell culture. In another embodiment of the present invention, at least 8 g of relaxin protein is produced for each liter of cell culture. In another embodiment of the present invention, at least 10 g of relaxin protein is produced for each liter of cell culture. In another embodiment of the present invention, at least 15 g of relaxin protein is produced for each liter of cell culture. In another embodiment of the present invention, at least 20 g of relaxin protein is produced for each liter of cell culture.',\n",
       "   'Escherichia coli strain W3110B57 [F-IN(rrnD-rrnE) lambda-araB::g1 tetA fhuA::dhfr ompT::cat] harboring plasmid pt_RLX_BA1_AV13am_p1395 (AXID2381) was used to produce RLX-BA1-AV13pAF. A single research cell bank (RCB) vial was removed from −80° C. and thawed at room temperature, then 50 μL was used to inoculate 50 mL of Seed Media (a chemically defined medium) supplemented with 50 μg/mL kanamycin sulfate in a 250 mL baffled Erlenmeyer flask. The primary seed culture was grown for approximately 18 hours at 37° C. and 250 rpm (1-inch throw). The primary seed culture was sub-cultured into a secondary seed culture to an optical density measured at 600 nm wavelength (0D600) of 0.05 in a 500 mL baffled Erlenmeyer flask containing 100 mL of Seed Medium supplemented with 50 μg/mL kanamycin sulfate. The secondary seed culture was grown at 37° C. and 250 rpm (1-inch throw) for approximately 8 hours or when the OD600 reached between 2 and 4.',\n",
       "   'Sartorius Biostat B 5-L vessels were filled with 2.1-L of Production Media (a chemically defined medium) supplemented with 50 μg/L of kanamycin sulfate. Secondary seed cultures were used to inoculate the fermentors to an initial OD600 of 0.035. The cultures were grown 37° C. and the dissolved oxygen was set to maintain 30% (air saturation) with a primary agitation (480-1200 rpm) cascade and a secondary O2 cascade. An air flow rate of 5 LPM with 6 psi back pressure was maintained throughout the fermentation. The pH of the culture was set at 7.2±0.05 with the addition of 15% ammonium hydroxide and Dow Chemical P2000 antifoam was added as needed for foam control. When the culture reached an OD600 of between 35±5 (when the initial glycerol in the batch medium was nearly depleted), a bolus feed of 200 mL was given initiated and at the same time the pH set point was adjusted from 7.2 to 6.6. After the initial bolus feed, a continuous feed was initiated at a rate of 0.25 mL/L/min and continued until harvest. Immediately after starting the feed, 2.5 mL/L (0.2 g/L final culture volume) of a 100 g/L L-Ala-pAcF dipeptide solution made in water was added to the fermentor. Fifteen minutes after dipeptide addition, the culture was induced by adding L-arabinose (recipe given in PTR-FGF-002) to a concentration of 2 g/L (final culture volume). The culture was grown 6 hours after arabinose addition and harvested.',\n",
       "   'FIG. 5 shows an SDS-PAGE gel of the prorelaxin produced by these methods with a chain Bi amino acid as Ala and a para-acetyl phenylalanine in the 13th amino acid position of the A chain, substituted for valine.']},\n",
       " {'number': 'Example 5',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 5,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 3, 5, 7, 9, 11) and each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NOs: 4, 6, 8, 10, 12) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-acetyl-phenylalanine into relaxin are SEQ ID NO: 1 and 2 (A and B chains of relaxin), and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and 15, 29, 30 or 31 (TyrRS LW1, 5, or 6) described in Example 2 above.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 3 and N is approximately 5,000 MW. The purified relaxin containing p-acetylphenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 6',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 20,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding amino acids in SEQ ID NOs: 2 and 3) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-aminophenylalanine into relaxin are SEQ ID NO: 4 and 5 or 6 (A and B chains of relaxin), and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and sequences described above and incorporated for site-specific incorporation of p-aminophenylalanine.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form: R-PEG(N)—O—(CH2)n-O—NH2 where R is methyl, n is 3 and N is approximately 20,000 MW. The purified relaxin containing p-aminophenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 7',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 20,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding amino acid positions in SEQ ID NOs: 2 and 3) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-aminophenylalanine into relaxin are SEQ ID NO: 13, and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and sequences described above and incorporated for site-specific incorporation of p-aminophenylalanine.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 3 and N is approximately 20,000 MW. The purified relaxin containing p-aminophenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 8',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 20,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding amino acid positions in SEQ ID NOs: 2 and 3) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-aminophenylalanine into relaxin are SEQ ID NO: 1, and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and sequences described above and incorporated for site-specific incorporation of p-aminophenylalanine.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 3 and N is approximately 20,000 MW. The purified relaxin containing p-aminophenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 9',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 30,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-aminophenylalanine into relaxin are SEQ ID NO: 1 (or SEQ ID NO: 2, or 3), and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and sequences described above and incorporated for site-specific incorporation of p-aminophenylalanine.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 3 and N is approximately 30,000 MW. The purified relaxin containing p-aminophenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 10',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 40,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-aminophenylalanine into relaxin are SEQ ID NO: 13 (or SEQ ID NO: 1, 2, or 14), and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and sequences described above and incorporated for site-specific incorporation of p-aminophenylalanine.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 3 and N is approximately 40,000 MW. The purified relaxin containing p-aminophenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 11',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of a carbonyl-containing amino acid and subsequent reaction with an aminooxy-containing PEG.',\n",
       "   'This Example demonstrates a method for the generation of a relaxin polypeptide that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently reacted with an aminooxy-containing PEG of approximately 10,000 MW. Each of the residues before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3) is separately substituted with a non-naturally encoded amino acid having the following structure:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-aminophenylalanine into relaxin are SEQ ID NO: 13 (or corresponding positions in SEQ ID NO: 1, 2, or 14), and SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii), and sequences described above and incorporated for site-specific incorporation of p-aminophenylalanine.',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 3 and N is approximately 10,000 MW. The purified relaxin containing p-aminophenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10-16 hours at room temperature (Jencks, W. J. Am. Chem. Soc. 1959, 81, pp 475). The PEG-relaxin is then diluted into appropriate buffer for immediate purification and analysis.']},\n",
       " {'number': 'Example 12',\n",
       "  'title': '',\n",
       "  'content': ['Conjugation with a PEG consisting of a hydroxylamine group linked to the PEG via an amide linkage.',\n",
       "   'A PEG reagent having the following structure is coupled to a ketone-containing non-naturally encoded amino acid using the procedure described in Examples 3-9:',\n",
       "   'R-PEG(N)—O—(CH2)2-NH—C(O)(CH2)n—O—NH2',\n",
       "   'where R=methyl, n=4 and N is approximately 5,000 MW-40,000 MW. The reaction, purification, and analysis conditions are as described and known in the art.']},\n",
       " {'number': 'Example 13',\n",
       "  'title': '',\n",
       "  'content': ['This example details the introduction of two distinct non-naturally encoded amino acids into relaxin polypeptides and relaxin analog polypeptides.',\n",
       "   'This example demonstrates a method for the generation of a relaxin polypeptide that incorporates non-naturally encoded amino acid comprising a ketone functionality at two positions among the following residues: before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3). The relaxin polypeptide is prepared as described above, except that the selector codon is introduced at two distinct sites within the nucleic acid.']},\n",
       " {'number': 'Example 14',\n",
       "  'title': '',\n",
       "  'content': ['This example details conjugation of relaxin polypeptide or relaxin analog polypeptide to a hydrazide-containing PEG and subsequent in situ reduction.',\n",
       "   'A relaxin polypeptide incorporating a carbonyl-containing amino acid is prepared according to the procedure described above. Once modified, a hydrazide-containing PEG having the following structure is conjugated to the relaxin polypeptide:',\n",
       "   'R-PEG(N)—O—(CH2)2-NH—C(O)(CH2)n-X—NH—NH2',\n",
       "   'where R=methyl, n=2 and N═5,000; 10,000, 20,000; 30,000; or 40,000 MW and X is a carbonyl (C═O) group. The purified relaxin containing p-acetylphenylalanine is dissolved at between 0.1-10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 1 to 100-fold excess of hydrazide-containing PEG, and the corresponding hydrazone is reduced in situ by addition of stock 1M NaCNBH3 (Sigma Chemical, St. Louis, Mo.), dissolved in H2O, to a final concentration of 10-50 mM. Reactions are carried out in the dark at 4° C. to RT for 18-24 hours. Reactions are stopped by addition of 1 M Tris (Sigma Chemical, St. Louis, Mo.) at about pH 7.6 to a final Tris concentration of 50 mM or diluted into appropriate buffer for immediate purification.']},\n",
       " {'number': 'Example 15',\n",
       "  'title': '',\n",
       "  'content': ['This example details conjugation of relaxin polypeptide or relaxin analog polypeptide to a hydrazide-containing PEG and subsequent in situ reduction.',\n",
       "   'A relaxin polypeptide incorporating a carbonyl-containing amino acid is prepared according to the procedure described above. Once modified, a hydrazide-containing PEG having the following structure is conjugated to the relaxin polypeptide:',\n",
       "   'R-PEG(N)—O—(CH2)2-NH—C(O)(CH2)n-X—NH—NH2',\n",
       "   'where R=methyl, n=2 and N═20,000 MW and X is a carbonyl (C═O) group. The purified relaxin containing p-acetylphenylalanine is dissolved at between 0.1-10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, Mo.) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, Mo.) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, Mo.) pH 4.5, is reacted with a 1 to 100-fold excess of hydrazide-containing PEG, and the corresponding hydrazone is reduced in situ by addition of stock 1M NaCNBH3 (Sigma Chemical, St. Louis, Mo.), dissolved in H2O, to a final concentration of 10-50 mM. Reactions are carried out in the dark at 4° C. to RT for 18-24 hours. Reactions are stopped by addition of 1 M Tris (Sigma Chemical, St. Louis, Mo.) at about pH 7.6 to a final Tris concentration of 50 mM or diluted into appropriate buffer for immediate purification.']},\n",
       " {'number': 'Example 16',\n",
       "  'title': '',\n",
       "  'content': ['This example details introduction of an alkyne-containing amino acid into a relaxin polypeptide or relaxin analog polypeptide and derivatization with mPEG-azide.',\n",
       "   'The following residues, before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3), are each substituted with the following non-naturally encoded amino acid:',\n",
       "   '',\n",
       "   'The sequences utilized for site-specific incorporation of p-propargyl-tyrosine into relaxin are SEQ ID NO: 1 (or corresponding positions in SEQ ID NO:2 or 3), SEQ ID NO: 16 or 17 (muttRNA, M. jannaschii ), and 22, 23 or 24 described above. The relaxin polypeptide containing the propargyl tyrosine is expressed in E. coli and purified using the conditions described above.',\n",
       "   'The purified relaxin containing propargyl-tyrosine dissolved at between 0.1-10 mg/mL in PB buffer (100 mM sodium phosphate, 0.15 M NaCl, pH=8) and a 10 to 1000-fold excess of an azide-containing PEG is added to the reaction mixture. A catalytic amount of CuSO4 and Cu wire are then added to the reaction mixture. After the mixture is incubated (including but not limited to, about 4 hours at room temperature or 37° C., or overnight at 4° C.), H2O is added and the mixture is filtered through a dialysis membrane. The sample can be analyzed for the addition, including but not limited to, by similar procedures described in Example 3. In this Example, the PEG will have the following structure:',\n",
       "   'R-PEG(N)—O—(CH2)2-NH—C(O)(CH2)n-N3',\n",
       "   'where R is methyl, n is 4 and N═5,000; 10,000, 20,000; 30,000; or 40,000 MW.']},\n",
       " {'number': 'Example 17',\n",
       "  'title': '',\n",
       "  'content': ['This example details substitution of a large, hydrophobic amino acid in a relaxin polypeptide with propargyl tyrosine.',\n",
       "   'A Phe, Trp or Tyr residue present within one the following regions of relaxin: before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3), is substituted with the following non-naturally encoded amino acid as described above:',\n",
       "   '',\n",
       "   'Once modified, a PEG is attached to the relaxin polypeptide variant comprising the alkyne-containing amino acid. The PEG will have the following structure:',\n",
       "   'Me-PEG(N)—O—(CH2)2-N3',\n",
       "   'and coupling procedures would follow those in examples above. This will generate a relaxin polypeptide variant comprising a non-naturally encoded amino acid that is approximately isosteric with one of the naturally-occurring, large hydrophobic amino acids and which is modified with a PEG derivative at a distinct site within the polypeptide.']},\n",
       " {'number': 'Example 18',\n",
       "  'title': '',\n",
       "  'content': ['This example details generation of a relaxin polypeptide homodimer, heterodimer, homomultimer, or heteromultimer separated by one or more PEG linkers. Relaxin polypeptide multimers may be formed between proinsulins or between mature A and B chain relaxin polypeptides of the invention.',\n",
       "   'The alkyne-containing relaxin polypeptide variant produced in the example above is reacted with a bifunctional PEG derivative of the form:',\n",
       "   'N3-(CH2)n-C(O)—NH—(CH2)2-O-PEG(N)—O—(CH2)2-NH—C(O)—(CH2)n-N3',\n",
       "   'where n is 4 and the PEG has an average MW of approximately 5,000; 10,000; 20,000; 30,000; or 40,000 MW to generate the corresponding relaxin polypeptide homodimer where the two relaxin molecules are physically separated by PEG. In an analogous manner a relaxin polypeptide may be coupled to one or more other polypeptides to form heterodimers, homomultimers, or heteromultimers. Coupling, purification, and analyses will be performed as in the examples above.']},\n",
       " {'number': 'Example 19',\n",
       "  'title': '',\n",
       "  'content': ['This example details generation of a relaxin polypeptide homodimer, heterodimer, homomultimer, or heteromultimer separated by one or more PEG linkers. Relaxin polypeptide multimers may be formed between A chains and other A chains or B chains and other B chains.',\n",
       "   'The alkyne-containing relaxin polypeptide variant produced in the example above is reacted with a bifunctional PEG derivative of the form:',\n",
       "   'N3-(CH2)n-C(O)—NH—(CH2)2-O-PEG(N)—O—(CH2)2-NH—C(O)—(CH2)n-N3',\n",
       "   'where n is 4 and the PEG has an average MW of approximately 5,000; 10,000; 20,000; 30,000; or 40,000 MW to generate the corresponding relaxin polypeptide homodimer where the two relaxin molecules are physically separated by PEG. In an analogous manner a relaxin polypeptide may be coupled to one or more other polypeptides to form heterodimers, homomultimers, or heteromultimers. Coupling, purification, and analyses will be performed as in the examples above.']},\n",
       " {'number': 'Example 20',\n",
       "  'title': '',\n",
       "  'content': ['This example details coupling of a saccharide moiety to a relaxin polypeptide.',\n",
       "   'One residue of the following is substituted with the non-naturally encoded amino acid below: before position 1 (i.e. at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54 (i.e., at the carboxyl terminus of the protein of SEQ ID NO: 1 or the corresponding positions in SEQ ID NOs: 2 and 3), as described above.',\n",
       "   '',\n",
       "   'Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid is reacted with a p-linked aminooxy analogue of N-acetylglucosamine (GlcNAc). The relaxin polypeptide variant (10 mg/mL) and the aminooxy saccharide (21 mM) are mixed in aqueous 100 mM sodium acetate buffer (pH 5.5) and incubated at 37° C. for 7 to 26 hours. A second saccharide is coupled to the first enzymatically by incubating the saccharide-conjugated relaxin polypeptide (5 mg/mL) with UDP-galactose (16 mM) and p-1,4-galacytosyltransferase (0.4 units/mL) in 150 mM HEPES buffer (pH 7.4) for 48 hours at ambient temperature (Schanbacher et al. J. Biol. Chem. 1970, 245, 5057-5061).']},\n",
       " {'number': 'Example 22',\n",
       "  'title': '',\n",
       "  'content': ['This example details generation of a PEGylated relaxin polypeptide antagonist.',\n",
       "   'A residue, including but not limited to, those involved in relaxin receptor binding is substituted with the following non-naturally encoded amino acid as described above. Once modified, the relaxin polypeptide variant comprising the carbonyl-containing amino acid will be reacted with an aminooxy-containing PEG derivative of the form:',\n",
       "   'R-PEG(N)—O—(CH2)n-O—NH2',\n",
       "   'where R is methyl, n is 4 and N is 5,000; 10,000; 20,000; 30,000; or 40,000 MW to generate a relaxin polypeptide antagonist comprising a non-naturally encoded amino acid that is modified with a PEG derivative at a single site within the polypeptide. Coupling, purification, and analyses are performed as described above.']},\n",
       " {'number': 'Example 21',\n",
       "  'title': '',\n",
       "  'content': ['Generation of a relaxin polypeptide homodimer, heterodimer, homomultimer, or heteromultimer in which the relaxin molecules are linked directly',\n",
       "   'A relaxin polypeptide variant comprising the alkyne-containing amino acid can be directly coupled to another relaxin polypeptide variant comprising the azido-containing amino acid. In an analogous manner a relaxin polypeptide polypeptide may be coupled to one or more other polypeptides to form heterodimers, homomultimers, or heteromultimers. More description regarding multimers which may be formed is provided above in Examples 16 and 17 and coupling, purification, and analyses are performed as described above.']},\n",
       " {'number': 'Example 22',\n",
       "  'title': '',\n",
       "  'content': ['',\n",
       "   'The polyalkylene glycol (P-OH) is reacted with the alkyl halide (A) to form the ether (B). In these compounds, n is an integer from one to nine and R′ can be a straight- or branched-chain, saturated or unsaturated C1, to C20 alkyl or heteroalkyl group. R′ can also be a C3 to C7 saturated or unsaturated cyclic alkyl or cyclic heteroalkyl, a substituted or unsubstituted aryl or heteroaryl group, or a substituted or unsubstituted alkaryl (the alkyl is a C1 to C20 saturated or unsaturated alkyl) or heteroalkaryl group. Typically, PEG-OH is polyethylene glycol (PEG) or monomethoxy polyethylene glycol (mPEG) having a molecular weight of 800 to 40,000 Daltons (Da).']},\n",
       " {'number': 'Example 23',\n",
       "  'title': '',\n",
       "  'content': ['mPEG-OH+Br—CH2—C≡CH→mPEGO—CH2—C≡CH',\n",
       "   'mPEG-OH with a molecular weight of 20,000 Da (mPEG-OH 20 kDa; 2.0 g, 0.1 mmol, Sunbio) was treated with NaH (12 mg, 0.5 mmol) in THF (35 mL). A solution of propargyl bromide, dissolved as an 80% weight solution in xylene (0.56 mL, 5 mmol, 50 equiv., Aldrich), and a catalytic amount of KI were then added to the solution and the resulting mixture was heated to reflux for 2 hours. Water (1 mL) was then added and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. This CH2Cl2 solution was added to diethyl ether (150 mL) drop-wise. The resulting precipitate was collected, washed with several portions of cold diethyl ether, and dried to afford propargyl-O—PEG.']},\n",
       " {'number': 'Example 24',\n",
       "  'title': '',\n",
       "  'content': ['mPEG-OH+Br—(CH2)3—C≡CH→mPEG-O—(CH2)3—C≡CH',\n",
       "   'The mPEG-OH with a molecular weight of 20,000 Da (mPEG-OH 20 kDa; 2.0 g, 0.1 mmol, Sunbio) was treated with NaH (12 mg, 0.5 mmol) in THF (35 mL). Fifty equivalents of 5-bromo-1-pentyne (0.53 mL, 5 mmol, Aldrich) and a catalytic amount of KI were then added to the mixture. The resulting mixture was heated to reflux for 16 hours. Water (1 mL) was then added and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. This CH2Cl2 solution was added to diethyl ether (150 mL) drop-wise. The resulting precipitate was collected, washed with several portions of cold diethyl ether, and dried to afford the corresponding alkyne. 5-chloro-1-pentyne may be used in a similar reaction.']},\n",
       " {'number': 'Example 25',\n",
       "  'title': '',\n",
       "  'content': ['m-HOCH2C6H4OH+NaOH+Br—CH2—C≡CH→m-HOCH2C6H4O—CH2—C≡CH\\u2003\\u2003(1)',\n",
       "   'm-HOCH2C6H4O—CH2—C≡CH+MsCl+N(Et)3→m-MsOCH2C6H4O—CH2—C≡CH\\u2003\\u2003(2)',\n",
       "   'm-MsOCH2C6H4O—CH2—C≡CH+LiBr→m-Br—CH2C6H4O—CH2—C≡CH\\u2003\\u2003(3)',\n",
       "   'mPEG-OH+m-Br—CH2C6H4O—CH2—C≡CH→mPEG-O—CH2—C6H4O—CH2—C≡CH\\u2003\\u2003(4)',\n",
       "   'To a solution of 3-hydroxybenzylalcohol (2.4 g, 20 mmol) in THE (50 mL) and water (2.5 mL) was first added powdered sodium hydroxide (1.5 g, 37.5 mmol) and then a solution of propargyl bromide, dissolved as an 80% weight solution in xylene (3.36 mL, 30 mmol). The reaction mixture was heated at reflux for 6 hours. To the mixture was added 10% citric acid (2.5 mL) and the solvent was removed under vacuum. The residue was extracted with ethyl acetate (3×15 mL) and the combined organic layers were washed with saturated NaCl solution (10 mL), dried over MgSO4 and concentrated to give the 3-propargyloxybenzyl alcohol.',\n",
       "   'Methanesulfonyl chloride (2.5 g, 15.7 mmol) and triethylamine (2.8 mL, 20 mmol) were added to a solution of compound 3 (2.0 g, 11.0 mmol) in CH2Cl2 at 0° C. and the reaction was placed in the refrigerator for 16 hours. A usual work-up afforded the mesylate as a pale yellow oil. This oil (2.4 g, 9.2 mmol) was dissolved in THE (20 mL) and LiBr (2.0 g, 23.0 mmol) was added. The reaction mixture was heated to reflux for 1 hour and was then cooled to room temperature. To the mixture was added water (2.5 mL) and the solvent was removed under vacuum. The residue was extracted with ethyl acetate (3×15 mL) and the combined organic layers were washed with saturated NaCl solution (10 mL), dried over anhydrous Na2SO4, and concentrated to give the desired bromide.',\n",
       "   'mPEG-OH 20 kDa (1.0 g, 0.05 mmol, Sunbio) was dissolved in THE (20 mL) and the solution was cooled in an ice bath. NaH (6 mg, 0.25 mmol) was added with vigorous stirring over a period of several minutes followed by addition of the bromide obtained from above (2.55 g, 11.4 mmol) and a catalytic amount of KI. The cooling bath was removed and the resulting mixture was heated to reflux for 12 hours. Water (1.0 mL) was added to the mixture and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. Dropwise addition to an ether solution (150 mL) resulted in a white precipitate, which was collected to yield the PEG derivative.']},\n",
       " {'number': 'Example 26',\n",
       "  'title': '',\n",
       "  'content': ['mPEG-NH2+X—C(O)—(CH2)n—C≡CR′→mPEG-NH—C(O)—(CH2)n—C≡CR′',\n",
       "   'The terminal alkyne-containing poly(ethylene glycol) polymers can also be obtained by coupling a poly(ethylene glycol) polymer containing a terminal functional group to a reactive molecule containing the alkyne functionality as shown above. n is between 1 and 10. R′ can be H or a small alkyl group from C1 to C4.']},\n",
       " {'number': 'Example 27',\n",
       "  'title': '',\n",
       "  'content': ['HO2C—(CH2)2—C≡CH+NHS+DCC→NHSO—C(O)—(CH2)2—C≡CH\\u2003\\u2003(1)',\n",
       "   'mPEG-NH2+NHSO—C(O)—(CH2)2—C≡CH→mPEG-NH—C(O)—(CH2)2—C≡CH\\u2003\\u2003(2)',\n",
       "   '4-pentynoic acid (2.943 g, 3.0 mmol) was dissolved in CH2Cl2 (25 mL). N-hydroxysuccinimide (3.80 g, 3.3 mmol) and DCC (4.66 g, 3.0 mmol) were added and the solution was stirred overnight at room temperature. The resulting crude NHS ester 7 was used in the following reaction without further purification.',\n",
       "   'mPEG-NH2 with a molecular weight of 5,000 Da (mPEG-NH2, 1 g, Sunbio) was dissolved in THF (50 mL) and the mixture was cooled to 4° C. NHS ester 7 (400 mg, 0.4 mmol) was added portion-wise with vigorous stirring. The mixture was allowed to stir for 3 hours while warming to room temperature. Water (2 mL) was then added and the solvent was removed under vacuum. To the residue was added CH2Cl2 (50 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. This CH2Cl2 solution was added to ether (150 mL) drop-wise. The resulting precipitate was collected and dried in vacuo.']},\n",
       " {'number': 'Example 28',\n",
       "  'title': '',\n",
       "  'content': ['This Example represents the preparation of the methane sulfonyl ester of poly(ethylene glycol), which can also be referred to as the methanesulfonate or mesylate of poly(ethylene glycol). The corresponding tosylate and the halides can be prepared by similar procedures.',\n",
       "   'mPEG-OH+CH3SO2Cl+N(Et)3→mPEG-O—SO2CH3→mPEG-N3',\n",
       "   'The mPEG-OH (MW=3,400, 25 g, 10 mmol) in 150 mL of toluene was azeotropically distilled for 2 hours under nitrogen and the solution was cooled to room temperature. 40 mL of dry CH2Cl2 and 2.1 mL of dry triethylamine (15 mmol) were added to the solution. The solution was cooled in an ice bath and 1.2 mL of distilled methanesulfonyl chloride (15 mmol) was added dropwise. The solution was stirred at room temperature under nitrogen overnight, and the reaction was quenched by adding 2 mL of absolute ethanol. The mixture was evaporated under vacuum to remove solvents, primarily those other than toluene, filtered, concentrated again under vacuum, and then precipitated into 100 mL of diethyl ether. The filtrate was washed with several portions of cold diethyl ether and dried in vacuo to afford the mesylate.',\n",
       "   'The mesylate (20 g, 8 mmol) was dissolved in 75 ml of THF and the solution was cooled to 4° C. To the cooled solution was added sodium azide (1.56 g, 24 mmol). The reaction was heated to reflux under nitrogen for 2 hours. The solvents were then evaporated and the residue diluted with CH2Cl2 (50 mL). The organic fraction was washed with NaCl solution and dried over anhydrous MgSO4. The volume was reduced to 20 ml and the product was precipitated by addition to 150 ml of cold dry ether.']},\n",
       " {'number': 'Example 29',\n",
       "  'title': '',\n",
       "  'content': ['N3—C6H4—CO2H→N3—C6H4CH2OH\\u2003\\u2003(1)',\n",
       "   'N3—C6H4CH2OH→Br—CH2—C6H4—N3\\u2003\\u2003(2)',\n",
       "   'mPEG-OH+Br—CH2—C6H4—N3→mPEGO—CH2—C6H4—N3\\u2003\\u2003(3)',\n",
       "   '4-azidobenzyl alcohol can be produced using the method described in U.S. Pat. No. 5,998,595, which is incorporated by reference herein. Methanesulfonyl chloride (2.5 g, 15.7 mmol) and triethylamine (2.8 mL, 20 mmol) were added to a solution of 4-azidobenzyl alcohol (1.75 g, 11.0 mmol) in CH2Cl2 at 0° C. and the reaction was placed in the refrigerator for 16 hours. A usual work-up afforded the mesylate as a pale yellow oil. This oil (9.2 mmol) was dissolved in THF (20 mL) and LiBr (2.0 g, 23.0 mmol) was added. The reaction mixture was heated to reflux for 1 hour and was then cooled to room temperature. To the mixture was added water (2.5 mL) and the solvent was removed under vacuum. The residue was extracted with ethyl acetate (3×15 mL) and the combined organic layers were washed with saturated NaCl solution (10 mL), dried over anhydrous Na2SO4, and concentrated to give the desired bromide.',\n",
       "   'mPEG-OH 20 kDa (2.0 g, 0.1 mmol, Sunbio) was treated with NaH (12 mg, 0.5 mmol) in THE (35 mL) and the bromide (3.32 g, 15 mmol) was added to the mixture along with a catalytic amount of KI. The resulting mixture was heated to reflux for 12 hours. Water (1.0 mL) was added to the mixture and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. Dropwise addition to an ether solution (150 mL) resulted in a precipitate, which was collected to yield mPEG-O—CH2-C6H4-N3.']},\n",
       " {'number': 'Example 30',\n",
       "  'title': '',\n",
       "  'content': ['NH2—PEGO—CH2CH2CO2H+N3—CH2CH2CO2—NHS→N3—CH2CH2—C(O)NH-PEGO—CH2CH2CO2H',\n",
       "   'NH2-PEG-O—CH2CH2CO2H (MW 3,400 Da, 2.0 g) was dissolved in a saturated aqueous solution of NaHCO3 (10 mL) and the solution was cooled to 0° C. 3-azido-1-N-hydroxysuccinimido propionate (5 equiv.) was added with vigorous stirring. After 3 hours, 20 mL of H2O was added and the mixture was stirred for an additional 45 minutes at room temperature. The pH was adjusted to 3 with 0.5 N H2SO4 and NaCl was added to a concentration of approximately 15 wt %. The reaction mixture was extracted with CH2Cl2 (100 mL×3), dried over Na2SO4 and concentrated. After precipitation with cold diethyl ether, the product was collected by filtration and dried under vacuum to yield the omega-carboxy-azide PEG derivative.']},\n",
       " {'number': 'Example 31',\n",
       "  'title': '',\n",
       "  'content': ['mPEG-OMs+HC≡CLi→mPEGO—CH2—CH2—C≡C—H',\n",
       "   'To a solution of lithium acetylide (4 equiv.), prepared as known in the art and cooled to −78° C. in THF, is added dropwise a solution of mPEG-OMs dissolved in THE with vigorous stirring. After 3 hours, the reaction is permitted to warm to room temperature and quenched with the addition of 1 mL of butanol. 20 mL of H2O is then added and the mixture was stirred for an additional 45 minutes at room temperature. The pH was adjusted to 3 with 0.5 N H2SO4 and NaCl was added to a concentration of approximately 15 wt %. The reaction mixture was extracted with CH2Cl2 (100 mL×3), dried over Na2SO4 and concentrated. After precipitation with cold diethyl ether, the product was collected by filtration and dried under vacuum to yield the 1-(but-3-ynyloxy)-methoxypolyethylene glycol (mPEG).']},\n",
       " {'number': 'Example 32',\n",
       "  'title': '',\n",
       "  'content': ['Azide- and acetylene-containing amino acids can be incorporated site-selectively into proteins using the methods described in L. Wang, et al., (2001), Science 292:498-500, J. W. Chin et al., Science 301:964-7 (2003)), J. W. Chin et al., (2002), Journal of the American Chemical Society 124:9026-9027; J. W. Chin, & P. G. Schultz, (2002), Chem Bio Chem 3(11):1135-1137; J. W. Chin, et al., (2002), PNAS United States of America 99:11020-11024: and, L. Wang, & P. G. Schultz, (2002), Chem. Comm., 1:1-11. Once the amino acids were incorporated, the cycloaddition reaction is carried out with 0.01 mM protein in phosphate buffer (PB), pH 8, in the presence of 2 mM PEG derivative, 1 mM CuSO4, and ˜1 mg Cu-wire for 4 hours at 37° C.']},\n",
       " {'number': 'Example 33',\n",
       "  'title': '',\n",
       "  'content': ['This example describes the synthesis of p-Acetyl-D,L-phenylalanine (pAF) and m-PEG-hydroxylamine derivatives.',\n",
       "   'The racemic pAF is synthesized using the previously described procedure in Zhang, Z., Smith, B. A. C., Wang, L., Brock, A., Cho, C. & Schultz, P. G., Biochemistry, (2003) 42, 6735-6746.',\n",
       "   'To synthesize the m-PEG-hydroxylamine derivative, the following procedures are completed. To a solution of (N-t-Boc-aminooxy)acetic acid (0.382 g, 2.0 mmol) and 1,3-Diisopropylcarbodiimide (0.16 mL, 1.0 mmol) in dichloromethane (DCM, 70 mL), which is stirred at room temperature (RT) for 1 hour, methoxy-polyethylene glycol amine (m-PEG-NH2, 7.5 g, 0.25 mmol, Mt. 30 K, from BioVectra) and Diisopropylethylamine (0.1 mL, 0.5 mmol) is added. The reaction is stirred at RT for 48 hours, and then is concentrated to about 100 mL. The mixture is added dropwise to cold ether (800 mL). The t-Boc-protected product precipitated out and is collected by filtering, washed by ether 3×100 mL. It is further purified by re-dissolving in DCM (100 mL) and precipitating in ether (800 mL) twice. The product is dried in vacuum yielding 7.2 g (96%), confirmed by NMR and Nihydrin test.',\n",
       "   'The deBoc of the protected product (7.0 g) obtained above is carried out in 50% TFA/DCM (40 mL) at 0° C. for 1 hour and then at RT for 1.5 hour. After removing most of TFA in vacuum, the TFA salt of the hydroxylamine derivative is converted to the HCl salt by adding 4N HCl in dioxane (1 mL) to the residue. The precipitate is dissolved in DCM (50 mL) and re-precipitated in ether (800 mL). The final product (6.8 g, 97%) is collected by filtering, washed with ether 3×100 mL, dried in vacuum, stored under nitrogen. Other PEG (5K, 20K) hydroxylamine derivatives are synthesized using the same procedure.']},\n",
       " {'number': 'Example 34',\n",
       "  'title': 'In Vivo Studies of PEGylated Relaxin',\n",
       "  'content': ['PEG-Relaxin, unmodified relaxin and buffer solution are administered to mice or rats. The results will show superior activity and prolonged half life of the PEGylated relaxin of the present invention compared to unmodified relaxin. Similarly, modified relaxin, unmodified relaxin, and buffer solution are administered to mice or rats.',\n",
       "   'A relaxin polypeptide of the invention is administered by intravenous or subcutaneous routes to mice. The animals are bled prior to and at time points after dosing. Plasma is collected from each sample and analyzed by radioimmunoassay. Elimination half-life can be calculated and compared between relaxin polypeptides comprising a non-naturally encoded amino acid and wild-type relaxin or various relaxin analog polypeptides of the invention. Similarly, relaxin polypeptides of the invention may be administered to cynomolgus monkeys. The animals are bled prior to and at time points after dosing. Plasma is collected from each sample and analyzed by radioimmunoassay.',\n",
       "   'The polypeptide may be administered the mice via multiple doses, continuous infusion, or a single dose, etc.']},\n",
       " {'number': 'Example 35',\n",
       "  'title': '',\n",
       "  'content': ['Relaxin is expressed using Novagen expression system (inducible T7 promoter; described in detail in the pET System Manual, version 9, hereby incorporated by reference), expression vector pET30a and expression strain BL21(DE3).',\n",
       "   '2 mL of of LB/Kanamycin (10 μg/ml) culture are inoculated with a sweep from BL21 (DE3) plate transformed with the desired analog. This decreases effects caused by colony to colony variability in expression levels. This culture is grown overnight at 37° C. with vigorous shaking and the following day, 10 ml LB/Kanamycin culture is inoculated with 1 ml from the overnight culture (OD600˜0.4-0.5). The remaining mL of the overnight culture may be frozen as glycerol stock.',\n",
       "   '10 mL of the grown culture is put at 37° C. and 250 rpm for 30-45 min until OD600 reaches 0.8-0.9. This is then induced with 1 mM IPTG (with 1 mL that may be set aside as non-induced culture control) and harvested usually 3-4 hours post-induction and analyzed on SDS-PAGE.',\n",
       "   'It is also possible to do a time-course of expression (e.g. points 1, 2, 4, 6 hours post-induction and O/N) to determine the rate of accumulation, protein stability, etc.',\n",
       "   'Gel Analysis: at a desired time point post-induction 1 mL is harvested from the culture, the cells are spun down, resuspended in 100 μl of 2×SDS-PAGE, sonicated to reduce viscosity and 10 μl are run on SDS-PAGE. If desired, this can be compared to non-induced control or controls and/or known positive control or standard and expression level may be estimated (e.g. good expression could be at >100 μg/ml). Western blot analysis may also be used. It is also possible to set aside 4 ml of the cultures, prepare inclusion bodies (if expressing insoluble analogs) and obtain mass spec analysis on these to confirm the identity of the over-expressed protein.',\n",
       "   'For larger scale protein expression, >250 mL of LB/Kanamycin (10 μg/ml) are inoculated with 250 μL of frozen glycerol stock and grown overnight. The following day, 10×1L LB/Kanamycin cultures are inoculated with 25 mL from the overnight culture (OD600˜0.1).',\n",
       "   '1L cultures are grown at 37° C. and 250 rpm˜2h until OD600 reaches 0.8-0.9. This is then induced with 1 mM IPTG and harvested 4 h post-induction or the following morning (harvest may use centrifugation for 15 min at 4,000 rpm). The pellets are rinsed with 50 mM Tris-HCl, pH 8.0 (50 ml per pellet+50 ml to rinse the bottle) if it is desired to reduce endotoxin and facilitate purification. Pellets are pooled together and spun again.']},\n",
       " {'number': 'Example 36',\n",
       "  'title': 'Pichia Expression Study—DNA Prep, Electroporation, Expression Protocols',\n",
       "  'content': ['This example provides a protocol for the preparation of relaxin polypeptides of the present invention in Pichia. SEQ ID NOs: 34, 35, 36, and 37 are used, and a plasmid can be used for cloning into Pichia and this or other modified plasmids may be used to obtain protein expression of relaxin polypeptides in Pichia, modifications made to the plasmid using methods known in the art.',\n",
       "   'On day 1 of the protocol, there is an overnight digestion, typically using 2U enzyme per μg DNA to be digested and 10 mL YPhyD culture is inoculated overnight in a 50 mL flask, shaking at 260 rpm at 30° C. from the glycerol stock.',\n",
       "   'DNA is precipitated by the addition first of 1/10th volume sterile 3M NaOAc and then of 0.7 volumes sterile IPA and then the sample is vigorously mixed and the precipitation is continued overnight at −20° C. or at −70° C. until frozen. The DNA is then pelleted by centrifugation (benchtop centrifuge 14,000 rpm/10 minutes), supernatant removed, and the pellet is washed using 500 μL of sterile 70% ETOH. Spin (bench-top centrifuge 14,000 rpm/10 minutes) and decant supernatant and air dry pellet for 15-20 minutes. Resuspend DNA pellet with sterile water to 1 μg/μl and transform Pichia with 10 μg DNA.',\n",
       "   'Using overnight culture with OD600, dilute in YPhyD to OD600=0.2. Shake culture at 260 rpm at 30° C. until OD600 reaches 0.8-1.0. Collect cells by centrifugation (4000 rpm/5 minutes). Decant medium, wash cells in 20 mL ice cold sterile water, decant again and repeat. After awter wash, wash pellet in 20 mL of ice-cold sterile 1 M sorbitol, decant, and resuspend washed cell pellet in 600 μL of 1 M cold sorbitol, then this may be stored on ice.',\n",
       "   'From the washed cells, mix 50 μL with 10 μg linearized DNA in sterile 1.5 mL eppendorf tube, mix gently and incubate on ice for 25 minutes. Transfer cell/DNA mixture to prechilled 0.2 cm cuvette using long pipette tips. Electroporate cells using BioRad GenePulsar II unit with the following settings: 2000 V, 200 Ohms, 25 pFd (use single pulse) and immediately add 0.5 mL YPhyD medium to the cuvette and mix by pipetting. Transfer entire contents to sterile round bottom tube and shake gently (200 rpm) for 30 minutes at 30° C. Plate and spread cells evenly and incubate plates, inverted, for 3 days at 30° C.',\n",
       "   'After three day incubation, pick colonies with a loop and inoculate 10 ml BYPhyD media in a 50 ml flask and incubate for 3 days at 30° C. Count the colonies on the 20 μl plates and record the average number and then harvest cells, first by preparing 2 sets of cryovials labeled with strain name and clone number, relaxin (i.e. protein expressed), and date. Transfer culture to 15 ml conical tube, take OD600 of each culture, dilute culture 1:50 or 1:20 in YPhyD medium. Save an alquot of culture for glycerol stock. Then pellet yeast at 4000 rpm for 5 min at RT, transfer the supernatant to a new, labeled 15 mL conical tube, and store at −20 or −80° C. until needed for analytical data.',\n",
       "   'Run samples on 4-12% NuPAGE TB gel (Novex). SDS-PAGE reagents used from Invitrogen, analyze by Western blot or Stained-gel analysis',\n",
       "   '',\n",
       "   'Yeast Extract\\n\\u200210 g/L\\n\\n\\n\\nPhytone Peptone\\n\\u200220 g/L\\n\\n\\n\\n1M potassium phosphate buffer (pH 6)\\n100 ml/L\\n\\n\\n\\n10X YNB\\n100 mL/L\\n\\n\\n\\n20% Dextrose\\n100 mL/L',\n",
       "   '',\n",
       "   'Yeast Extract\\n\\u200210 g/L\\n\\n\\n\\nPhytone Peptone\\n\\u200220 g/L\\n\\n\\n\\n20% Dextrose\\n100 ml/L\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n10×YNB (13.4% Yeast Nitrogen Base with Ammonium Sulfate without amino acids)',\n",
       "   'Yeast Nitrogen Base\\n134 g/L']},\n",
       " {'number': 'Example 37',\n",
       "  'title': 'Relaxin A21G Production',\n",
       "  'content': [\"In this example, 4.0 L culture were fermented to produce 13.4 g wet cell paste and an inclusion body preparation was performed with and without Triton-X100. 2.07 g wet inclusion bodies were produced in this manner, and solubilization and refolding followed. The inclusion bodies were resuspended with 200 mL H2O per gram of wet inclusion bodies (TBs) to a final concentration of 3 mM and cysteine is added to the resuspension. IB's are then solubilized by pH increase to 11.5 for 1 hour at RT. Refolding was then allowed to occur by dropping the pH of the solubilized material to 10.6±0.1 and stored at 2-8° C. for ˜72 hours. The refold reaction was stopped by addition of HCl to a final pH of 3.0, 0.45 μM filtered and stored at 2-8° C. until further processing.\",\n",
       "   'The refolded protein was purified by increasing the pH of quenched refold to 8.0 with Tris base and directly loading onto a Q HP column. Conductivity of load in the instance shown was >3.5 mS/cm. Run conditions were (A) 20 mM Tris, 8.0; (B) 20 mM Tris, 8.0; 200 mM NaCl and there was 0-100% B over 30CV. The correctly refolded proinsulin was pooled and 79 mg proinsulin was recovered.',\n",
       "   'Ultrafiltration/diafiltration (UF/DF) was done and precipitation was performed with 25 mM zinc, precipitated protein was resuspended to concentration of 2 mg/mL with 20 mM NaOAc, 4.0, 30% ACN, 5 mM EDTA and 20K PEG was added to a final molar ratio of 10:1 PEG to protein and allowed to incubate for 48-72 hours at 28° C.',\n",
       "   'PEG reaction was diluted 1:10 in 0.5×PEG buffer A, 0.22 μM filtered and run over an SP 650S column. The run conditions were (A) 10 mM NaOAc, 4.0, 1 mM EDTA; (B) 10 mM NaOAc, 4.0, 1 mM EDTA, 0.4M NaCl; 0-50% B over 20CV and PEG samples formulated in 10 mM NaCitrate, 6.5; 150 mM NaCl and this is shown in FIGS. 12A-B.',\n",
       "   'These methods were used to produce a variety of relaxin polypeptides with non-natural amino acids and a range of 0.1-22 mg for the end protein amounts of the purified and PEGylated variants. ACN was found to help solubilize PEG/protein mixture in PEG reaction and zinc precipitation at pI facilitated concentrating in the presence of CAN.']},\n",
       " {'number': 'Example 38',\n",
       "  'title': '',\n",
       "  'content': ['Human Clinical Trial of the Safety and/or Efficacy of PEGylated Relaxin Comprising a Non-Naturally Encoded Amino Acid.',\n",
       "   'Objective: To observe the safety and pharmacokinetics of subcutaneously administered PEGylated recombinant human relaxin comprising a non-naturally encoded amino acid.',\n",
       "   'Patients Eighteen healthy volunteers ranging between 20-40 years of age and weighing between 60-90 kg are enrolled in the study. The subjects will have no clinically significant abnormal laboratory values for hematology or serum chemistry, and a negative urine toxicology screen, HIV screen, and hepatitis B surface antigen. They should not have any evidence of the following: hypertension; a history of any primary hematologic disease; history of significant hepatic, renal, cardiovascular, gastrointestinal, genitourinary, metabolic, neurologic disease; a history of anemia or seizure disorder; a known sensitivity to bacterial or mammalian-derived products, PEG, or human serum albumin; habitual and heavy consumer to beverages containing caffeine; participation in any other clinical trial or had blood transfused or donated within 30 days of study entry; had exposure to relaxin within three months of study entry; had an illness within seven days of study entry; and have significant abnormalities on the pre-study physical examination or the clinical laboratory evaluations within 14 days of study entry. All subjects are evaluable for safety and all blood collections for pharmacokinetic analysis are collected as scheduled. All studies are performed with institutional ethics committee approval and patient consent.',\n",
       "   'Study Design: This will be a Phase I, single-center, open-label, randomized, two-period crossover study in healthy male volunteers. Eighteen subjects are randomly assigned to one of two treatment sequence groups (nine subjects/group). Relaxin is administered over two separate dosing periods as a bolus s.c. injection in the upper thigh using equivalent doses of the PEGylated relaxin comprising a non-naturally encoded amino acid and the commercially available product chosen. The dose and frequency of administration of the commercially available product is as instructed in the package label. Additional dosing, dosing frequency, or other parameter as desired, using the commercially available products may be added to the study by including additional groups of subjects. Each dosing period is separated by a 14-day washout period. Subjects are confined to the study center at least 12 hours prior to and 72 hours following dosing for each of the two dosing periods, but not between dosing periods. Additional groups of subjects may be added if there are to be additional dosing, frequency, or other parameter, to be tested for the PEGylated relaxin as well. The experimental formulation of relaxin is the PEGylated relaxin comprising a non-naturally encoded amino acid.',\n",
       "   'Blood Sampling: Serial blood is drawn by direct vein puncture before and after administration of relaxin. Venous blood samples (5 mL) for determination of serum relaxin concentrations are obtained at about 30, 20, and 10 minutes prior to dosing (3 baseline samples) and at approximately the following times after dosing: 30 minutes and at 1, 2, 5, 8, 12, 15, 18, 24, 30, 36, 48, 60 and 72 hours. Each serum sample is divided into two aliquots. All serum samples are stored at −20° C. Serum samples are shipped on dry ice. Fasting clinical laboratory tests (hematology, serum chemistry, and urinalysis) are performed immediately prior to the initial dose on day 1, the morning of day 4, immediately prior to dosing on day 16, and the morning of day 19.',\n",
       "   'Bioanalytical Methods: An ELISA kit is used for the determination of serum relaxin concentrations.',\n",
       "   'Safety Determinations: Vital signs are recorded immediately prior to each dosing (Days 1 and 16), and at 6, 24, 48, and 72 hours after each dosing. Safety determinations are based on the incidence and type of adverse events and the changes in clinical laboratory tests from baseline. In addition, changes from pre-study in vital sign measurements, including blood pressure, and physical examination results are evaluated.',\n",
       "   'Data Analysis Post-dose serum concentration values are corrected for pre-dose baseline relaxin concentrations by subtracting from each of the post-dose values the mean baseline relaxin concentration determined from averaging the relaxin levels from the three samples collected at 30, 20, and 10 minutes before dosing. Pre-dose serum relaxin concentrations are not included in the calculation of the mean value if they are below the quantification level of the assay. Pharmacokinetic parameters are determined from serum concentration data corrected for baseline relaxin concentrations. Pharmacokinetic parameters are calculated by model independent methods on a Digital Equipment Corporation VAX 8600 computer system using the latest version of the BIOAVL software. The following pharmacokinetics parameters are determined: peak serum concentration (Cmax); time to peak serum concentration (tmax); area under the concentration-time curve (AUC) from time zero to the last blood sampling time (AUC0-72) calculated with the use of the linear trapezoidal rule; and terminal elimination half-life (t1/2), computed from the elimination rate constant. The elimination rate constant is estimated by linear regression of consecutive data points in the terminal linear region of the log-linear concentration-time plot. The mean, standard deviation (SD), and coefficient of variation (CV) of the pharmacokinetic parameters are calculated for each treatment. The ratio of the parameter means (preserved formulation/non-preserved formulation) is calculated.',\n",
       "   'Safety Results: The incidence of adverse events is equally distributed across the treatment groups. There are no clinically significant changes from baseline or pre-study clinical laboratory tests or blood pressures, and no notable changes from pre-study in physical examination results and vital sign measurements. The safety profiles for the two treatment groups should appear similar.',\n",
       "   'Pharmacokinetic Results: Mean serum relaxin concentration-time profiles (uncorrected for baseline relaxin levels) in all 18 subjects after receiving PEGylated relaxin comprising a non-naturally encoded amino acid at each time point measured. All subjects should have pre-dose baseline relaxin concentrations within the normal physiologic range. Pharmacokinetic parameters are determined from serum data corrected for pre-dose mean baseline relaxin concentrations and the Cmax and tmax are determined. The mean tmax for the any clinical comparator(s) chosen is significantly shorter than the tmax for the PEGylated relaxin comprising the non-naturally encoded amino acid. Terminal half-life values are significantly shorter for the preclinical comparator(s) tested compared with the terminal half-life for the PEGylated relaxin comprising a non-naturally encoded amino acid.',\n",
       "   'Although the present study is conducted in healthy male subjects, similar absorption characteristics and safety profiles would be anticipated in other patient populations; such as male or female patients with diabetes, male or female patients with cancer or chronic renal failure, pediatric renal failure patients, patients in autologous predeposit programs, or patients scheduled for elective surgery.',\n",
       "   'In conclusion, subcutaneously administered single doses of PEGylated relaxin comprising non-naturally encoded amino acid will be safe and well tolerated by healthy male subjects. Based on a comparative incidence of adverse events, clinical laboratory values, vital signs, and physical examination results, the safety profiles of the commercially available forms of relaxin and PEGylated relaxin comprising non-naturally encoded amino acid will be equivalent. The PEGylated relaxin comprising non-naturally encoded amino acid potentially provides large clinical utility to patients and health care providers.']},\n",
       " {'number': 'Example 39',\n",
       "  'title': '',\n",
       "  'content': ['Relaxin Functional Assay Development',\n",
       "   'This example provides the details of the relaxin functional assay.',\n",
       "   \"Human peripheral blood monocytes, THP-1 cells, were used to demonstrate measureable cAMP increase alongside positive controls Isoproterenol and Forskolin. THP-1 cells were preincubated in 500 uM IBMX for 30 minutes, RLX co-stimulation with 2 uM Forskolin for 20 min. Isoproterenol, Forskolin, and relaxin polypeptides, including wild-type A (with Alanine in the 1st amino acid position of the B chain) and the variant RLX-BA1-AV13pAF (relaxin variant with the backbone amino acid sequence of the Alanine in the 1st amino acid position of the B chain with a pAF substituted for position 13 (a valine) in the A chain, with four (4) different size PEG's attached; 5K, 10K, 20K, and 30K. TABLE 5\",\n",
       "   'TABLE 5\\n\\n\\n\\n\\n\\n\\n\\nFunctional Assay Raw EC50 Values [ng/mL]\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSample\\nNov. 4, 2010\\nNov. 5, 2010\\nNov. 5, 2010\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nRLX-D-WT\\n1.5\\n1.5\\n1.0\\n\\n\\nRLX-A-WT-001\\n3.6\\n3.3\\n2.6\\n\\n\\nRLX-A-AQ1-20KPEG-001\\n\\n38\\n\\n\\n\\nRLX-A-AA5-20KPEG-001\\n\\n41\\n\\n\\n\\nRLX-A-AV13-20KPEG-001\\n\\n56\\n\\n\\n\\nRLX-A-AR18-20KPEG-001\\n\\n68\\n\\n\\n\\nRLX-A-BV7-20KPEG-001\\n54\\n\\n\\n\\n\\nRLX-A-BA18-20KPEG-001\\n172\\n\\n\\n\\n\\nRLX-A-BW28-20KPEG-001\\n172\\n\\n\\n\\n\\nRLX-A-BE5-20KPEG-001\\n\\n\\n45\\n\\n\\nRLX-D-BE5-20KPEG-001\\n\\n\\n58\\n\\n\\nRLX-D-AL2-20KPEG-001\\n\\n\\n43']},\n",
       " {'number': 'Example 40',\n",
       "  'title': '',\n",
       "  'content': ['This example evaluates the pharmacokinetic properties of 20 kDa PEGylated relaxin polypeptides following a single subcutaneous injection in SD rats.',\n",
       "   'Sprague-Dawley (SD) Rats were received from Charles River Laboratories (CRL) at approximately 7-8 weeks of age (approximately 280 g at study start). The animals were received having been jugular vein catheterized at CRL. Animals then acclimated for 3 days prior to being placed on study.',\n",
       "   'Animals received a single subcutaneous injection on day 1 and PK samples were collected over the subsequent 80 hours. Blood samples were taken from animals treated with PEG-relaxin for analysis of serum concentration according to the following sampling schedule (sampling times are approximate):',\n",
       "   'Day 1: pre-dose, 1, 2, 4, 8, 12, 25, 34, 50, 58, 73 and 80 hours post-dose',\n",
       "   'Compound concentrations were measured using a bridging ECLA based on an assay which was developed at Ambrx. Concentrations were calculated using a standard curve generated from the corresponding dosed compound and reported in an excel spreadsheet format (see appendix). Pharmacokinetic parameters were estimated using the modeling program WinNonlin (Pharsight, version 5.1). Noncompartmental analysis for individual animal data with linear-up/log-down trapezoidal integration was used, and concentration data was uniformly weighted. Compartmental analysis was performed using two compartment, 1st order elimination model and Gauss-Newton (Levenberg-Hartley) model fit equation. Table 6 shows group mean PEG-Relaxin serum concentration values versus time. FIG. 8A compares group mean serum concentration versus time for all PEG-Relaxin compounds dosed. All dose groups had measurable serum PEG-Relaxin levels.',\n",
       "   'Individual serum concentration versus time was plotted in FIG. 8B from animals dosed SC with 0.5 mg/kg PEG20K-AQ1-RLX. Individual serum concentration versus time was plotted in FIG. 9A from animals dosed SC with 0.5 mg/kg PEG20K-AA5-RLX. Individual serum concentration versus time was plotted in FIG. 9B from animals dosed SC with 0.5 mg/kg PEG20K-AR18-RLX. Individual serum concentration versus time was plotted in FIG. 10A from animals dosed SC with 0.5 mg/kg PEG20K-BV7-RLX. Individual serum concentration versus time was plotted in FIG. 10B from animals dosed SC with 0.5 mg/kg PEG20K-BW28-RLX. Individual serum concentration versus time was plotted in FIG. 11 from animals dosed SC with 0.5 mg/kg PEG20K-AV13.',\n",
       "   'Non-compartmental analysis of serum concentration versus time data from subcutaneously dosed animals is summarized in Table 6.',\n",
       "   'TABLE 6\\n\\n\\n\\n\\n\\n\\n\\nMean serum concentrations for SD rats following a single dose of PEG-Relaxin.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nDose\\n\\n\\nTime\\nMean Conc.\\nSD\\n\\n\\n\\nGroup\\nTest Article\\n(mg/kg)\\nRoute\\nGender\\n(hr)\\n(ng/mL)\\n(ng/mL)\\nN\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n1\\n30.8\\n19.4\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n2\\n87.4\\n25.2\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n4\\n184.8\\n50.2\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n8\\n237.6\\n61.9\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n12\\n371.2\\n106.1\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n25\\n394.0\\n50.1\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n34\\n278.7\\n59.1\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n50\\n63.4\\n11.9\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n58\\n45.6\\n6.8\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n73\\n20.3\\n5.6\\n5\\n\\n\\n1\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n80\\n11.6\\n1.3\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n1\\n19.9\\n6.4\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n2\\n100.1\\n51.8\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n4\\n185.0\\n104.8\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n8\\n264.7\\n128.0\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n12\\n434.3\\n135.0\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n25\\n438.0\\n55.2\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n34\\n353.5\\n44.2\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n50\\n86.9\\n19.1\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n58\\n62.4\\n10.6\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n73\\n32.8\\n8.1\\n5\\n\\n\\n2\\n20KPEG-AA5\\n0.5\\nSC\\nMale\\n80\\n22.1\\n5.4\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n1\\n33.9\\n17.3\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n2\\n109.7\\n32.9\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n4\\n172.7\\n48.2\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n8\\n270.5\\n55.3\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n12\\n332.5\\n57.7\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n25\\n398.6\\n37.6\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n34\\n264.3\\n33.8\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n50\\n76.7\\n6.9\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n58\\n61.8\\n9.0\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n73\\n25.0\\n4.8\\n5\\n\\n\\n3\\n20KPEG-AR18\\n0.5\\nSC\\nMale\\n80\\n14.9\\n3.7\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n1\\n25.7\\n4.6\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n2\\n98.9\\n20.1\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n4\\n248.5\\n75.5\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n8\\n343.5\\n81.8\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n12\\n457.3\\n91.0\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n25\\n518.5\\n57.7\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n34\\n270.4\\n64.5\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n50\\n104.0\\n14.8\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n58\\n63.5\\n8.1\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n73\\n26.0\\n3.1\\n5\\n\\n\\n4\\n20KPEG-BV7\\n0.5\\nSC\\nMale\\n80\\n22.6\\n2.6\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n1\\n36.4\\n15.4\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n2\\n107.9\\n61.5\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n4\\n228.5\\n86.0\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n8\\n380.9\\n144.0\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n12\\n486.6\\n135.4\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n25\\n511.0\\n60.3\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n34\\n404.8\\n51.2\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n50\\n184.2\\n31.3\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n58\\n122.3\\n37.2\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n73\\n48.8\\n6.1\\n5\\n\\n\\n5\\n20KPEG-BW28\\n0.5\\nSC\\nMale\\n80\\n37.1\\n5.1\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n1\\n44.9\\n16.8\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n2\\n138.9\\n60.1\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n4\\n345.7\\n117.1\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n8\\n533.6\\n157.4\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n12\\n630.1\\n201.2\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n25\\n742.5\\n117.4\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n34\\n540.7\\n31.0\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n50\\n320.9\\n21.3\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n58\\n209.4\\n22.5\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n73\\n75.0\\n4.8\\n5\\n\\n\\n6\\n20KPEG-AV13\\n0.5\\nSC\\nMale\\n80\\n58.7\\n7.7\\n5\\n\\n\\n\\n\\n\\nNE, not evaluated;\\n\\n\\nBQL, below quantifiable limit;\\n\\n\\nPD, Pre-dose',\n",
       "   'TABLE 7\\n\\n\\n\\n\\n\\n\\n\\nAQ1-\\nAA5-\\nAR18-\\nBV7-\\nBW28-\\nAV13-\\n\\n\\n\\nRLX\\nRLX\\nRLX\\nRLX\\nRLX\\nRLX\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nTerminal\\n10.7\\n12.2\\n12.5\\n13.1\\n13.9\\n14.6\\n\\n\\nHL (hr)\\n\\n\\nCmax (ng/mL)\\n394.0\\n438.0\\n345.9\\n471.6\\n511.0\\n742.5\\n\\n\\nTmax (hr)\\n25.0\\n25.0\\n25.0\\n12.0\\n25.0\\n25.0\\n\\n\\nAUCinf\\n14237.9\\n16095.9\\n12985.5\\n17260.9\\n22191.2\\n32230.5\\n\\n\\n(ng*hr/mL)\\n\\n\\nVz (mL/kg)\\n540.7\\n547.4\\n694.7\\n546.9\\n452.6\\n325.4\\n\\n\\nCL (mL/hr)\\n35.1\\n31.1\\n38.4\\n28.9\\n22.5\\n15.5\\n\\n\\nMRT (hr)\\n26.7\\n29.4\\n27.7\\n27.9\\n31.1\\n32.7\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nConcentration versus time curves were evaluated by non-compartmental analysis (Pharsight, version 4.1). N=5 rats per group. terminal HL, terminal half-life; Cmax, maximum serum concentration measured; Tmax, time at which Cmax occurred; AUCinf, area under the concentration-time curve for all serum sample/timepoints extrapolated to infinity; Cl, apparent total serum clearance; Vz, apparent volume of distribution during terminal phase.',\n",
       "   'Dose solutions were measured with the ECLIA methods used for the serum concentration measurements. Dosing solutions were diluted so as to be within the range of the assay. All 20KPEG-RLX dose solutions fell within the specified 30 percent difference from theoretical (PDT). Table 8 below summarizes the results of the dose solution analyses for this study.',\n",
       "   'TABLE 8\\n\\n\\n\\n\\n\\nPre-Dose \\nNominal\\n\\n\\n\\n\\n\\nin Buffer\\nConc.\\nDilution\\nConc.\\n\\n\\n\\n(DSA1)\\n(ng/mL)\\nFactor\\n(ng/mL)\\n% PDT\\n\\n\\n\\n\\n\\n\\n\\nRLX A-AQ1-20K PEG\\n500000\\n20000\\n495012\\n−1\\n\\n\\nRLX-A-AA5-20K PEG\\n500000\\n20000\\n474478\\n−5\\n\\n\\nRLX-A-AR18-20K PEG\\n500000\\n20000\\n432033\\n−14\\n\\n\\nRLX-A-BV7-20K PEG\\n500000\\n20000\\n377302\\n−25\\n\\n\\nRLX-A-BV7-20K PEG\\n500000\\n20000\\n475452\\n−5\\n\\n\\nRLX-A-AV13-20K PEG\\n500000\\n20000\\n571645\\n\\u2009\\u200914']},\n",
       " {'number': 'Example 41',\n",
       "  'title': '',\n",
       "  'content': ['This example evaluates the pharmacokinetic properties of wild-type (WT) Relaxin compound following a single subcutaneous injection in SD rats.',\n",
       "   'SD Rats were received from Charles River Laboratories (CRL) at approximately 5 weeks of age (approximately 280 g at study start). The animals were received having been jugular vein catheterized at CRL. Animals then acclimated for 3 days prior to being placed on study.',\n",
       "   'Animals received a single subcutaneous injection on day 1 and PK samples were collected over the subsequent 12 hours. Blood samples were taken from animals treated with WT rhRelaxin for analysis of serum concentration according to the following sampling schedule (sampling times are approximate): Day 1: pre-dose, 0.33, 0.66, 1, 1.5, 2, 3, 4, 5, 6, 9 and 12 hours post-dose.',\n",
       "   'Compound concentrations were measured using a bridging ECLA based on an assay which was developed at Ambrx. Concentrations were calculated using a standard curve generated from the corresponding dosed compound and reported in an excel spreadsheet format (see appendix). Pharmacokinetic parameters were estimated using the modeling program WinNonlin (Pharsight, version 5.1). Noncompartmental analysis for individual animal data with linear-up/log-down trapezoidal integration was used, and concentration data was uniformly weighted.',\n",
       "   'Table 9 shows group mean wt rhRelaxin serum concentration values versus time. FIG. 12A compares group mean serum concentration versus time for wt rhRelaxin. All animals had measurable serum Relaxin levels.',\n",
       "   'Individual serum concentration versus time is plotted in FIG. 12B from animals dosed SC with 0.5 mg/kg wt Relaxin. Non-compartmental analysis of serum concentration versus time data from subcutaneously dosed animals is summarized in Table 10. Table 11 is a summary of the dose solution analyses. The dosing solutions met the acceptable criteria of less than or equal to 30% PDT.',\n",
       "   'TABLE 9\\n\\n\\n\\n\\n\\n\\n\\nDose\\n\\n\\nTime\\nMean Conc.\\nSD\\n\\n\\n\\nGroup\\nTest Article\\n(mg/kg)\\nRoute\\nGender\\n(hr)\\n(ng/mL)\\n(ng/mL)\\nN\\n\\n\\n\\n\\n\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n0.33\\n244.0\\n18.7\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n0.66\\n227.9\\n45.6\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n1\\n211.7\\n45.9\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n1.5\\n166.7\\n38.7\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n2\\n119.9\\n24.5\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n3\\n52.5\\n23.0\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n4\\n24.1\\n11.4\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n5\\n7.7\\n\\u20022.6\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n6\\nBQL\\nNE\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n9\\nBQL\\nNE\\n5\\n\\n\\n1\\nwt rhRelaxin\\n0.5\\nSC\\nMale\\n12\\nBQL\\nNE\\n5\\n\\n\\n\\n\\n\\nNE, not evaluated;\\n\\n\\nBQL, below quantifiable limit;\\n\\n\\nPD, Pre-dose',\n",
       "   'TABLE 10\\n\\n\\n\\n\\n\\n\\n\\nPharmacokinetic parameter values for \\n\\n\\nwt rhRelaxin dosed in SD rats.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nwt rh Relaxin\\n\\n\\n\\n\\n\\n\\n\\nTerminal\\n\\u20020.8 (0.1)\\n\\n\\n\\nHL (hr)\\n\\n\\n\\n\\ncmax\\n258.1 (26.1)\\n\\n\\n\\n(ng/mL)\\n\\n\\n\\n\\nTmax (hr)\\n\\u20020.5 (0.2)\\n\\n\\n\\nAUCinf\\n508.9 (81.8)\\n\\n\\n\\n(ng * hr/mL)\\n\\n\\n\\n\\nVz (mL/kg)\\n1159 (284)\\n\\n\\n\\nCL (mL/hr)\\n1006 (185)\\n\\n\\n\\nMRT (hr)\\n\\u20021.56 (0.16)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nConcentration versus time curves were evaluated by non-compartmental analysis (Pharsight, version 4.1). N=5 rats per group. terminal HL, terminal half-life; Cmax, maximum serum concentration measured; Tmax, time at which Cmax occurred; AUCinf, area under the concentration-time curve for all serum sample/timepoints extrapolated to infinity; Cl, apparent total serum clearance; Vz, apparent volume of distribution during terminal phase. Numbers are mean with SD in parentheses.',\n",
       "   'TABLE 11\\n\\n\\n\\n\\n\\n\\n\\nDose solution analyses of test article.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nNominal Cone.\\nDilution\\nCone.\\n\\n\\n\\nDose Solution Analysis\\n(ng/mL)\\nFactor\\n(ng/mL)\\n% PDT\\n\\n\\n\\n\\n\\n0.5 mg/mL Pre-Dose in\\n250000\\n10000\\n208830\\n−16\\n\\n\\nformulation buffer (DSA1)\\n\\n\\n\\n\\n\\n\\n0.5 mg/mL Pre-Dose in\\n250000\\n10000\\n225898\\n−10\\n\\n\\nserum (DSA2)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nDose solutions were measured with the ECLA methods used for the serum concentration measurements. Dosing solutions were diluted so as to be within the range of the assay. All wt rhRelaxin dose solutions fell within the specified 30 percent difference from theoretical (PDT). Table 3 below summarizes the results of the dose solution analyses for this study.']},\n",
       " {'number': 'Example 42',\n",
       "  'title': '',\n",
       "  'content': ['This example evaluated the pharmacokinetic properties of a 20 kDa PEGylated Relaxin compound following a single subcutaneous or intravenous injection in SD rats.',\n",
       "   'SD Rats were received from Charles River Laboratories (CRL) at approximately 7-8 weeks of age (approximately 280 g at study start). The animals were received having been jugular vein catheterized at CRL. Animals then acclimated for 3 days prior to being placed on study. Animals received a single subcutaneous injection on day 1 and PK samples were collected over the subsequent 82 hours. Blood samples were taken from animals treated with PEG-Relaxin for analysis of serum concentration according to the following sampling schedule (sampling times are approximate): Day 1: pre-dose, 1, 3, 5, 10, 25, 34, 48, 58, 72 and 82 hours post-dose.',\n",
       "   'Compound concentrations were measured using a bridging ECLA based on an assay which was developed at Ambrx. Concentrations were calculated using a standard curve generated from the corresponding dosed compound and reported in an excel spreadsheet format (see appendix). Pharmacokinetic parameters were estimated using the modeling program WinNonlin (Pharsight, version 5.1). Noncompartmental analysis for individual animal data with linear-up/log-down trapezoidal integration was used, and concentration data was uniformly weighted. Compartmental analysis was performed using two compartment, 1 st order elimination model and Gauss-Newton (Levenberg-Hartley) model fit equation.',\n",
       "   'Table 12 shows group mean PEG-Relaxin serum concentration values versus time.',\n",
       "   'TABLE 12\\n\\n\\n\\n\\n\\n\\n\\nDose\\n\\n\\nTime\\nMean Conc.\\nSD\\n\\n\\n\\nGroup\\nTest Article\\n(mg/kg)\\nRoute\\nGender\\n(hr)\\n(ng/mL)\\n(ng/mL)\\nN\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\nPD\\nBQL\\nNE\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n1\\n2912.8\\n203.7\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n3\\n1310.8\\n115.2\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n5\\n700.7\\n68.3\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n10\\n241.6\\n27.2\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n25\\n61.2\\n5.4\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n34\\n27.0\\n2.6\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n48\\n13.7\\n2.2\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n58\\n8.0\\n2.2\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n72\\n4.3\\n1.4\\n4\\n\\n\\n1\\n20KPEG-AQ1\\n0.25\\nIV\\nMale\\n82\\n2.2\\n0.6\\n4\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n1\\n16.3\\n4.4\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n3\\n75.8\\n20.2\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n5\\n96.3\\n26.7\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n10\\n135.1\\n30.1\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n25\\n257.4\\n48.5\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n34\\n184.9\\n26.8\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n48\\n159.9\\n31.7\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n58\\n86.5\\n24.6\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n72\\n20.3\\n1.4\\n5\\n\\n\\n2\\n20KPEG-AQ1\\n0.5\\nSC\\nMale\\n82\\n11.8\\n1.1\\n5\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\nPD\\nBQL\\nNE\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n1\\n12.8\\n1.3\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n3\\n45.3\\n6.9\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n5\\n62.2\\n8.6\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n10\\n90.1\\n10.9\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n25\\n127.4\\n20.2\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n34\\n83.2\\n13.8\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n48\\n32.6\\n2.6\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n58\\n16.3\\n0.3\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n72\\n4.9\\n0.9\\n3\\n\\n\\n3\\n20KPEG-AQ1\\n0.25\\nSC\\nMale\\n82\\n2.7\\n0.4\\n3\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\nPD\\nBQL\\nNE\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n1\\n5.7\\n1.5\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n3\\n26.4\\n6.2\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n5\\n37.2\\n8.6\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n10\\n50.1\\n6.1\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n25\\n75.9\\n8.8\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n34\\n46.9\\n6.7\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n48\\n20.8\\n8.2\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n58\\n8.4\\n2.4\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n72\\n2.3\\n0.8\\n5\\n\\n\\n4\\n20KPEG-AQ1\\n0.125\\nSC\\nMale\\n82\\n1.3\\n0.4\\n5\\n\\n\\n\\n\\n\\nNE, not evaluated;\\n\\n\\nBQL, below quantifiable limit;\\n\\n\\nPD, Pre-dose']},\n",
       " {'number': 'Example 43',\n",
       "  'title': '',\n",
       "  'content': ['This example evaluated the pharmacologically active dose and systemic exposure of wild-type relaxin and a PEG-relaxin variant in female Long-Evans rats.',\n",
       "   'The objective of these signal generation studies was to establish in vivo activity and define a pharmacologically active dose of the PEG-relaxin variant. To accomplish these objectives, physiologically relevant endpoints responsive to relaxin were evaluated in female Long-Evans rats including water intake, urine output, and select urine and blood clinical chemistry. To enable identification of candidate endpoints, wild-type relaxin was tested first (Phase 1), at doses calculated to achieve plasma concentrations of 0.3×, 1× and 10× of an in vitro target inhibition. Target doses were delivered by bolus intravenous (IV) dosing followed by 6-hour continuous infusion. Doses of 0.3×, 1×, and 10× wild-type relaxin induced increases of 95%, 68%, and 32%, respectively, in water intake compared to vehicle. Changes in mean plasma sodium concentrations from baseline to 2 and 6 hours were −9.5 and 1.2 mEq/L for vehicle (0×); −5.8 and −9.0 mEq/L at 0.3×; 1.0 and −1.5 mEq/L at 1×; −4.7 and −1.4 mEq/L at 10×. The changes in plasma osmolarity from baseline to 2 and 6 hours were −20.7 and 1.0 mosmol/kg water for vehicle (0×); −11.0 and −19.1 mosmol/kg water at 0.3×; 0.3 and −4.4 mosmol/kg water for 1×; −10.3 and −3.8 mosmol/kg water for 10×.',\n",
       "   'PEG-relaxin (Phase 2) was administered at a bolus volume of 1 μL per gram of body weight with blood collected 2 and 6 hours after dosing and urine was collected for the 6 hour period following dosing. Treatment with PEG-relaxin at 0.1×, 0.3× and 1× doses resulted in 93%, 128% and 105% increases, respectively, in water intake compared to the vehicle group. Changes in mean plasma sodium concentration from baseline to 2 and 6 hours were 2.5 and 1.5 mEq/L for vehicle (0×); −1.5 and −4.8 mEq/L at 0.1×; −4.0 and −2.9 mEq/L at 0.3×; −2.3 and −4.0 mEq/L at 1×. Changes in plasma osmolarity from baseline to 2 and 6 hours were 4.5 and 2.1 mosmol/kg water for vehicle (0×); −5.1 and −11.0 mosmol/kg water at 0.1×; −8.2 and −5.0 mosmol/kg water at 0.3×; −5.6 and −9.5 mosmol/kg water at 1×. There were no clear changes in urine clinical chemistry values following treatment with wild-type or PEG-relaxin. These data establish in vivo activity and enable rationale dose selection for subsequent in vivo disease model studies with PEG-relaxin.',\n",
       "   'TABLE 13\\n\\n\\n\\n\\n\\n\\n\\nAnimal Group Design\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nPlasma\\n\\n\\n\\n\\n\\n\\n\\n\\nConcentration\\nIV Bolus Dose\\nIV Infusion\\n#\\n\\n\\n\\nTest\\nAnimal\\n(Fold Increase\\nConcentration\\nConcentration\\nAnimals\\n\\n\\nGroup\\nArticle\\nIDs\\nOver Target)\\n(mg/mL)\\n(mg/mL)c\\nper Group\\n\\n\\n\\n\\n\\n\\n\\nPhase 1 Wild-Type Relaxin Dosinga\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n1\\nWild type-\\n1F001-\\n\\u20021X\\n0.003\\n0.08\\n4\\n\\n\\n\\nrelaxin\\n1F004\\n\\n\\n2\\nWild type-\\n2F001-\\n10X\\n0.03\\u2002\\n0.8\\n4\\n\\n\\n\\nrelaxin\\n2F004\\n\\n\\n\\u20023b\\nWild type-\\n3F001-\\nvehicle\\nvehicle\\nvehicle\\n4\\n\\n\\n\\nrelaxin\\n3F004\\n\\n\\n\\u20024b\\nWild type-\\n4F001-\\n0.30X\\u2003\\n0.001\\n0.026\\n4\\n\\n\\n\\nrelaxin\\n4F004\\n\\n\\n\\n\\n\\naGroups 1 and 2 (1x and 10x) were evaluated first, and because test article-related effects on drinking water consumption, hematocrit, and urine output were apparent, doses for the second set of rats were revised to vehicle and 0.3X in Groups 3 and 4, respectively. There was a washout period of at least 36 hours between the Groups 1 and 2. A washout period of at least 36 hours also occurred between Groups 3 and 4.\\n\\n\\nbRats in Group 1 were subjected to a second randomization and assigned to Group 2 following the washout period. Rats in Group 3 were subjected to a second randomization and assigned to Group 4. Animal numbers, 001-004 remained the same in Groups 2 and 4 as they were in Groups 1 and 3; however, the group number designations, Group 2 and 4, were used to enable tube labeling and sample identification clarity.\\n\\n\\ncWild-type relaxin was injected intravenously in a bolus volume of 100 μL followed by an infusion rate of 50 μL/hour.',\n",
       "   'TABLE 14\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nIV Bolus Dose\\n\\n\\n\\n\\n\\nAnimal\\nPlasma Concentration\\nConcentration\\n# Animals\\n\\n\\nGroup\\nTest Article\\nIDs\\n(Fold Increase Over Target)\\n(mg/mL)a\\nper Group\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nPhase 2 PEG-Relaxin Dosing\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n5\\nPEG-Relaxin\\n5F001-5F004\\nVehicle\\nNA\\n4\\n\\n\\n6\\nPEG-Relaxin\\n6F001-6F004\\n0.3X\\n0.03\\n4\\n\\n\\n7\\nPEG-Relaxin\\n7F001-7F004\\n0.1X\\n0.01\\n4\\n\\n\\n8\\nPEG-Relaxin\\n8F001-8F004\\n1.0X\\n0.1\\n4\\n\\n\\n\\n\\n\\naBolus dose was administered at a volume of 1 μL/g body weight.',\n",
       "   'Female Long-Evans rats 12-14 weeks of age with bi-lateral jugular vein catheters surgically placed by the vendor were obtained from Charles River Laboratories. Two sets of four rats were used. Each set was used to evaluate two doses of wild-type relaxin. There was a washout period of 36 hours or more between Groups 1 and 2 and Groups 3 and 4 as rats from Groups 1 and 3 were reused in Groups 2 and 4, respectively. Rats had ad libitum access to food and water and were housed in the Culex ABS metabolic caging system (Culex Automated Blood Sampler, Bioanalytical System Inc) the night prior to the initiation of dosing and for the 6 hours after the IV bolus dose and initiation of infusion dosing.',\n",
       "   'In phase 1 each rat was given a one-time IV bolus and a 6 hour IV infusion. The IV bolus injection via surgically implanted jugular catheters followed by IV infusion via surgically implanted jugular catheters. At least 36 hours between the Groups 1 and 2 in the first set of rats and Groups 3 and 4 in the second set of rats was allows for washout.',\n",
       "   'Female Long-Evans rats 12-14 weeks of age with bi-lateral jugular vein catheters surgically placed by the vendor were obtained from Charles River Laboratories. Four sets of four rats were used, with each set used to evaluate one dose of PEG-Relaxin. Rats receiving PEG-Relaxin were not reused following a washout period due to the extended plasma exposure of PEGylated compounds. Rats had ad libitum access to food and water and were housed in the Culex ABS metabolic cages.',\n",
       "   'In Phase 2, the rats received a one-time IV injection via surgically implanted jugular catheters.',\n",
       "   'All bolus and IV dosing solutions were formulated according to these thawing and mixing instructions for all dose solutions:',\n",
       "   'The frozen sample bottle for each dosing formulation was allowed to thaw at 4° C. for 3 to 4 hours, with hourly inspections of the thawing process. Once thawing was complete, the formulation was mixed with a gentle inversion taking care not to create bubbles. Care was taken to gently and thoroughly mix each formulation just prior to intravenous dosing. Each dosing formulation was maintained at 4° C. until used for dosing.',\n",
       "   'The doses were prepared immediately prior to preparation and each group received dose concentrations of the following:',\n",
       "   'Phase 1 (Groups 1-4)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nGroup 1: (IX) \\nIV Bolus 0.003 mg/mL\\n\\n\\n\\n\\nIV Infusion 0.08 mg/mL\\n\\n\\n\\nGroup 2: (10X) \\nIV Bolus 0.03 mg/mL\\n\\n\\n\\n\\nIV Infusion 0.8 mg/mL\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nGroup 3: (vehicle) Bolus and infusion\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nGroup 4: (0.3X) \\nIV Bolus 0.001 mg/mL\\n\\n\\n\\n\\nIV Infusion 0.026 mg/mL\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nPhase 2 (Groups 5-8)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nGroup 5: Vehicle \\nIV Bolus NA\\n\\n\\n\\nGroup 6: (0.3X)\\nIV Bolus 0.03 mg/mL\\n\\n\\n\\nGroup 7: (0.1X) \\nIV Bolus 0.01 mg/mL\\n\\n\\n\\nGroup 8: (1.0X) \\nIV Bolus 0.1 mg/mL',\n",
       "   'Animals used on this study were selected on the basis of acceptable findings from body weight measurements, jugular catheter patency, and functionality. Animals identified with catheters unsuitable for dosing or blood collection prior to study start or during the blood collection period following initiation of dosing were removed from the study and replaced by rats with suitable catheters. Replacement of catheterized rats and compound dosing occurred at the earliest time possible relative to the dosing schedule, staffing availability, and receipt of rats from the vendor.',\n",
       "   'Phase 1: The animals were randomized according to pre-study body weight and assigned to Group 1. Following completion of dosing, rats in Group 1 were subjected to a second randomization following the washout period and reassigned to Group 2. Following completion of dosing, rats in Group 3 were subjected to a second randomization following the washout period and reassigned to Group 4.',\n",
       "   'Phase 2: The animals were randomized according to pre-study body weight and assigned to Groups 5 through 8.',\n",
       "   'Daily at approximately 30 minutes and 1, 2, 4, and 6 hours after initiation of dosing. Visual inspections of physical and behavioral changes were performed and animals were examined for changes in body posture, hair coat, activity, excreta, etc. Routine body weights were taken and recorded prior to dosing.',\n",
       "   'Phase 1 Study Protocol',\n",
       "   'Baseline urine collection were started the night prior to dose administrations, continued for a period of 15-18 hours, and were collected into a chilled (wet ice) vial. Baseline blood samples (˜400 μL whole blood) were collected at the end of baseline urine collection. On the day of the experiment, rats were given an IV bolus injection of the test article followed by continuous infusion with wild-type relaxin at a constant infusion rate delivered by a syringe pump (Harvard 11 plus) for 6 hours via left jugular vein catheter at volumes of 50 μL/hour. Two additional blood samples (˜400 μL whole blood each) were collected at 2 and 6 hours post-relaxin infusion. All blood samples were collected via right jugular catheter into sample tubes containing K3EDTA anticoagulant using Culex ABS programmed sampling method and samples were stored in a refrigerated environment until sample processing. Urine was continuously collected into a chilled vial throughout the duration of the relaxin infusion period. The total urine volume from each collection was recorded, and 2-5 mL of each urine sample was stored in a Seventh Wave Labs freezer set to maintain approximately −80°. Water intake during the 6-hour infusion period was recorded by comparing pre- and post-infusion weight of bottle+water. The infusion pump was stopped after 6 hours of continuous infusion, and rats underwent a washout period of at least 36 hours before they were subjected to the second dose evaluation.',\n",
       "   'Continuous IV infusion was required to maintain targeted, stable plasma drug concentration.',\n",
       "   'Phase 2 Study Protocol',\n",
       "   'Baseline urine collection started the night prior to dose administrations for a period of 15-18 hours, and samples were collected into a chilled vial. Baseline blood samples (˜400 μL whole blood) were collected at the end of the baseline urine collection period. On the day of the experiment, rats were administered a single dose of PEG-Relaxin intravenously at a volume not to exceed 10 mL/kg/day (one time dose). Two additional blood samples (˜400 μL whole blood) were collected at 2 and 6 hours post-relaxin administration. All blood samples were collected via right jugular catheter into sample tubes containing K3EDTA anticoagulant using Culex ABS programmed sampling method and stored in a refrigerated environment until processing. Urine samples were continuously collected from time 0 to 6 hours post administration of PEG-Relaxin into a chilled vial during sampling. The total urine volume from each collection was recorded, and 2-5 mL of each urine sample was stored in a Seventh Wave Labs freezer set to maintain approximately −80° C. Rats were euthanized 6 hours after dosing. Water intake during the 6-hour infusion period was recorded by comparing weight of the bottle+water at the beginning and the end of drug infusion period.',\n",
       "   'Urine samples were collected as already described in this example. Two to 5 mL of chilled urine samples were frozen on dry ice and stored in a freezer set to maintain approximately −80° C. until shipped on dry ice to AVL for analysis of urine creatinine and BUN for determination of creatinine clearance.',\n",
       "   'A blood sample was collected using the Culex ABS using K3EDTA as an anticoagulant. Fifteen microliters of whole blood samples were filled into untreated capillary tubes, sealed with clay on one end, and centrifuged in a hematocrit centrifuge (International Equipment Company, IEC MB centrifuge) for five minutes. Hematocrit results were obtained using a microhematocrit reading device provided by the manufacturer.',\n",
       "   'Blood was collected for potential determination of systemic exposure of wild-type and PEG-Relaxin in accordance with the collection schedule and procedures listed below. There were three collection intervals. Collection time points were 0, 2, and 6 hours post-dose for Phase 1 and Phase 2 rats. Phase 1: 8 animals per time point. Phase 2: 16 animals per time point. Collection volume was nearly 400 micro liters of whole blood. All animals in Groups 1 through 8 were bled at three time points: baseline (pre-dose t=0) and 2 and 6 hours post-initiation of infusion for wild-type relaxin IV dosing and post IV dosing for PEG-Relaxin dosing groups, respectively. The time of initiation of IV infusion dosing and the actual time of each bleed were recorded in the raw data for each animal. Per Sponsor decision, blood was not sent to Ambrx for systemic exposure determinations during the study conduct. Samples have been stored frozen at in a Seventh Wave Labs freezer set to maintain approximately −80° C. and will be returned to Sponsor.',\n",
       "   'The average water intake during the 6 hour infusion in vehicle (0×) treated rats was 1.8 mL per 100 grams of body weight, which increased to 3.6, 3.1, and 2.4 mL per 100 grams of body weight in the 0.3×, 1× and 10× wild-type relaxin treated rats, respectively. These are 95%, 68%, 32% increases over vehicle for the 0.3×, 1× and 10× dosing groups respectively.',\n",
       "   'Average hematocrit at baseline, 2 and 6 hours after initiation of dosing was 35.1%, 35.0%, and 34.3% in vehicle treated rats, respectively; 33.1%, 33.7%, and 32.3% in the 0.3× group; 31.4 T, 32.0% and 33.7% in the 1× group; 29.7%, 31.9% and 30.4% in the 10× group.',\n",
       "   'The average urine output during the 6 hour infusion was 1.8, 1.5, 1.2 and 1.2 mL per 100 grams of body weight for the vehicle (0×), 0.3×, 1× and 10× groups respectively.',\n",
       "   'The mean plasma sodium concentration at baseline, 2 and 6 hours after initiation of dosing was 139.3, 129.8, 140.5 mEq/L in vehicle (0×) group; 141.8, 136.0 and 132.8 mEq/L in the 0.3× group; 138.8, 139.8 and 137.3 mEq/L in 1× group; 137.0, 132.3 and 135.6 mEq/L in 10× group. The change from baseline at 2 and 6 hours was −9.5 and 1.2 mEq/L for vehicle (0×); −5.8 and −9.5 mEq/L at 0.3×; 1.0 and −1.5 mEq/L at 1×; −4.7 and −1.4 mEq/L at 10×.',\n",
       "   'Plasma osmolarity was calculated using the following equation: osmolarity (OSM)=(2*na)+(Glu/18)+(BUN/2.8). The mean plasma osmolarity at baseline, 2 and 6 hours after initiation of 277.3 mosmol/kg water in the 0.3× group; 291.0, 291.3 and 286.6 mosmol/kg water in the 1× group; 286.4, 276.1 and 282.6 mosmol/kg water in the 10× group. The change from baseline at 2 and 6 hours was −10.7 and 1.0 mosmol/kg water for vehicle (0×); =11.0 and =19.1 mosmol/kg awter at 0.3×; 0.3 and −4.4 mosmol/kg water for 1×; −10.3 and −3.8 mosmol/kg water for 10×. Urine clinical chemistry data of BUN/Cr, Na excretion are summarized in the tables below:',\n",
       "   'TABLE 3\\n\\n\\n\\n\\n\\n\\n\\nUrine Clinical Chemistry Data Summary of Wild-Type Relaxin (Phase 1 Study)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nNa\\n\\nBUN\\n\\nCr Cl\\n\\n\\n\\n\\nBUN/\\n\\nExcretion\\n\\nExcretion\\n\\n(mL/min/\\n\\n\\nGroups\\nCr\\nsem\\n(mEq/hr)\\nsem\\n(mg/hr)\\nsem\\n100 g)\\nsem\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nVehicle\\n(baseline)\\n36\\n12\\n0.083\\n0.010\\n13.5\\n4.4\\n0.65\\n0.04\\n\\n\\nVehicle\\n(6 hr study)\\n26\\n3\\n0.079\\n0.018\\n9.4\\n2.3\\n0.60\\n0.15\\n\\n\\n0.3X\\n(baseline)\\n66\\n9\\n0.055\\n0.005\\n17.3\\n3.9\\n0.38\\n0.06\\n\\n\\n0.3X\\n(6 hr study)\\n43\\n6\\n0.071\\n0.015\\n12.1\\n1.3\\n0.45\\n0.07\\n\\n\\n1X\\n(baseline)\\n18\\n2\\n0.046\\n0.010\\n7.6\\n1.5\\n0.62\\n0.09\\n\\n\\n1X\\n(6 hr study)\\n19\\n4\\n0.030\\n0.003\\n4.6\\n1.5\\n0.36\\n0.05\\n\\n\\n10X\\n(baseline)\\n21\\n1\\n0.063\\n0.001\\n8.5\\n1.1\\n0.69\\n0.04\\n\\n\\n10X\\n(6 hr study)\\n17\\n4\\n0.055\\n0.005\\n4.9\\n0.7\\n0.81\\n0.11',\n",
       "   'TABLE 4\\n\\n\\n\\n\\n\\n\\n\\nUrine Clinical Chemistry Raw Data of\\n\\n\\nWild-Type Relaxin (Phase 1 Study)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nCrCl\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n(ml/min/\\n\\n\\nDOSE\\nGroup\\nID\\nK\\nNa\\nCreat\\nBUN\\n100 g)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nVehicle\\n3F001 T0\\n68232\\n136\\n119\\n84\\n1080\\n0.702\\n\\n\\nVehicle\\n3F001 T6\\n68233\\n74\\n67\\n33\\n850\\n0.257\\n\\n\\nVehicle\\n3F002 T0\\n68234\\n122\\n130\\n59\\n980\\n0.692\\n\\n\\nVehicle\\n3F002 T6\\n68235\\n99\\n66\\n39\\n920\\n0.979\\n\\n\\nVehicle\\n3F003 T0\\n68236\\n276\\n130\\n54\\n2950\\n0.694\\n\\n\\nVehicle\\n3F003 T6\\n68237\\n110\\n154\\n58\\n1980\\n0.552\\n\\n\\nVehicle\\n3F004 T0\\n68238\\n355\\n126\\n52\\n3000\\n0.522\\n\\n\\nVehicle\\n3F004 T6\\n68239\\n176\\n152\\n71\\n1320\\n0.614\\n\\n\\n0.3X\\n4F001 T0\\n68791\\n272\\n47\\n22\\n1950\\n0.413\\n\\n\\n0.3X\\n4F001 T6\\n68792\\n174\\n112\\n25\\n1500\\n0.356\\n\\n\\n0.3X\\n4F002 T0\\n68793\\n274\\n68\\n42\\n1860\\n0.522\\n\\n\\n0.3X\\n4F002 T6\\n68794\\n180\\n88\\n42\\n1500\\n0.657\\n\\n\\n0.3X\\n4F003 T0\\n68795\\n230\\n54\\n25\\n1800\\n0.329\\n\\n\\n0.3X\\n4F003 T6\\n68796\\n180\\n82\\n60\\n1880\\n0.449\\n\\n\\n0.3X\\n4F004 T0\\n68797\\n225\\n68\\n23\\n1440\\n0.260\\n\\n\\n0.3X\\n4F004 T6\\n68798\\n365\\n152\\n68\\n3100\\n0.352\\n\\n\\n\\u20031X\\n1F001-T0\\n67488\\n194\\n<40\\n64\\n1480\\n0.526\\n\\n\\n\\u20031X\\n1F001-T6\\n67489\\n138\\n62\\n57\\n960\\n0.394\\n\\n\\n\\u20031X\\n1F002-T0\\n67487\\n203\\n74\\n93\\n1540\\n0.579\\n\\n\\n\\u20031X\\n1F002-T6\\n67496\\n296\\n161\\n88\\n1300\\n0.337\\n\\n\\n\\u20031X\\n1F003-T0\\n67490\\n110\\n85\\n64\\n820\\n0.495\\n\\n\\n\\u20031X\\n1F003-T6\\n67495\\n176\\n167\\n104\\n1440\\n0.236\\n\\n\\n\\u20031X\\n1F004-T0\\n67483\\n224\\n74\\n67\\n1420\\n0.882\\n\\n\\n\\u20031X\\n1F004-T6\\n67492\\n94\\n<40\\n24\\n740\\n0.476\\n\\n\\n\\u200910X\\n2F001-T0\\n67497\\n140\\n64\\n40\\n900\\n0.670\\n\\n\\n\\u200910X\\n2F001-T6\\n67498\\n54\\n68\\n64\\n500\\n0.729\\n\\n\\n\\u200910X\\n2F002-T0\\n67484\\n166\\n101\\n75\\n1120\\n0.598\\n\\n\\n\\u200910X\\n2F002-T6\\n67485\\n214\\n126\\n57\\n1260\\n0.618\\n\\n\\n\\u200910X\\n2F003-T0\\n67491\\n254\\n<40\\n103\\n1650\\n0.696\\n\\n\\n\\u200910X\\n2F003-T6\\n67486\\n60\\n<40\\n106\\n800\\n1.129\\n\\n\\n\\u200910X\\n2F004-T0\\n67494\\n294\\n88\\n74\\n1600\\n0.809\\n\\n\\n\\u200910X\\n2F004-T6\\n67493\\n212\\n125\\n85\\n1400\\n0.756\\n\\n\\n\\n\\n\\nUnits: Na (mEq/L); Creatinine (mg/dL); BUN (mg/dL), glucose (mg/dL), K (mmol/L), CrCl: mL/min/100 g BW\\n\\n\\nCreatinine Clearance (CrCl) = (U creat. * U vol)/(P creat.* U time)',\n",
       "   'TABLE 5\\n\\n\\n\\n\\n\\nPhysiology Data Summary of Wild-Type Relaxin (Phase 1 Study)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nWater Intake\\nBaseline Urine\\nUrine\\n\\n\\n\\n(0-6 hr)\\n(−16-0 hr)\\n(0-6 hr)\\n\\n\\n\\nmL/100 g BW\\nmL/100 g BW\\nmL/100 g BW\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nDose\\nmean\\nsem\\nmean\\nsem\\nmean\\nsem\\n\\n\\n\\n\\n\\nVehicle\\n1.8\\n0.4\\n4.1\\n0.4\\n1.8\\n0.3\\n\\n\\n0.3X\\n3.6\\n0.2\\n5.7\\n1.1\\n1.5\\n0.4\\n\\n\\n\\u20031X\\n3.1\\n0.6\\n3.8\\n0.6\\n1.2\\n0.6\\n\\n\\n\\u200910X\\n2.4\\n0.7\\n4.3\\n0.8\\n1.2\\n0.1\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nHematocrit (%)\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n0 hr\\n\\n2 hr\\n\\n6 hr\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nDose\\nmean\\nsem\\nmean\\nsem\\nmean\\nsem\\n\\n\\n\\n\\n\\nVehicle\\n35.1\\n1.0\\n35.0\\n0.9\\n34.3\\n0.5\\n\\n\\n0.3X\\n29.7\\n1.1\\n31.9\\n1.2\\n30.4\\n0.4\\n\\n\\n\\u20031X\\n31.4\\n2.7\\n32.0\\n2.9\\n33.7\\n2.1\\n\\n\\n\\u200910X\\n29.7\\n1.1\\n31.9\\n1.2\\n30.4\\n0.4',\n",
       "   'The average water intake during the 6 hour post-dose period in vehicle (0×) treated rats was 1.5 mL per 100 grams of body weight, which increased to 2.9, 3.4, and 3.1 mL per 100 grams of body weight in 0.1×, 0.3× and 1×PEG-relaxin variant treated rats, respectively. These represent 93%, 128% and 105% increases over the mean of vehicle groups for the 0.1×, 0.3× and 1× groups respectively.',\n",
       "   'The average urine output during the 6 hour post-dose period was 0.9, 0.2, 0.5 and 0.4 mL per 100 grams of body weight for vehicle (0×, 0.3× and 1× groups respectively). The mean plasma sodium concentrations at baseline, 2 and 6 hours after dosing were 136.3, 138.8 and 137.8 mEq/L in the vehicle (0×) group, respectively; 138.5, 137.0 and 133.7 mEq/L in the 0.1× group, respectively; 137.5, 133.5 and 134.6 mEq/L in the 0.3× group, respectively; 137.3, 135.0 and 133.3 mEq/L in the 1× group, respectively. The change from baseline at 2 and 6 hours was 2.5 and 1.5 mEq/L for vehicle (0×); −1.5 and −4.8 mEq/L at 0.1×; −4.0 and −2.9 mEq/L at 0.3×; −2.3 and −4.0 mEq/L at 1×.',\n",
       "   'The mean plasma osmolarity at baseline, 2 and 6 hours after dosing was 285.4, 289.9, and 287.5 mosmol/kg water in the vehicle (0×) group, respectively; 292.7, 287.6 and 281.7 mosmol/kg water in the 0.1× group, respectively; 287.6, 279.4 and 282.6 mosmol/kg water in the 0.3× group, respectively; 289.3, 283.7 and 279.4 mosmol/kg water in the 1× group, respectively. The change from baseline at 2 and 6 hours was 4.5 and 2.1 mosmol/kg water for vehicle (0×); −5.1 and −11.0 mosmol/kg water at 0.1×; −8.2 and −5.0 mosmol/kg water for 0.3×; −5.6 and −9.9 mosmol kg water for 1×.',\n",
       "   'It is understood that the examples and embodiments described herein are for illustrative purposes only and that various modifications or changes in light thereof will be suggested to those of ordinary skill in the art and are to be included within the spirit and purview of this application and scope of the appended claims. All publications, patents, patent applications, and/or other documents cited in this application are incorporated by reference in their entirety for all purposes to the same extent as if each individual publication, patent, patent application, and/or other document were individually indicated to be incorporated by reference for all purposes.']}]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = extract_experiments_w_heading(xml_parts[28760])\n",
    "process_siblings(a[0].find_next_siblings())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "object of type 'NoneType' has no len()",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "Input \u001b[1;32mIn [6]\u001b[0m, in \u001b[0;36m<cell line: 3>\u001b[1;34m()\u001b[0m\n\u001b[0;32m     11\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m example \u001b[38;5;129;01min\u001b[39;00m heading:\n\u001b[0;32m     12\u001b[0m     extracted_ex_start_w_word \u001b[38;5;241m=\u001b[39m process_siblings(example\u001b[38;5;241m.\u001b[39mfind_next_siblings())\n\u001b[1;32m---> 13\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28;43mlen\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mextracted_ex_start_w_word\u001b[49m\u001b[43m)\u001b[49m \u001b[38;5;241m>\u001b[39m \u001b[38;5;241m0\u001b[39m:\n\u001b[0;32m     14\u001b[0m         extracted_examples\u001b[38;5;241m.\u001b[39mappend(extracted_ex_start_w_word)\n\u001b[0;32m     15\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "\u001b[1;31mTypeError\u001b[0m: object of type 'NoneType' has no len()"
     ]
    }
   ],
   "source": [
    "\n",
    "doc_w_exp = {}\n",
    "i = 0\n",
    "for xml in xml_no_dup:\n",
    "    i += 1\n",
    "    if i % 1000 == 0:\n",
    "        print(f\"{i}/{len(xml_no_dup)} so far found {len(doc_w_exp)} docs with experiments\")\n",
    "    heading = extract_experiments_w_heading(xml)\n",
    "    if heading:\n",
    "        extracted_examples = []\n",
    "        if len(heading) >1:\n",
    "            for example in heading:\n",
    "                extracted_ex_start_w_word = process_siblings(example.find_next_siblings())\n",
    "                if extracted_ex_start_w_word and len(extracted_ex_start_w_word) > 0:\n",
    "                    extracted_examples.append(extracted_ex_start_w_word)\n",
    "                else:\n",
    "                    extracted_ex_w_word = extract_examples_start_w_word(example.find_next_siblings())\n",
    "                    if extracted_ex_w_word:\n",
    "                        if len(extracted_ex_w_word) > 0:\n",
    "                            extracted_examples.append(extracted_ex_w_word)\n",
    "                    else:\n",
    "                        num_dot_examples = extract_num_dot_examples(str(heading[0].find_next_siblings()))\n",
    "                        if len(num_dot_examples) > 0:\n",
    "                            extracted_examples.append(num_dot_examples)\n",
    "        elif len(heading) == 1:\n",
    "            example_start_w = process_siblings(heading[0].find_next_siblings())\n",
    "            if example_start_w:\n",
    "                if len(example_start_w[0][\"content\"]) == 0:\n",
    "                    extracted_ex_w_word = extract_examples_start_w_word(heading[0].find_next_siblings())\n",
    "                    if len(extracted_ex_w_word) > 0:\n",
    "                        extracted_examples.append(extracted_ex_w_word)\n",
    "                else:\n",
    "                    extracted_examples.append(example_start_w)\n",
    "            else:\n",
    "                extracted_ex_w_word = extract_examples_start_w_word(heading[0].find_next_siblings())\n",
    "                if extracted_ex_w_word:\n",
    "                    if len(extracted_ex_w_word) > 0:\n",
    "                        extracted_examples.append(extracted_ex_w_word)\n",
    "                else:\n",
    "                    num_dot_examples = extract_num_dot_examples(str(heading[0].find_next_siblings()))\n",
    "                    if len(num_dot_examples) > 0:\n",
    "                        extracted_examples.append(num_dot_examples)\n",
    "        else:\n",
    "            extracted_ex_w_word = extract_examples_w_word(xml)\n",
    "            if extracted_ex_w_word:\n",
    "                extracted_examples.append(extracted_ex_w_word)\n",
    "            # else:\n",
    "            #     num_dot_examples = extract_num_dot_examples(str(heading[0].find_next_siblings()))\n",
    "            #     extracted_examples.append(num_dot_examples)\n",
    "    \n",
    "        \n",
    "    \n",
    "        if len(extracted_examples) > 0:\n",
    "            doc_w_exp[int(find_doc_number(xml)[0])] = extracted_examples\n",
    "            # if len(doc_w_exp) == 100:\n",
    "            #     break\n",
    "\n",
    "\n",
    "\"\"\"100/8048 so far found 17 docs with experiments\n",
    "200/8048 so far found 30 docs with experiments\n",
    "300/8048 so far found 33 docs with experiments\n",
    "400/8048 so far found 38 docs with experiments\n",
    "500/8048 so far found 42 docs with experiments\n",
    "600/8048 so far found 45 docs with experiments\n",
    "700/8048 so far found 46 docs with experiments\n",
    "800/8048 so far found 90 docs with experiments\n",
    "900/8048 so far found 146 docs with experiments\n",
    "1000/8048 so far found 187 docs with experiments\n",
    "1100/8048 so far found 233 docs with experiments\n",
    "1200/8048 so far found 251 docs with experiments\n",
    "1300/8048 so far found 258 docs with experiments\n",
    "1400/8048 so far found 263 docs with experiments\n",
    "1500/8048 so far found 294 docs with experiments\n",
    "1600/8048 so far found 298 docs with experiments\n",
    "1700/8048 so far found 302 docs with experiments\n",
    "1800/8048 so far found 320 docs with experiments\n",
    "1900/8048 so far found 326 docs with experiments\n",
    "2000/8048 so far found 329 docs with experiments\n",
    "2100/8048 so far found 331 docs with experiments\n",
    "2200/8048 so far found 332 docs with experiments\n",
    "2300/8048 so far found 334 docs with experiments\n",
    "2400/8048 so far found 361 docs with experiments\n",
    "2500/8048 so far found 435 docs with experiments\n",
    "with dup: 1196   out of 8048 in 12 minutes\"\"\"\n",
    "\n",
    "\"\"\"100/7824 so far found 20 docs with experiments\n",
    "200/7824 so far found 34 docs with experiments\n",
    "300/7824 so far found 36 docs with experiments\n",
    "400/7824 so far found 42 docs with experiments\n",
    "500/7824 so far found 45 docs with experiments\n",
    "600/7824 so far found 49 docs with experiments\n",
    "700/7824 so far found 51 docs with experiments\n",
    "800/7824 so far found 98 docs with experiments\n",
    "900/7824 so far found 162 docs with experiments\n",
    "1000/7824 so far found 217 docs with experiments\n",
    "1100/7824 so far found 253 docs with experiments\n",
    "1200/7824 so far found 261 docs with experiments\n",
    "1300/7824 so far found 266 docs with experiments\n",
    "1400/7824 so far found 295 docs with experiments\n",
    "1500/7824 so far found 304 docs with experiments\n",
    "1600/7824 so far found 309 docs with experiments\n",
    "1700/7824 so far found 328 docs with experiments\n",
    "1800/7824 so far found 333 docs with experiments\n",
    "1900/7824 so far found 339 docs with experiments\n",
    "2000/7824 so far found 342 docs with experiments\n",
    "2100/7824 so far found 343 docs with experiments\n",
    "2200/7824 so far found 345 docs with experiments\n",
    "2300/7824 so far found 358 docs with experiments\n",
    "2400/7824 so far found 439 docs with experiments\n",
    "2500/7824 so far found 510 docs with experiments\n",
    "no dup 949  out of 7824 in 13.5 minutes on own laptop\n",
    "\n",
    "\n",
    "4537/46862 in 39min on uni pc \n",
    "2432 /30085 with examples etc being the only thing in the heading when fiding examples section\n",
    "2370 /29405  without duplicates with extract_examples_start_w_word first \"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "extract_examples_start_w_word(example.find_next_siblings())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('\\n'\n",
      " '<!DOCTYPE us-patent-application SYSTEM '\n",
      " '\"us-patent-application-v46-2022-02-17.dtd\" [ ]>\\n'\n",
      " '<us-patent-application lang=\"EN\" dtd-version=\"v4.6 2022-02-17\" '\n",
      " 'file=\"US20230000035A1-20230105.XML\" status=\"PRODUCTION\" '\n",
      " 'id=\"us-patent-application\" country=\"US\" date-produced=\"20221220\" '\n",
      " 'date-publ=\"20230105\">\\n'\n",
      " '<us-bibliographic-data-application lang=\"EN\" country=\"US\">\\n'\n",
      " '<publication-reference>\\n'\n",
      " '<document-id>\\n'\n",
      " '<country>US</country>\\n'\n",
      " '<doc-number>20230000035</doc-number>\\n'\n",
      " '<kind>A1</kind>\\n'\n",
      " '<date>20230105</date>\\n'\n",
      " '</document-id>\\n'\n",
      " '</publication-reference>\\n'\n",
      " '<application-reference appl-type=\"utility\">\\n'\n",
      " '<document-id>\\n'\n",
      " '<country>US</country>\\n'\n",
      " '<doc-number>17849401</doc-number>\\n'\n",
      " '<date>20220624</date>\\n'\n",
      " '</document-id>\\n'\n",
      " '</application-reference>\\n'\n",
      " '<us-application-series-code>17</us-application-series-code>\\n'\n",
      " '<classifications-ipcr>\\n'\n",
      " '<classification-ipcr>\\n'\n",
      " '<ipc-version-indicator><date>20060101</date></ipc-version-indicator>\\n'\n",
      " '<classification-level>A</classification-level>\\n'\n",
      " '<section>A</section>\\n'\n",
      " '<class>01</class>\\n'\n",
      " '<subclass>H</subclass>\\n'\n",
      " '<main-group>6</main-group>\\n'\n",
      " '<subgroup>34</subgroup>\\n'\n",
      " '<symbol-position>F</symbol-position>\\n'\n",
      " '<classification-value>I</classification-value>\\n'\n",
      " '<action-date><date>20230105</date></action-date>\\n'\n",
      " '<generating-office><country>US</country></generating-office>\\n'\n",
      " '<classification-status>B</classification-status>\\n'\n",
      " '<classification-data-source>H</classification-data-source>\\n'\n",
      " '</classification-ipcr>\\n'\n",
      " '<classification-ipcr>\\n'\n",
      " '<ipc-version-indicator><date>20060101</date></ipc-version-indicator>\\n'\n",
      " '<classification-level>A</classification-level>\\n'\n",
      " '<section>A</section>\\n'\n",
      " '<class>01</class>\\n'\n",
      " '<subclass>H</subclass>\\n'\n",
      " '<main-group>5</main-group>\\n'\n",
      " '<subgroup>08</subgroup>\\n'\n",
      " '<symbol-position>L</symbol-position>\\n'\n",
      " '<classification-value>I</classification-value>\\n'\n",
      " '<action-date><date>20230105</date></action-date>\\n'\n",
      " '<generating-office><country>US</country></generating-office>\\n'\n",
      " '<classification-status>B</classification-status>\\n'\n",
      " '<classification-data-source>H</classification-data-source>\\n'\n",
      " '</classification-ipcr>\\n'\n",
      " '<classification-ipcr>\\n'\n",
      " '<ipc-version-indicator><date>20060101</date></ipc-version-indicator>\\n'\n",
      " '<classification-level>A</classification-level>\\n'\n",
      " '<section>A</section>\\n'\n",
      " '<class>01</class>\\n'\n",
      " '<subclass>H</subclass>\\n'\n",
      " '<main-group>1</main-group>\\n'\n",
      " '<subgroup>04</subgroup>\\n'\n",
      " '<symbol-position>L</symbol-position>\\n'\n",
      " '<classification-value>I</classification-value>\\n'\n",
      " '<action-date><date>20230105</date></action-date>\\n'\n",
      " '<generating-office><country>US</country></generating-office>\\n'\n",
      " '<classification-status>B</classification-status>\\n'\n",
      " '<classification-data-source>H</classification-data-source>\\n'\n",
      " '</classification-ipcr>\\n'\n",
      " '</classifications-ipcr>\\n'\n",
      " '<classifications-cpc>\\n'\n",
      " '<main-cpc>\\n'\n",
      " '<classification-cpc>\\n'\n",
      " '<cpc-version-indicator><date>20180501</date></cpc-version-indicator>\\n'\n",
      " '<section>A</section>\\n'\n",
      " '<class>01</class>\\n'\n",
      " '<subclass>H</subclass>\\n'\n",
      " '<main-group>6</main-group>\\n'\n",
      " '<subgroup>342</subgroup>\\n'\n",
      " '<symbol-position>F</symbol-position>\\n'\n",
      " '<classification-value>I</classification-value>\\n'\n",
      " '<action-date><date>20230105</date></action-date>\\n'\n",
      " '<generating-office><country>US</country></generating-office>\\n'\n",
      " '<classification-status>B</classification-status>\\n'\n",
      " '<classification-data-source>H</classification-data-source>\\n'\n",
      " '<scheme-origination-code>C</scheme-origination-code>\\n'\n",
      " '</classification-cpc>\\n'\n",
      " '</main-cpc>\\n'\n",
      " '<further-cpc>\\n'\n",
      " '<classification-cpc>\\n'\n",
      " '<cpc-version-indicator><date>20130101</date></cpc-version-indicator>\\n'\n",
      " '<section>A</section>\\n'\n",
      " '<class>01</class>\\n'\n",
      " '<subclass>H</subclass>\\n'\n",
      " '<main-group>5</main-group>\\n'\n",
      " '<subgroup>08</subgroup>\\n'\n",
      " '<symbol-position>L</symbol-position>\\n'\n",
      " '<classification-value>I</classification-value>\\n'\n",
      " '<action-date><date>20230105</date></action-date>\\n'\n",
      " '<generating-office><country>US</country></generating-office>\\n'\n",
      " '<classification-status>B</classification-status>\\n'\n",
      " '<classification-data-source>H</classification-data-source>\\n'\n",
      " '<scheme-origination-code>C</scheme-origination-code>\\n'\n",
      " '</classification-cpc>\\n'\n",
      " '<classification-cpc>\\n'\n",
      " '<cpc-version-indicator><date>20130101</date></cpc-version-indicator>\\n'\n",
      " '<section>A</section>\\n'\n",
      " '<class>01</class>\\n'\n",
      " '<subclass>H</subclass>\\n'\n",
      " '<main-group>1</main-group>\\n'\n",
      " '<subgroup>04</subgroup>\\n'\n",
      " '<symbol-position>L</symbol-position>\\n'\n",
      " '<classification-value>I</classification-value>\\n'\n",
      " '<action-date><date>20230105</date></action-date>\\n'\n",
      " '<generating-office><country>US</country></generating-office>\\n'\n",
      " '<classification-status>B</classification-status>\\n'\n",
      " '<classification-data-source>H</classification-data-source>\\n'\n",
      " '<scheme-origination-code>C</scheme-origination-code>\\n'\n",
      " '</classification-cpc>\\n'\n",
      " '</further-cpc>\\n'\n",
      " '</classifications-cpc>\\n'\n",
      " '<invention-title id=\"d2e43\">SPOTTED WATERMELON PLANTS</invention-title>\\n'\n",
      " '<us-related-documents>\\n'\n",
      " '<us-provisional-application>\\n'\n",
      " '<document-id>\\n'\n",
      " '<country>US</country>\\n'\n",
      " '<doc-number>63214697</doc-number>\\n'\n",
      " '<date>20210624</date>\\n'\n",
      " '</document-id>\\n'\n",
      " '</us-provisional-application>\\n'\n",
      " '</us-related-documents>\\n'\n",
      " '<us-parties>\\n'\n",
      " '<us-applicants>\\n'\n",
      " '<us-applicant sequence=\"00\" app-type=\"applicant\" designation=\"us-only\" '\n",
      " 'applicant-authority-category=\"assignee\">\\n'\n",
      " '<addressbook>\\n'\n",
      " '<orgname>Enza Zaden Beheer B.V.</orgname>\\n'\n",
      " '<address>\\n'\n",
      " '<city>Enkhuizen</city>\\n'\n",
      " '<country>NL</country>\\n'\n",
      " '</address>\\n'\n",
      " '</addressbook>\\n'\n",
      " '<residence>\\n'\n",
      " '<country>NL</country>\\n'\n",
      " '</residence>\\n'\n",
      " '</us-applicant>\\n'\n",
      " '</us-applicants>\\n'\n",
      " '<inventors>\\n'\n",
      " '<inventor sequence=\"00\" designation=\"us-only\">\\n'\n",
      " '<addressbook>\\n'\n",
      " '<last-name>BROWN</last-name>\\n'\n",
      " '<first-name>Laura Barham</first-name>\\n'\n",
      " '<address>\\n'\n",
      " '<city>Salinas</city>\\n'\n",
      " '<state>CA</state>\\n'\n",
      " '<country>US</country>\\n'\n",
      " '</address>\\n'\n",
      " '</addressbook>\\n'\n",
      " '</inventor>\\n'\n",
      " '<inventor sequence=\"01\" designation=\"us-only\">\\n'\n",
      " '<addressbook>\\n'\n",
      " '<last-name>BARHAM</last-name>\\n'\n",
      " '<first-name>Robert</first-name>\\n'\n",
      " '<address>\\n'\n",
      " '<city>Salinas</city>\\n'\n",
      " '<state>CA</state>\\n'\n",
      " '<country>US</country>\\n'\n",
      " '</address>\\n'\n",
      " '</addressbook>\\n'\n",
      " '</inventor>\\n'\n",
      " '</inventors>\\n'\n",
      " '</us-parties>\\n'\n",
      " '<assignees>\\n'\n",
      " '<assignee>\\n'\n",
      " '<addressbook>\\n'\n",
      " '<orgname>Enza Zaden Beheer B.V.</orgname>\\n'\n",
      " '<role>03</role>\\n'\n",
      " '<address>\\n'\n",
      " '<city>Enkhuizen</city>\\n'\n",
      " '<country>NL</country>\\n'\n",
      " '</address>\\n'\n",
      " '</addressbook>\\n'\n",
      " '</assignee>\\n'\n",
      " '</assignees>\\n'\n",
      " '</us-bibliographic-data-application>\\n'\n",
      " '<abstract id=\"abstract\">\\n'\n",
      " '<p id=\"p-0001\" num=\"0000\">The present invention relates to methods of '\n",
      " 'breeding with spotted watermelon, <i>Citrullus lanatus</i>, lines and '\n",
      " 'varieties. More specifically, methods of making homozygous-spotted diploid '\n",
      " 'watermelon lines (HSDL) for use in breeding and methods of making '\n",
      " 'homozygous-spotted tetraploid watermelon lines (HSDL) for use in breeding '\n",
      " 'are described. Further, methods of producing 33%, 66%, or 100% spotted '\n",
      " 'triploid hybrid watermelon plants, as well as methods of producing 50% or '\n",
      " '100% spotted diploid hybrid watermelon plants, are described. This invention '\n",
      " 'further relates to markers able to determine the percentage of spotted '\n",
      " 'allele present in spotted watermelon lines and varieties. In addition, this '\n",
      " 'invention relates to new and distinctive lines and varieties of watermelon '\n",
      " 'with a spotted phenotype.</p>\\n'\n",
      " '</abstract>\\n'\n",
      " '<drawings id=\"DRAWINGS\">\\n'\n",
      " '<figure id=\"Fig-EMI-D00000\" num=\"00000\">\\n'\n",
      " '<img id=\"EMI-D00000\" he=\"64.26mm\" wi=\"158.75mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00000.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00001\" num=\"00001\">\\n'\n",
      " '<img id=\"EMI-D00001\" he=\"191.77mm\" wi=\"158.07mm\" orientation=\"landscape\" '\n",
      " 'file=\"US20230000035A1-20230105-D00001.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00002\" num=\"00002\">\\n'\n",
      " '<img id=\"EMI-D00002\" he=\"191.69mm\" wi=\"157.73mm\" orientation=\"landscape\" '\n",
      " 'file=\"US20230000035A1-20230105-D00002.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00003\" num=\"00003\">\\n'\n",
      " '<img id=\"EMI-D00003\" he=\"194.65mm\" wi=\"153.75mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00003.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00004\" num=\"00004\">\\n'\n",
      " '<img id=\"EMI-D00004\" he=\"208.28mm\" wi=\"153.75mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00004.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00005\" num=\"00005\">\\n'\n",
      " '<img id=\"EMI-D00005\" he=\"194.65mm\" wi=\"163.07mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00005.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00006\" num=\"00006\">\\n'\n",
      " '<img id=\"EMI-D00006\" he=\"194.65mm\" wi=\"131.74mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00006.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00007\" num=\"00007\">\\n'\n",
      " '<img id=\"EMI-D00007\" he=\"194.65mm\" wi=\"126.15mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00007.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00008\" num=\"00008\">\\n'\n",
      " '<img id=\"EMI-D00008\" he=\"194.65mm\" wi=\"163.91mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00008.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00009\" num=\"00009\">\\n'\n",
      " '<img id=\"EMI-D00009\" he=\"203.54mm\" wi=\"159.51mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00009.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00010\" num=\"00010\">\\n'\n",
      " '<img id=\"EMI-D00010\" he=\"206.33mm\" wi=\"147.83mm\" orientation=\"landscape\" '\n",
      " 'file=\"US20230000035A1-20230105-D00010.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00011\" num=\"00011\">\\n'\n",
      " '<img id=\"EMI-D00011\" he=\"192.53mm\" wi=\"126.15mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00011.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00012\" num=\"00012\">\\n'\n",
      " '<img id=\"EMI-D00012\" he=\"192.53mm\" wi=\"142.16mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00012.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00013\" num=\"00013\">\\n'\n",
      " '<img id=\"EMI-D00013\" he=\"192.53mm\" wi=\"137.08mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00013.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00014\" num=\"00014\">\\n'\n",
      " '<img id=\"EMI-D00014\" he=\"192.53mm\" wi=\"153.59mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00014.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00015\" num=\"00015\">\\n'\n",
      " '<img id=\"EMI-D00015\" he=\"180.42mm\" wi=\"141.05mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00015.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00016\" num=\"00016\">\\n'\n",
      " '<img id=\"EMI-D00016\" he=\"188.47mm\" wi=\"177.04mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00016.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00017\" num=\"00017\">\\n'\n",
      " '<img id=\"EMI-D00017\" he=\"191.26mm\" wi=\"140.89mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00017.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00018\" num=\"00018\">\\n'\n",
      " '<img id=\"EMI-D00018\" he=\"191.26mm\" wi=\"129.54mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00018.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00019\" num=\"00019\">\\n'\n",
      " '<img id=\"EMI-D00019\" he=\"239.95mm\" wi=\"134.45mm\" orientation=\"landscape\" '\n",
      " 'file=\"US20230000035A1-20230105-D00019.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00020\" num=\"00020\">\\n'\n",
      " '<img id=\"EMI-D00020\" he=\"186.01mm\" wi=\"155.79mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00020.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00021\" num=\"00021\">\\n'\n",
      " '<img id=\"EMI-D00021\" he=\"210.90mm\" wi=\"144.61mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00021.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00022\" num=\"00022\">\\n'\n",
      " '<img id=\"EMI-D00022\" he=\"212.01mm\" wi=\"146.30mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00022.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00023\" num=\"00023\">\\n'\n",
      " '<img id=\"EMI-D00023\" he=\"212.01mm\" wi=\"146.30mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00023.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '<figure id=\"Fig-EMI-D00024\" num=\"00024\">\\n'\n",
      " '<img id=\"EMI-D00024\" he=\"212.01mm\" wi=\"146.30mm\" '\n",
      " 'file=\"US20230000035A1-20230105-D00024.TIF\" alt=\"embedded image\" '\n",
      " 'img-content=\"drawing\" img-format=\"tif\"/>\\n'\n",
      " '</figure>\\n'\n",
      " '</drawings>\\n'\n",
      " '<description id=\"description\">\\n'\n",
      " '<?cross-reference-to-related-applications description=\"Cross Reference To '\n",
      " 'Related Applications\" end=\"lead\"?>\\n'\n",
      " '<heading id=\"h-0001\" level=\"1\">CROSS-REFERENCE TO RELATED '\n",
      " 'APPLICATIONS</heading>\\n'\n",
      " '<p id=\"p-0002\" num=\"0001\">This application claims the benefit of U.S. '\n",
      " 'Provisional Application No. 63/214,697, filed Jun. 24, 2021, which is hereby '\n",
      " 'incorporated by reference in its entirety.</p>\\n'\n",
      " '<?cross-reference-to-related-applications description=\"Cross Reference To '\n",
      " 'Related Applications\" end=\"tail\"?>\\n'\n",
      " '<?summary-of-invention description=\"Summary of Invention\" end=\"lead\"?>\\n'\n",
      " '<heading id=\"h-0002\" level=\"1\">SUBMISSION OF SEQUENCE LISTING AS ASCII TEXT '\n",
      " 'FILE</heading>\\n'\n",
      " '<p id=\"p-0003\" num=\"0002\">The content of the following submission on ASCII '\n",
      " 'text file is incorporated herein by reference in its entirety: a computer '\n",
      " 'readable form (CRF) of the Sequence Listing (file name: '\n",
      " '70180-2019400SEQLIST.TXT, date recorded: Jun. 23, 2022, size: 3,024 '\n",
      " 'bytes).</p>\\n'\n",
      " '<heading id=\"h-0003\" level=\"1\">FIELD OF THE INVENTION</heading>\\n'\n",
      " '<p id=\"p-0004\" num=\"0003\">The present invention relates to the field of '\n",
      " 'plant breeding. In particular, this invention relates to methods of breeding '\n",
      " 'with spotted watermelon, <i>Citrullus lanatus</i>, lines and varieties. This '\n",
      " 'invention further relates to markers able to determine the percentage of '\n",
      " 'spotted allele present in spotted watermelon lines and varieties. In '\n",
      " 'addition, this invention relates to new and distinctive lines and varieties '\n",
      " 'of watermelon with a spotted phenotype.</p>\\n'\n",
      " '<heading id=\"h-0004\" level=\"1\">BACKGROUND OF THE INVENTION</heading>\\n'\n",
      " '<p id=\"p-0005\" num=\"0004\">All cultivated forms of watermelon belong to the '\n",
      " 'polymorphic species <i>Citrullus lanatus</i>. As a crop, watermelons are '\n",
      " 'grown commercially wherever environmental conditions permit the production '\n",
      " 'of an economically viable yield. Cultivated watermelons grow as annual '\n",
      " 'plants with large, broad leaves. The leaves may be lobed or simple and are '\n",
      " 'typically orbicular to triangular-ovate in shape. The flowers are '\n",
      " 'monoecious, producing both male and female flowers. The flesh color of '\n",
      " 'cultivated watermelons is red, yellow, or white with ovate to oblong '\n",
      " 'strongly compressed seeds that may be brown or white in color. The '\n",
      " 'characterization of the mature fruit can vary widely among varieties of '\n",
      " 'watermelon. Fruits may be round to oblong or elliptical with rind colors '\n",
      " 'varying from dark green to yellow and rind patterning varying widely.</p>\\n'\n",
      " '<p id=\"p-0006\" num=\"0005\">Many changes that occurred with the domestication '\n",
      " 'of the watermelon relate to fruit morphology, with a specialization in fruit '\n",
      " 'shape, size, and flavor. Cultivated watermelons can vary from 5 to 45 '\n",
      " 'pounds, depending on the variety or line. In the United States, watermelon '\n",
      " 'is grown in at least 44 states, but the principal watermelon growing states '\n",
      " 'are Georgia, Florida, Texas, and California. Asia is the largest producer of '\n",
      " 'watermelon, with 83.4% of the world production in 2011. Fresh watermelons '\n",
      " 'are consumed in many forms, generally fresh, sliced, or as an ingredient in '\n",
      " 'prepared foods. Over 80% of watermelons grown in the United States are '\n",
      " 'triploid seedless varieties, preferred for their ease of consumption, '\n",
      " 'increased yield, and premium market value over diploid seeded '\n",
      " 'varieties.</p>\\n'\n",
      " '<p id=\"p-0007\" num=\"0006\"><i>Citrullus lanatus </i>is a member of the family '\n",
      " 'Cucurbitaceae, which contains about 90 genera and 700 to 760 species. The '\n",
      " 'family includes melons, pumpkins, squashes, gourds, cucumber, loofah, and '\n",
      " 'many weeds. The genus <i>Citrullus</i>, to which the watermelon belongs, '\n",
      " 'consists of about four species, including <i>C. colocynthis, C. rehmii</i>, '\n",
      " 'and <i>C. ecirrhosus</i>, all of which may be crossed with each other '\n",
      " 'successfully. The watermelon is believed to be native to southern Africa and '\n",
      " 'has been cultivated there for about 4000 years.</p>\\n'\n",
      " '<p id=\"p-0008\" num=\"0007\">Regular, seeded watermelon is diploid and has '\n",
      " 'eleven pairs of chromosomes (2n=2x=22). There also exist tetraploid '\n",
      " 'varieties, which have 44 chromosomes in their somatic cells (2n=4x=44). '\n",
      " 'Popular &#x201c;seedless&#x201d; varieties are triploid, meaning they have '\n",
      " '33 chromosomes in their somatic cells (3x=33) and are derived from a cross '\n",
      " 'between male diploid and female tetraploid parents. Triploid plants are '\n",
      " 'unable to produce viable gametes; therefore, when triploid plants are grown '\n",
      " 'in the presence of diploid plants, the triploid plants produce seedless '\n",
      " 'fruit. These seedless varieties may sometimes produce fruit with small, '\n",
      " 'edible white ovules, similar to those in immature cucumbers.</p>\\n'\n",
      " '<p id=\"p-0009\" num=\"0008\">Watermelon is an important and valuable field crop '\n",
      " 'with an array of observable and detectable traits of importance both to '\n",
      " 'watermelon consumers and watermelon growers. Traits of importance include '\n",
      " 'taste, texture, size, rind patterns, and shapes. Watermelon consumers desire '\n",
      " 'watermelons that have excellent taste and sweetness. In addition, the '\n",
      " 'texture and firmness of flesh is critical to the consumer experience of '\n",
      " 'eating watermelons. Watermelon fruit may have a wide range of sizes, dozens '\n",
      " 'of possible rind patterns, and an array of shapes from round to cylindrical. '\n",
      " 'Consumers in different markets tend to prefer watermelons with particular '\n",
      " 'sizes, rind patterns, and fruit shapes. Growers also focus on watermelon '\n",
      " 'varieties that have high yields, disease resistances, and can produce fruit '\n",
      " 'that have sufficiently good color and good appearance, as well as be free of '\n",
      " 'defects such as hollow-heart, internal growth, or &#x201c;seeds&#x201d; in '\n",
      " 'the seedless triploid hybrids.</p>\\n'\n",
      " '<p id=\"p-0010\" num=\"0009\">One of the wide variety of colorations and '\n",
      " 'patterns available in watermelon is a spotted pattern. The previously '\n",
      " 'available watermelon that produced fruit with a spotted rind was primarily '\n",
      " 'the heirloom variety &#x2018;Moon and Stars&#x2019;. While this variety has '\n",
      " 'an attractive appearance, it is lacking in commercially desirable '\n",
      " 'characteristics. For example, the fruit flesh of &#x2018;Moon and '\n",
      " 'Stars&#x2019; is less saturated in color and less firm in consistency than '\n",
      " 'desirable, and has a fibrous, stringy flesh texture. Further, the spotting '\n",
      " 'of &#x2018;Moon and Stars&#x2019; only represents one possible spotting '\n",
      " 'type. In addition, the ability to breed new spotted varieties has been '\n",
      " 'limited, because the &#x2018;Moon and Stars&#x2019; heirloom variety is only '\n",
      " 'available as a diploid. Specifically, in order to breed 66% and 100% spotted '\n",
      " 'triploid hybrids, it is necessary to have a spotted tetraploid, which has '\n",
      " 'not previously been available. There exists a need for spotted watermelon '\n",
      " 'lines of different ploidies for use in breeding, and in order to produce '\n",
      " 'varieties with a diverse array of spotted patterns and with commercially '\n",
      " 'desirable characteristics. There also exists a need for markers suitable for '\n",
      " 'use in breeding lines with spotted phenotypes.</p>\\n'\n",
      " '<heading id=\"h-0005\" level=\"1\">SUMMARY OF THE INVENTION</heading>\\n'\n",
      " '<p id=\"p-0011\" num=\"0010\">In order to meet these needs, the present '\n",
      " 'invention is directed to methods of producing high yielding and uniform '\n",
      " 'spotted watermelon lines, and methods of using these lines to produce hybrid '\n",
      " 'varieties with commercially desirable characteristics. Specifically, methods '\n",
      " 'of making homozygous-spotted diploid watermelon lines (HSDL) for use in '\n",
      " 'breeding and methods of making homozygous-spotted tetraploid watermelon '\n",
      " 'lines (HSDL) for use in breeding are described. Further, methods of '\n",
      " 'producing 33%, 66%, or 100% spotted triploid hybrid watermelon plants, as '\n",
      " 'well as methods of producing 50% or 100% spotted diploid hybrid watermelon '\n",
      " 'plants, are described. In addition, this invention is directed to markers '\n",
      " 'able to determine the percentage of spotted allele present in spotted '\n",
      " 'watermelon lines and varieties.</p>\\n'\n",
      " '<p id=\"p-0012\" num=\"0011\">In one aspect, the present invention is directed '\n",
      " 'to methods of producing a homozygous-spotted diploid watermelon line (HSDL), '\n",
      " 'including: (a) crossing a first diploid watermelon plant with a spotted '\n",
      " 'trait with a second diploid watermelon plant without the spotted trait to '\n",
      " 'produce progeny watermelon seed, wherein the presence of the spotted trait '\n",
      " 'is determined by visually phenotyping for the spotted trait or by genotyping '\n",
      " 'to identify a presence of a marker linked to a spotted allele, wherein the '\n",
      " 'spotted allele is located between position 134886 and position 86340 on '\n",
      " 'Chromosome 4 of a &#x2018;Charleston Gray&#x2019; genome; (b) crossing a '\n",
      " 'watermelon plant grown from the progeny watermelon seed with itself or a '\n",
      " 'second watermelon plant to produce additional progeny watermelon seed; (c) '\n",
      " 'planting the additional progeny watermelon seed of step (b) to produce '\n",
      " 'additional watermelon plants; (d) selecting watermelon plants with the '\n",
      " 'spotted trait from the additional watermelon plants of step (c); (e) '\n",
      " 'repeating the crossing of step (b), the planting of step (c), and the '\n",
      " 'selecting of step (d) for an additional 3-10 generations to produce a '\n",
      " 'homozygous-spotted diploid watermelon line (HSDL), wherein the HSDL is 100% '\n",
      " 'spotted. In one embodiment, the first diploid plant of step (a) is '\n",
      " 'homozygous for the spotted allele or heterozygous for the spotted allele. In '\n",
      " 'another embodiment, the marker includes a first marker including a T at '\n",
      " 'position 101 of SEQ ID NO: 1 and/or a second marker including a G at '\n",
      " 'position 101 of SEQ ID NO: 6. In a further embodiment, genotyping of the '\n",
      " 'first marker includes the use of a forward primer including SEQ ID NO: 3, a '\n",
      " 'reverse primer including SEQ ID NO: 4, and/or a probe including SEQ ID NO: '\n",
      " '5, and genotyping of the second marker includes the use of a forward primer '\n",
      " 'including SEQ ID NO: 8, a reverse primer including SEQ ID NO: 9, and/or a '\n",
      " 'probe including SEQ ID NO: 10.</p>\\n'\n",
      " '<p id=\"p-0013\" num=\"0012\">Some embodiments of the present invention are '\n",
      " 'directed to the HSDL plant produced by the method of any of the preceding '\n",
      " 'embodiments.</p>\\n'\n",
      " '<p id=\"p-0014\" num=\"0013\">Further embodiments of the present invention are '\n",
      " 'directed to methods of producing a homozygous-spotted tetraploid watermelon '\n",
      " 'line (HSTL), including: (a) doubling a chromosome number of the HSDL plant '\n",
      " 'of claim <b>5</b>, thereby producing a homozygous-spotted tetraploid '\n",
      " 'watermelon line (HSTL), wherein the HSTL is 100% spotted.</p>\\n'\n",
      " '<p id=\"p-0015\" num=\"0014\">Additional embodiments of the present invention '\n",
      " 'are directed to methods of producing a 33% spotted triploid hybrid '\n",
      " 'watermelon plant, including: (a) crossing the HSDL plant of claim <b>5</b> '\n",
      " 'onto a 0% spotted tetraploid inbred plant, wherein the HSDL plant is a male '\n",
      " 'parent and the tetraploid inbred plant is a female parent, to produce a 33% '\n",
      " 'spotted triploid hybrid watermelon plant. Some embodiments of the present '\n",
      " 'invention are directed to the 33% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of the preceding embodiment.</p>\\n'\n",
      " '<p id=\"p-0016\" num=\"0015\">Yet further embodiments of the present invention '\n",
      " 'are directed to methods of producing a 50% or 100% spotted diploid hybrid '\n",
      " 'watermelon plant, including: (a) crossing the HSDL plant of claim <b>5</b> '\n",
      " 'onto a 0% spotted diploid inbred plant or onto a second HSDL plant of claim '\n",
      " '<b>5</b>, thereby producing a 50% or 100% spotted diploid hybrid watermelon '\n",
      " 'plant. Some embodiments of the present invention are directed to the 50% or '\n",
      " '100% spotted diploid hybrid watermelon plant produced by the method of the '\n",
      " 'preceding embodiment.</p>\\n'\n",
      " '<p id=\"p-0017\" num=\"0016\">In another aspect, the present invention is '\n",
      " 'directed to methods of producing a homozygous-spotted tetraploid watermelon '\n",
      " 'line (HSTL), including: (a) providing a diploid watermelon plant with a '\n",
      " 'spotted trait, wherein the presence of the spotted trait is determined by '\n",
      " 'visually phenotyping for the spotted trait or by genotyping to identify a '\n",
      " 'presence of a marker linked to a spotted allele, and wherein the spotted '\n",
      " 'allele is located between position 134886 and position 86340 on Chromosome 4 '\n",
      " 'of a &#x2018;Charleston Gray&#x2019; genome; (b) doubling a chromosome '\n",
      " 'number of the diploid watermelon plant to produce a tetraploid watermelon '\n",
      " 'plant with the spotted trait; (c) crossing the tetraploid watermelon plant '\n",
      " 'with a non-spotted watermelon plant to produce a F1 generation of watermelon '\n",
      " 'plants; (d) self-pollinating or open pollinating the F1 generation of '\n",
      " 'watermelon plants to produce a F2 generation of watermelon plants; (e) '\n",
      " 'phenotyping progeny watermelon plants from the F2 generation to identify the '\n",
      " 'presence of the spotted trait and/or genotyping progeny watermelon plants '\n",
      " 'from the F2 generation to identify the presence of the marker linked to the '\n",
      " 'spotted allele and using the marker to determine a dosage of the spotted '\n",
      " 'allele; (f) selecting progeny watermelon plants with the highest dosage of '\n",
      " 'the spotted allele; (g) repeating the self-pollinating of step (d), the '\n",
      " 'genotyping of step (e), and the selecting of step (f) for an additional 3-10 '\n",
      " 'generations to produce a homozygous-spotted tetraploid watermelon line '\n",
      " '(HSTL), wherein the HSTL is 100% spotted. In one embodiment, the tetraploid '\n",
      " 'plant of step (b) is homozygous for the spotted allele or heterozygous for '\n",
      " 'the spotted allele. In another embodiment, the marker includes a first '\n",
      " 'marker including a T at position 101 of SEQ ID NO: 1 and/or a second marker '\n",
      " 'including a G at position 101 of SEQ ID NO: 6. In a further embodiment, '\n",
      " 'genotyping of the first marker includes the use of a forward primer '\n",
      " 'including SEQ ID NO: 3, a reverse primer including SEQ ID NO: 4, and/or a '\n",
      " 'probe including SEQ ID NO: 5, and wherein genotyping of the second marker '\n",
      " 'includes the use of a forward primer including SEQ ID NO: 8, a reverse '\n",
      " 'primer including SEQ ID NO: 9, and/or a probe including SEQ ID NO: 10.</p>\\n'\n",
      " '<p id=\"p-0018\" num=\"0017\">Some embodiments of the present invention are '\n",
      " 'directed to the HSTL plant produced by the method of any of the preceding '\n",
      " 'embodiments.</p>\\n'\n",
      " '<p id=\"p-0019\" num=\"0018\">Additional embodiments of the present invention '\n",
      " 'are directed to methods of producing a 66% spotted triploid hybrid '\n",
      " 'watermelon plant, including: (a) crossing the HSTL plant of claim <b>13</b> '\n",
      " 'onto a 0% spotted diploid inbred plant, wherein the HSTL plant is a female '\n",
      " 'parent and the diploid inbred plant is a male parent, to produce a 66% '\n",
      " 'spotted triploid hybrid watermelon plant. Some embodiments of the present '\n",
      " 'invention are directed to the 66% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of the preceding embodiment.</p>\\n'\n",
      " '<p id=\"p-0020\" num=\"0019\">Further embodiments of the present invention are '\n",
      " 'directed to methods of producing a 100% spotted triploid hybrid watermelon '\n",
      " 'plant, including: (a) crossing the HSDL plant of claim <b>5</b> with the '\n",
      " 'HSTL plant of claim <b>13</b> to produce a 100% spotted triploid hybrid '\n",
      " 'watermelon plant. Some embodiments of the present invention are directed to '\n",
      " 'the 100% spotted triploid hybrid watermelon plant produced by the method of '\n",
      " 'the preceding embodiment.</p>\\n'\n",
      " '<p id=\"p-0021\" num=\"0020\">In yet another aspect, the present invention is '\n",
      " 'directed to methods of selecting a spotted watermelon plant, including: (a) '\n",
      " 'crossing a spotted watermelon plant to a watermelon plant to produce a '\n",
      " 'population of progeny watermelon plants; (b) genotyping the population of '\n",
      " 'progeny watermelon plants for a presence of a marker linked to a spotted '\n",
      " 'allele, wherein the spotted allele is located between position 134886 and '\n",
      " 'position 86340 on Chromosome 4 of a &#x2018;Charleston Gray&#x2019; genome; '\n",
      " 'and (c) selecting a progeny plant on the basis of the presence of the one or '\n",
      " 'more sequences genotyped in (b), wherein the progeny watermelon plant is '\n",
      " 'spotted. In one embodiment, the marker includes a first marker including a T '\n",
      " 'at position 101 of SEQ ID NO: 1 and/or a second marker including a G at '\n",
      " 'position 101 of SEQ ID NO: 6, and wherein genotyping of the first marker '\n",
      " 'includes the use of a forward primer including SEQ ID NO: 3, a reverse '\n",
      " 'primer including SEQ ID NO: 4, and/or a probe including SEQ ID NO: 5, and '\n",
      " 'wherein genotyping of the second marker includes the use of a forward primer '\n",
      " 'including SEQ ID NO: 8, a reverse primer including SEQ ID NO: 9, and/or a '\n",
      " 'probe including SEQ ID NO: 10.</p>\\n'\n",
      " '<p id=\"p-0022\" num=\"0021\">In still another aspect, the present invention is '\n",
      " 'directed to methods of selecting a spotted watermelon plant, including: (a) '\n",
      " 'providing a test spotted watermelon plant and a control spotted watermelon '\n",
      " 'plant; (b) using a PCR-based technique to assess the presence of one or more '\n",
      " 'sequences selected from the group of a sequence including T at position 101 '\n",
      " 'of SEQ ID NO: 1 or a sequence including a G at position 101 of SEQ ID NO: 6; '\n",
      " 'and (c) determining the percentage of the spotted allele in the test spotted '\n",
      " 'watermelon plant based on a comparison between the results of step (b) for '\n",
      " 'the test spotted watermelon plant and the control spotted watermelon plant. '\n",
      " 'In one embodiment, the PCR-based technique of step (b) includes the use of '\n",
      " 'primers and probes selected from the group of SEQ ID NO: 3, SEQ ID NO: 4, '\n",
      " 'SEQ ID NO: 5, SEQ ID NO: 8, SEQ ID NO: 9, or SEQ ID NO: 10.</p>\\n'\n",
      " '<heading id=\"h-0006\" level=\"1\">Enumerated Embodiments</heading>\\n'\n",
      " '<p id=\"p-0023\" num=\"0000\">\\n'\n",
      " '<ul id=\"ul0001\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0001-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0002\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0002-0001\" num=\"0022\">1. A method of producing a '\n",
      " 'homozygous-spotted diploid watermelon line (HSDL), comprising:\\n'\n",
      " '        <ul id=\"ul0003\" list-style=\"none\">\\n'\n",
      " '            <li id=\"ul0003-0001\" num=\"0023\">(a) crossing a first diploid '\n",
      " 'watermelon plant with a spotted trait with a second diploid watermelon plant '\n",
      " 'without the spotted trait to produce progeny watermelon seed;</li>\\n'\n",
      " '            <li id=\"ul0003-0002\" num=\"0024\">(b) crossing a watermelon plant '\n",
      " 'grown from the progeny watermelon seed with itself or a second watermelon '\n",
      " 'plant to produce additional progeny watermelon seed;</li>\\n'\n",
      " '            <li id=\"ul0003-0003\" num=\"0025\">(c) planting the additional '\n",
      " 'progeny watermelon seed of step (b) to produce additional watermelon '\n",
      " 'plants;</li>\\n'\n",
      " '            <li id=\"ul0003-0004\" num=\"0026\">(d) selecting watermelon plants '\n",
      " 'with the spotted trait from the additional watermelon plants of step '\n",
      " '(c);</li>\\n'\n",
      " '            <li id=\"ul0003-0005\" num=\"0027\">(d) repeating the crossing of '\n",
      " 'step (b), the planting of step (c), and the selecting of step (d) for an '\n",
      " 'additional 3-10 generations to produce a homozygous-spotted diploid '\n",
      " 'watermelon line (HSDL).</li>\\n'\n",
      " '        </ul>\\n'\n",
      " '        </li>\\n'\n",
      " '        <li id=\"ul0002-0002\" num=\"0028\">2. The method of embodiment 1, '\n",
      " 'wherein the first diploid plant of step (a) is homozygous or heterozygous '\n",
      " 'for a genetic determinant conferring the spotted trait.</li>\\n'\n",
      " '        <li id=\"ul0002-0003\" num=\"0029\">3. The method of embodiment 1 or '\n",
      " 'embodiment 2, further comprising selecting one or more additional traits of '\n",
      " 'interest at step (d), wherein the one or more additional traits of interest '\n",
      " 'are selected from the group consisting of a fruit flesh color trait, a fruit '\n",
      " 'flesh firmness trait, and a fruit taste trait.</li>\\n'\n",
      " '        <li id=\"ul0002-0004\" num=\"0030\">4. A HSDL plant produced by the '\n",
      " 'method of any one of embodiments 1-3.</li>\\n'\n",
      " '        <li id=\"ul0002-0005\" num=\"0031\">5. A plant part from the plant of '\n",
      " 'embodiment 4.</li>\\n'\n",
      " '        <li id=\"ul0002-0006\" num=\"0032\">6. The plant part of embodiment 5, '\n",
      " 'wherein said part is a leaf, an ovule, a pollen grain, a seed, a fruit, a '\n",
      " 'cell, or a portion thereof.</li>\\n'\n",
      " '        <li id=\"ul0002-0007\" num=\"0033\">7. A protoplast produced from the '\n",
      " 'plant of embodiment 4.</li>\\n'\n",
      " '        <li id=\"ul0002-0008\" num=\"0034\">8. A tissue culture of the plant of '\n",
      " 'embodiment 4.</li>\\n'\n",
      " '        <li id=\"ul0002-0009\" num=\"0035\">9. The tissue culture of embodiment '\n",
      " '8, wherein said tissue culture is produced from a plant part selected from '\n",
      " 'the group consisting of leaf, anther, pistil, stem, petiole, root, root tip, '\n",
      " 'fruit, seed, flower, cotyledon, hypocotyl, embryo, and meristematic '\n",
      " 'cell.</li>\\n'\n",
      " '        <li id=\"ul0002-0010\" num=\"0036\">10. A watermelon plant regenerated '\n",
      " 'from the tissue culture of embodiment 9, wherein the plant has all of the '\n",
      " 'morphological and physiological characteristics of the HSDL plant of '\n",
      " 'embodiment 4.</li>\\n'\n",
      " '        <li id=\"ul0002-0011\" num=\"0037\">11. A method of producing a 33% '\n",
      " 'spotted triploid hybrid watermelon plant, comprising: crossing the HSDL '\n",
      " 'plant of embodiment 4 onto a tetraploid inbred plant without the spotted '\n",
      " 'trait, wherein the HSDL plant is a male parent and the tetraploid inbred '\n",
      " 'plant is a female parent, to produce a 33% spotted triploid hybrid '\n",
      " 'watermelon plant.</li>\\n'\n",
      " '        <li id=\"ul0002-0012\" num=\"0038\">12. The 33% spotted triploid hybrid '\n",
      " 'watermelon plant produced by the method of embodiment 11.</li>\\n'\n",
      " '        <li id=\"ul0002-0013\" num=\"0039\">13. A plant part from the plant of '\n",
      " 'embodiment 12.</li>\\n'\n",
      " '        <li id=\"ul0002-0014\" num=\"0040\">14. The plant part of embodiment 13, '\n",
      " 'wherein said part is a leaf, an ovule, a pollen grain, a seed, a fruit, a '\n",
      " 'cell, or a portion thereof.</li>\\n'\n",
      " '        <li id=\"ul0002-0015\" num=\"0041\">15. A protoplast produced from the '\n",
      " 'plant of embodiment 12.</li>\\n'\n",
      " '        <li id=\"ul0002-0016\" num=\"0042\">16. A tissue culture of the plant of '\n",
      " 'embodiment 12.</li>\\n'\n",
      " '        <li id=\"ul0002-0017\" num=\"0043\">17. The tissue culture of embodiment '\n",
      " '16, wherein said tissue culture is produced from a plant part selected from '\n",
      " 'the group consisting of leaf, anther, pistil, stem, petiole, root, root tip, '\n",
      " 'fruit, seed, flower, cotyledon, hypocotyl, embryo, and meristematic '\n",
      " 'cell.</li>\\n'\n",
      " '        <li id=\"ul0002-0018\" num=\"0044\">18. A watermelon plant regenerated '\n",
      " 'from the tissue culture of embodiment 17, wherein the plant has all of the '\n",
      " 'morphological and physiological characteristics of the 33% spotted triploid '\n",
      " 'hybrid watermelon plant of embodiment 12.</li>\\n'\n",
      " '        <li id=\"ul0002-0019\" num=\"0045\">19. A watermelon seed designated as '\n",
      " '&#x2018;E26S.00171&#x2019;, representative sample of seed having been '\n",
      " 'deposited under NCIMB Accession Number X1.</li>\\n'\n",
      " '        <li id=\"ul0002-0020\" num=\"0046\">20. A watermelon plant produced by '\n",
      " 'growing the seed of embodiment 19.</li>\\n'\n",
      " '        <li id=\"ul0002-0021\" num=\"0047\">21. A plant part from the plant of '\n",
      " 'embodiment 20.</li>\\n'\n",
      " '        <li id=\"ul0002-0022\" num=\"0048\">22. The plant part of embodiment 21, '\n",
      " 'wherein said part is a leaf, an ovule, a pollen grain, a seed, a fruit, a '\n",
      " 'cell, or a portion thereof.</li>\\n'\n",
      " '        <li id=\"ul0002-0023\" num=\"0049\">23. The plant part of embodiment 22, '\n",
      " 'wherein said part is a fruit.</li>\\n'\n",
      " '        <li id=\"ul0002-0024\" num=\"0050\">24. A watermelon plant having all '\n",
      " 'the physiological and morphological characteristics of the watermelon plant '\n",
      " 'of embodiment 20.</li>\\n'\n",
      " '        <li id=\"ul0002-0025\" num=\"0051\">25. A plant part from the plant of '\n",
      " 'embodiment 24.</li>\\n'\n",
      " '        <li id=\"ul0002-0026\" num=\"0052\">26. The plant part of embodiment 25, '\n",
      " 'wherein said part is a leaf, an ovule, a pollen grain, a seed, a fruit, a '\n",
      " 'cell, or a portion thereof.</li>\\n'\n",
      " '        <li id=\"ul0002-0027\" num=\"0053\">27. A plant part of embodiment 26, '\n",
      " 'wherein said part is a fruit.</li>\\n'\n",
      " '        <li id=\"ul0002-0028\" num=\"0054\">28. A pollen grain or an ovule of '\n",
      " 'the plant of embodiment 20.</li>\\n'\n",
      " '        <li id=\"ul0002-0029\" num=\"0055\">29. A protoplast produced from the '\n",
      " 'plant of embodiment 20.</li>\\n'\n",
      " '        <li id=\"ul0002-0030\" num=\"0056\">30. A tissue culture of the plant of '\n",
      " 'embodiment 20.</li>\\n'\n",
      " '        <li id=\"ul0002-0031\" num=\"0057\">31. The tissue culture of embodiment '\n",
      " '30, wherein said tissue culture is produced from a plant part selected from '\n",
      " 'the group consisting of leaf, anther, pistil, stem, petiole, root, root tip, '\n",
      " 'fruit, seed, flower, cotyledon, hypocotyl, embryo, and meristematic '\n",
      " 'cell.</li>\\n'\n",
      " '        <li id=\"ul0002-0032\" num=\"0058\">32. A watermelon plant regenerated '\n",
      " 'from the tissue culture of embodiment 31, wherein the plant has all of the '\n",
      " 'morphological and physiological characteristics of a watermelon plant '\n",
      " 'produced by growing watermelon seed designated as '\n",
      " '&#x2018;E26S.00171&#x2019;, representative sample of seed having been '\n",
      " 'deposited under NCIMB Accession Number X1.</li>\\n'\n",
      " '        <li id=\"ul0002-0033\" num=\"0059\">33. A method of producing seedless '\n",
      " 'watermelon fruit, the method comprising:\\n'\n",
      " '        <ul id=\"ul0004\" list-style=\"none\">\\n'\n",
      " '            <li id=\"ul0004-0001\" num=\"0060\">(a) crossing a first '\n",
      " '&#x2018;E26S.00171&#x2019; watermelon plant, representative sample of seed '\n",
      " 'having been deposited under NCIMB Accession Number X1, with a second diploid '\n",
      " 'watermelon plant;</li>\\n'\n",
      " '            <li id=\"ul0004-0002\" num=\"0061\">(b) allowing seedless fruit to '\n",
      " 'form; and</li>\\n'\n",
      " '            <li id=\"ul0004-0003\" num=\"0062\">(c) harvesting the seedless '\n",
      " 'fruit.</li>\\n'\n",
      " '        </ul>\\n'\n",
      " '        </li>\\n'\n",
      " '        <li id=\"ul0002-0034\" num=\"0063\">34. A diploid watermelon plant that '\n",
      " 'produces fruit with a spotted rind, a dark red fruit flesh color, and a firm '\n",
      " 'fruit flesh texture.</li>\\n'\n",
      " '        <li id=\"ul0002-0035\" num=\"0064\">35. A triploid watermelon plant that '\n",
      " 'produces fruit with a spotted rind, a dark red fruit flesh color, and a firm '\n",
      " 'fruit flesh texture.</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0024\" num=\"0065\">In addition to the exemplary aspects and '\n",
      " 'embodiments described above, further aspects and embodiments will become '\n",
      " 'apparent by reference by study of the following descriptions.</p>\\n'\n",
      " '<?summary-of-invention description=\"Summary of Invention\" end=\"tail\"?>\\n'\n",
      " '<?brief-description-of-drawings description=\"Brief Description of Drawings\" '\n",
      " 'end=\"lead\"?>\\n'\n",
      " '<description-of-drawings>\\n'\n",
      " '<heading id=\"h-0007\" level=\"1\">BRIEF DESCRIPTION OF THE DRAWINGS</heading>\\n'\n",
      " '<p id=\"p-0025\" num=\"0066\">The patent or application file contains at least '\n",
      " 'one drawing executed in color. Copies of this patent or patent application '\n",
      " 'publication with color drawings will be provided by the office upon request '\n",
      " 'and payment of the necessary fee.</p>\\n'\n",
      " '<p id=\"p-0026\" num=\"0067\"><figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>1</b>A-<b>1</b>I</figref> show watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;. <figref idref=\"DRAWINGS\">FIG. <b>1</b>A</figref> '\n",
      " 'shows whole fruit of watermelon hybrid &#x2018;E26S.00171&#x2019;. <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>B</figref> shows longitudinal cross-sections '\n",
      " 'of fruit of watermelon hybrid &#x2018;E26S.00171&#x2019;. <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>C</figref> shows the upper side of a leaf with '\n",
      " 'spots of watermelon hybrid &#x2018;E26S.00171&#x2019;. <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>D</figref> shows the lower side of a leaf with '\n",
      " 'spots of watermelon hybrid &#x2018;E26S.00171&#x2019;. <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>E</figref> shows a female flower with a spot '\n",
      " 'of watermelon hybrid &#x2018;E26S.00171&#x2019;. <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>F</figref> shows a female flower of watermelon '\n",
      " 'hybrid &#x2018;E26S.00171&#x2019;. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>1</b>G</figref> shows a male flower of watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;. <figref idref=\"DRAWINGS\">FIG. <b>1</b>H</figref> '\n",
      " 'shows an immature fruit with a spot of watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;. <figref idref=\"DRAWINGS\">FIG. <b>1</b>I</figref> '\n",
      " 'shows a plant with spots of watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;.</p>\\n'\n",
      " '<p id=\"p-0027\" num=\"0068\"><figref idref=\"DRAWINGS\">FIG. <b>2</b></figref> '\n",
      " 'shows fruit of watermelon variety &#x2018;Harvest Moon&#x2019; (U.S. Pat. '\n",
      " 'No. 9,545,065).</p>\\n'\n",
      " '<p id=\"p-0028\" num=\"0069\"><figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>3</b>A-<b>3</b>H</figref> show spotted phenotypes on leaves of the '\n",
      " 'diploid F2 population used for mapping. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>A</figref> shows leaves of sample 2. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>B</figref> shows leaves of sample 7. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>C</figref> shows leaves of sample 11. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>D</figref> shows leaves of sample 73. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>E</figref> shows leaves of sample 5. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>F</figref> shows leaves of sample 34. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>G</figref> shows leaves of sample 42. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>3</b>H</figref> shows leaves of sample 69. The leaves shown in <figref '\n",
      " 'idref=\"DRAWINGS\">FIGS. <b>3</b>A-<b>3</b>D</figref> were later confirmed to '\n",
      " 'all be homozygous for the spotted allele, and the leaves shown in <figref '\n",
      " 'idref=\"DRAWINGS\">FIGS. <b>3</b>E-<b>3</b>H</figref> were later confirmed to '\n",
      " 'all be heterozygous for the spotted allele.</p>\\n'\n",
      " '<p id=\"p-0029\" num=\"0070\"><figref idref=\"DRAWINGS\">FIG. <b>4</b></figref> '\n",
      " 'shows the results of high resolution melting curve analysis. Each group of '\n",
      " 'curves shown in different grey shades corresponds to a specific percentage '\n",
      " 'of spot allele, from top to bottom: 100% spot allele, 0% spot allele, 25% '\n",
      " 'spot allele, 33% spot allele, 50% spot allele, 66% spot allele, and 75% spot '\n",
      " 'allele (black circle designates grouping of curves).</p>\\n'\n",
      " '<p id=\"p-0030\" num=\"0071\"><figref idref=\"DRAWINGS\">FIG. <b>5</b></figref> '\n",
      " 'shows cross-sections of a mature fruit showing fruit flesh (left) and fruit '\n",
      " 'rind (right) and leaves of an exemplary diploid watermelon line with 100% '\n",
      " 'spot allele (i.e., two spot alleles). The fruit rind and the leaves (upper '\n",
      " 'and lower surface) are spotted.</p>\\n'\n",
      " '<p id=\"p-0031\" num=\"0072\"><figref idref=\"DRAWINGS\">FIG. <b>6</b></figref> '\n",
      " 'shows cross-sections of mature fruit showing fruit flesh (bottom left and '\n",
      " 'top right) and fruit rind (bottom right and top left) and leaves of an '\n",
      " 'exemplary tetraploid watermelon line with 100% spot allele (i.e., four spot '\n",
      " 'alleles). The fruit rind and the leaves (upper and lower surface) are '\n",
      " 'spotted.</p>\\n'\n",
      " '<p id=\"p-0032\" num=\"0073\"><figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>7</b>A-<b>7</b>B</figref> show exemplary triploid watermelon varieties '\n",
      " 'with 100% spot allele (i.e., three spot alleles). <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>7</b>A</figref> shows a cross-section of mature '\n",
      " 'fruit showing fruit flesh (bottom), an upper surface of a leaf (middle), and '\n",
      " 'a mature fruit of a first exemplary triploid watermelon variety with 100% '\n",
      " 'spot allele. <figref idref=\"DRAWINGS\">FIG. <b>7</b>B</figref> shows an upper '\n",
      " 'surface of a leaf (bottom) and a mature fruit of a second exemplary triploid '\n",
      " 'watermelon variety with 100% spot allele. In <figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>7</b>A-<b>7</b>B</figref>, the fruit rind and the leaves (upper and lower '\n",
      " 'surface) are spotted.</p>\\n'\n",
      " '<p id=\"p-0033\" num=\"0074\"><figref idref=\"DRAWINGS\">FIG. <b>8</b></figref> '\n",
      " 'shows upper surfaces of leaves (bottom) and a mature fruit (top) of an '\n",
      " 'exemplary triploid watermelon variety with 66% spot allele (i.e., two spot '\n",
      " 'alleles). The fruit rind and the leaves (upper and lower surface) are '\n",
      " 'spotted.</p>\\n'\n",
      " '</description-of-drawings>\\n'\n",
      " '<?brief-description-of-drawings description=\"Brief Description of Drawings\" '\n",
      " 'end=\"tail\"?>\\n'\n",
      " '<?detailed-description description=\"Detailed Description\" end=\"lead\"?>\\n'\n",
      " '<heading id=\"h-0008\" level=\"1\">DETAILED DESCRIPTION OF THE '\n",
      " 'INVENTION</heading>\\n'\n",
      " '<p id=\"p-0034\" num=\"0075\">The present invention is directed towards '\n",
      " 'watermelon varieties exhibiting desired spotted phenotypes. Desired spotted '\n",
      " 'phenotypes include fruit rinds with at least one yellow spot of a diameter '\n",
      " 'of at least 0.2 cm, preferably multiple yellow spots scattered over a '\n",
      " 'non-yellow background color. The background color may include any known '\n",
      " 'watermelon color and rind pattern, including but not limited to dark green, '\n",
      " 'medium green, peacock, gray, dark mottled stripe, and tiger stripe. Desired '\n",
      " 'flesh characteristics include dark red, intense red, or vivid red color. '\n",
      " 'Further, desired flesh characteristics include medium to firm flesh '\n",
      " 'firmness. In an exemplary embodiment, the present invention is directed '\n",
      " 'toward triploid watermelon plants that produce fruit with yellow-spotted '\n",
      " 'rinds, a vivid red flesh color, and medium to firm flesh firmness.</p>\\n'\n",
      " '<p id=\"p-0035\" num=\"0076\">These desired rind and flesh characteristics can '\n",
      " 'be accomplished by crossing two watermelon plants such that the progeny '\n",
      " 'watermelon plants exhibit rind and flesh characteristics of both parent '\n",
      " 'plants. In an exemplary embodiment, the present invention is directed '\n",
      " 'towards diploid watermelon plants that can be used to breed triploid, '\n",
      " 'seedless, watermelon plants with spotted rinds and red flesh color. The '\n",
      " 'diploid parent is used as the male parent, and the tetraploid parent is used '\n",
      " 'as the female parent when breeding a triploid. In a preferred embodiment, '\n",
      " 'the diploid watermelon plants used for breeding have spotted rinds, '\n",
      " 'medium-firm flesh, fine (little fiber) flesh coarseness, and red flesh. In a '\n",
      " 'further exemplary embodiment, the present invention is directed towards '\n",
      " 'tetraploid watermelon plants that can be used to breed triploid, seedless, '\n",
      " 'watermelon plants with spotted rinds and red flesh color. The tetraploid '\n",
      " 'parent is used as the female parent, and the diploid parent is used as the '\n",
      " 'male parent when breeding a triploid. In a preferred embodiment, the '\n",
      " 'tetraploid watermelon plants used for breeding have spotted rinds, '\n",
      " 'medium-firm flesh, fine (little fiber) flesh coarseness, and red flesh.</p>\\n'\n",
      " '<p id=\"p-0036\" num=\"0077\">The present invention is directed towards diploid, '\n",
      " 'triploid, and tetraploid varieties of watermelon exhibiting spotted '\n",
      " 'patterning. This spotted patterning can be accomplished by crossing two '\n",
      " 'watermelon plants such that the progeny watermelon plants exhibit patterning '\n",
      " 'of both parent plants. In an exemplary embodiment, the present invention is '\n",
      " 'directed towards triploid seedless watermelon plants. Triploid plants are '\n",
      " 'produced by the cross of a diploid male and tetraploid female. Genes '\n",
      " 'exhibited by triploid plants can be thought of as contributed by the parents '\n",
      " 'in &#x201c;doses.&#x201d; By way of example, if a diploid (2&#xd7;) spotted '\n",
      " 'plant A is crossed with a tetraploid (4&#xd7;) plant B, the result of the '\n",
      " 'cross would be triploid (3&#xd7;) plant C, wherein plant C receives one set '\n",
      " 'of genes from A and two sets of genes from B. It can be said that, in this '\n",
      " 'exemplary cross, that plant C has one &#x201c;dose&#x201d; of genes from '\n",
      " 'plant A and two &#x201c;doses&#x201d; of genes from plant B. By this model, '\n",
      " 'genes can be transferred to progeny in a dose-dependent manner. In a further '\n",
      " 'exemplary embodiment, the present invention is directed toward diploid '\n",
      " 'hybrid watermelon plants. Diploid plants are produced by the cross of a '\n",
      " 'diploid male and diploid female. By way of example, if a diploid (2&#xd7;) '\n",
      " 'spotted plant A is crossed with a diploid (2&#xd7;) plant B, the result of '\n",
      " 'the cross would be diploid (2&#xd7;) plant C, wherein plant C receives one '\n",
      " 'set of genes from A and one set of genes from B. It can be said that, in '\n",
      " 'this exemplary cross, that plant C has one &#x201c;dose&#x201d; of genes '\n",
      " 'from plant A and one &#x201c;dose&#x201d; of genes from plant B.</p>\\n'\n",
      " '<p id=\"p-0037\" num=\"0078\">The dosage effect of the alleles underlying the '\n",
      " 'spotted trait is additive. Therefore, a 33% (i.e., one-dose) spotted '\n",
      " 'triploid hybrid watermelon plant, which has the spotted trait from its '\n",
      " 'diploid male parent (one dose) and a non-spotted trait from its tetraploid '\n",
      " 'female parent (two doses), has an appearance that is intermediate between '\n",
      " 'non-spotted and fully spotted. It is, in essence, 66% non-spotted and 33% '\n",
      " 'spotted, with an intermediate level of spotting. This intermediate level of '\n",
      " 'spotting is highly valued as an attractive and desired level of spotting '\n",
      " '(i.e., the fruit is spotted, but not too spotted).</p>\\n'\n",
      " '<p id=\"p-0038\" num=\"0079\">Watermelon hybrid &#x2018;E26S.00171&#x2019; is an '\n",
      " 'exemplary 33% spotted triploid hybrid. As can be seen in <figref '\n",
      " 'idref=\"DRAWINGS\">FIGS. <b>1</b>A-<b>1</b>I</figref>, the spotting is present '\n",
      " 'on the plant as well as the fruit. <figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>1</b>C-<b>1</b>D</figref> show spotting on the leaves. <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>E</figref> shows a spot on the immature female '\n",
      " 'flower. <figref idref=\"DRAWINGS\">FIG. <b>1</b>H</figref> shows a spot on the '\n",
      " 'immature fruit. In some embodiments, the fruit may show spots when immature '\n",
      " '(i.e., at young stages). In some embodiments, the fruit may show spots when '\n",
      " 'more mature (i.e., when immature, spots are not shown on the fruit). '\n",
      " 'Further, the immature fruit has a lighter green background rind pattern (see '\n",
      " '<figref idref=\"DRAWINGS\">FIG. <b>1</b>H</figref>) as compared to the mature '\n",
      " 'rind pattern (see <figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>1</b>A-<b>1</b>B</figref>). <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>1</b>I</figref> shows a whole plant with spotting throughout. The spots '\n",
      " 'on watermelon hybrid &#x2018;E26S.00171&#x2019; are yellow. The majority of '\n",
      " 'the spots are less than 2 cm in diameter, and spots range from 0.2 cm to 15 '\n",
      " 'cm in diameter.</p>\\n'\n",
      " '<p id=\"p-0039\" num=\"0080\">The present invention is also directed toward '\n",
      " 'triploid watermelon varieties with different dosages of spots. <figref '\n",
      " 'idref=\"DRAWINGS\">FIGS. <b>7</b>A-<b>7</b>B</figref> show exemplary triploid '\n",
      " 'watermelon varieties with 100% spot allele (i.e., three spot alleles), or '\n",
      " '100% spotted triploid hybrids. <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>8</b></figref> shows an exemplary triploid watermelon variety with 66% '\n",
      " 'spot allele (i.e., two spot alleles), or a 66% spotted triploid hybrid. For '\n",
      " 'both the 100% spotted and the 66% spotted varieties, the fruit rind and the '\n",
      " 'leaves (upper and lower surface) are spotted.</p>\\n'\n",
      " '<p id=\"p-0040\" num=\"0081\">The present invention is further directed towards '\n",
      " 'diploid and tetraploid varieties and/or lines of watermelon exhibiting the '\n",
      " 'spotted phenotype (e.g., on fruit rind and leaves). <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>5</b></figref> shows an exemplary 100% spotted '\n",
      " 'diploid line, and <figref idref=\"DRAWINGS\">FIG. <b>6</b></figref> shows an '\n",
      " 'exemplary 100% spotted tetraploid line.</p>\\n'\n",
      " '<p id=\"p-0041\" num=\"0082\">The percentage or dosage of the alleles underlying '\n",
      " 'the spotted rind patterning trait is linked to ploidy. For all ploidies '\n",
      " '(diploid, triploid, and tetraploid), 100% means the allele is homozygously '\n",
      " 'present, and 0% means the allele is absent (i.e., homozygously absent). The '\n",
      " 'number of alleles present, however, varies by ploidy. If &#x201c;A&#x201d; '\n",
      " 'is used to designate the spotted allele, and &#x201c;a&#x201d; is used to '\n",
      " 'designate the non-spotted allele, a 100% spotted diploid line has two '\n",
      " 'spotted alleles (AA), a 100% spotted triploid line has three spotted alleles '\n",
      " '(AAA), and a 100% spotted tetraploid line has four spotted alleles (AAAA). '\n",
      " 'For tetraploid lines, 25% means there is one spotted allele present of a '\n",
      " 'possible four (Aaaa), 50% means there are two spotted alleles present '\n",
      " '(AAaa), and 75% means there are three spotted alleles present (AAAa). For '\n",
      " 'triploid varieties, 33% means there is one spotted allele present of a '\n",
      " 'possible three (Aaa), and 66% means there are two spotted alleles present '\n",
      " '(AAa). For diploid lines, 50% means there is one spotted allele present of a '\n",
      " 'possible two (i.e., the line is heterozygous; Aa). It is thought that the '\n",
      " 'percentage of the alleles corresponds to the density of the spots, e.g., on '\n",
      " 'leaves, fruit, etc.</p>\\n'\n",
      " '<p id=\"p-0042\" num=\"0083\">There are numerous steps in the development of any '\n",
      " 'novel, desirable plant germplasm. Plant breeding begins with the analysis '\n",
      " 'and definition of problems and weaknesses of the current germplasm, the '\n",
      " 'establishment of program goals, and the definition of specific breeding '\n",
      " 'objectives. The next step is selection of germplasm that possess the traits '\n",
      " 'to meet the program goals. The selected germplasm is crossed in order to '\n",
      " 'recombine the desired traits, and then through selection, varieties or '\n",
      " 'parent lines are developed. The goal is to combine in a single line, '\n",
      " 'variety, or hybrid an improved combination of desirable traits from the '\n",
      " 'parental germplasm. These important traits may include higher yield, field '\n",
      " 'performance, resistance to diseases and insects, and tolerance to drought '\n",
      " 'and heat. These important traits may also include fruit and agronomic '\n",
      " 'quality such as fruit shape, fruit appearance, fruit flesh firmness, and '\n",
      " 'internal quality of fruit.</p>\\n'\n",
      " '<p id=\"p-0043\" num=\"0084\">Choice of breeding or selection methods can depend '\n",
      " 'on the mode of plant reproduction, the heritability of the trait(s) being '\n",
      " 'improved, and the type of cultivar used commercially (e.g., F<sub>1 '\n",
      " '</sub>hybrid cultivar, pureline cultivar, etc.). For highly heritable '\n",
      " 'traits, a choice of superior individual plants evaluated at a single '\n",
      " 'location will be effective, whereas for traits with low heritability, '\n",
      " 'selection should be based on mean values obtained from replicated '\n",
      " 'evaluations of families of related plants. Popular selection methods '\n",
      " 'commonly include pedigree selection, modified pedigree selection, mass '\n",
      " 'selection, and recurrent selection.</p>\\n'\n",
      " '<p id=\"p-0044\" num=\"0085\">The complexity of inheritance influences choice of '\n",
      " 'the breeding method. Backcross breeding is used to transfer one or a few '\n",
      " 'favorable genes for a highly heritable trait into a desirable cultivar. This '\n",
      " 'approach has been used extensively for breeding disease-resistant varieties. '\n",
      " 'Various recurrent selection techniques are used to improve quantitatively '\n",
      " 'inherited traits controlled by numerous genes. The use of recurrent '\n",
      " 'selection in self-pollinating crops depends on the ease of pollination, the '\n",
      " 'frequency of successful hybrids from each pollination, and the number of '\n",
      " 'hybrid offspring from each successful cross.</p>\\n'\n",
      " '<p id=\"p-0045\" num=\"0086\">Each breeding program may include a periodic, '\n",
      " 'objective evaluation of the efficiency of the breeding procedure. Evaluation '\n",
      " 'criteria vary depending on the goal and objectives, and can include gain '\n",
      " 'from selection per year based on comparisons to an appropriate standard, '\n",
      " 'overall value of the advanced breeding lines, and number of successful '\n",
      " 'cultivars produced per unit of input (e.g., per year, per dollar expended, '\n",
      " 'etc.).</p>\\n'\n",
      " '<p id=\"p-0046\" num=\"0087\">Promising advanced breeding lines are thoroughly '\n",
      " 'tested and compared to appropriate standards in environments representative '\n",
      " 'of the commercial target area(s) for at least three years. The best lines '\n",
      " 'can then be candidates for new commercial cultivars. Those still deficient '\n",
      " 'in a few traits may be used as parents to produce new populations for '\n",
      " 'further selection. These processes, which lead to the final step of '\n",
      " 'marketing and distribution, may take from ten to twenty years from the time '\n",
      " 'the first cross or selection is made.</p>\\n'\n",
      " '<p id=\"p-0047\" num=\"0088\">One goal of watermelon plant breeding is to '\n",
      " 'develop new, unique, and genetically superior watermelon cultivars and '\n",
      " 'hybrids. A breeder can initially select and cross two or more parental '\n",
      " 'lines, followed by repeated selfing and selection, producing many new '\n",
      " 'genetic combinations. A plant breeder can then select which germplasms to '\n",
      " 'advance to the next generation. These germplasms may then be grown under '\n",
      " 'different geographical, climatic, and soil conditions, and further '\n",
      " 'selections can be made during, and at the end of, the growing season.</p>\\n'\n",
      " '<p id=\"p-0048\" num=\"0089\">The development of commercial watermelon cultivars '\n",
      " 'thus requires the development of watermelon parental lines, the crossing of '\n",
      " 'these lines, and the evaluation of the crosses. Pedigree breeding and '\n",
      " 'recurrent selection breeding methods may be used to develop cultivars from '\n",
      " 'breeding populations. Breeding programs can be used to combine desirable '\n",
      " 'traits from two or more varieties or various broad-based sources into '\n",
      " 'breeding pools from which lines are developed by selfing and selection of '\n",
      " 'desired phenotypes. The new lines are crossed with other lines and the '\n",
      " 'hybrids from these crosses are evaluated to determine which have commercial '\n",
      " 'potential.</p>\\n'\n",
      " '<p id=\"p-0049\" num=\"0090\">Pedigree breeding is generally used for the '\n",
      " 'improvement of self-pollinating crops or inbred lines of cross-pollinating '\n",
      " 'crops. Two parents which possess favorable, complementary traits are crossed '\n",
      " 'to produce an F<sub>1</sub>. An F<sub>2 </sub>population is produced by '\n",
      " 'selfing one or several F<sub>1</sub>s or by intercrossing two F<sub>1</sub>s '\n",
      " '(sib mating). Selection of the best individuals is usually begun in the '\n",
      " 'F<sub>2 </sub>population; then, beginning in the F<sub>3</sub>, the best '\n",
      " 'individuals in the best families are selected. Replicated testing of '\n",
      " 'families, or hybrid combinations involving individuals of these families, '\n",
      " 'often follows in the F<sub>4 </sub>generation to improve the effectiveness '\n",
      " 'of selection for traits with low heritability. At an advanced stage of '\n",
      " 'inbreeding (i.e., F<sub>6 </sub>and F<sub>7</sub>), the best lines or '\n",
      " 'mixtures of phenotypically similar lines are tested for potential release as '\n",
      " 'new cultivars.</p>\\n'\n",
      " '<p id=\"p-0050\" num=\"0091\">Mass and recurrent selections can be used to '\n",
      " 'improve populations of either self- or cross-pollinating crops. A '\n",
      " 'genetically variable population of heterozygous individuals is either '\n",
      " 'identified or created by intercrossing several different parents. The best '\n",
      " 'plants are selected based on individual superiority, outstanding progeny, or '\n",
      " 'excellent combining ability. The selected plants are intercrossed to produce '\n",
      " 'a new population in which further cycles of selection are continued.</p>\\n'\n",
      " '<p id=\"p-0051\" num=\"0092\">Backcross breeding may be used to transfer genes '\n",
      " 'for a simply inherited, highly heritable trait into a desirable homozygous '\n",
      " 'cultivar or line that is the recurrent parent. The source of the trait to be '\n",
      " 'transferred is called the donor parent. The resulting plant is expected to '\n",
      " 'have the attributes of the recurrent parent (e.g., cultivar) and the '\n",
      " 'desirable trait transferred from the donor parent. After the initial cross, '\n",
      " 'individuals possessing the phenotype of the donor parent are selected and '\n",
      " 'repeatedly crossed (backcrossed) to the recurrent parent. The resulting '\n",
      " 'plant is expected to have the attributes of the recurrent parent (e.g., '\n",
      " 'cultivar) and the desirable trait transferred from the donor parent.</p>\\n'\n",
      " '<p id=\"p-0052\" num=\"0093\">New varieties can also be developed from more than '\n",
      " 'two parents. The technique, known as modified backcrossing, uses different '\n",
      " 'recurrent parents during the backcrossing. Modified backcrossing may be used '\n",
      " 'to replace the original recurrent parent with a variety having certain more '\n",
      " 'desirable characteristics or multiple parents may be used to obtain '\n",
      " 'different desirable characteristics from each.</p>\\n'\n",
      " '<p id=\"p-0053\" num=\"0094\">In addition to being used to create a backcross '\n",
      " 'conversion, backcrossing can also be used in combination with pedigree '\n",
      " 'breeding. As discussed previously, backcrossing can be used to transfer one '\n",
      " 'or more specifically desirable traits from one variety, the donor parent, to '\n",
      " 'a developed variety called the recurrent parent, which has overall good '\n",
      " 'agronomic characteristics yet lacks that desirable trait or traits. However, '\n",
      " 'the same procedure can be used to move the progeny toward the genotype of '\n",
      " 'the recurrent parent, but at the same time retain many components of the '\n",
      " 'nonrecurrent parent by stopping the backcrossing at an early stage and '\n",
      " 'proceeding with selfing and selection. For example, an inbred watermelon '\n",
      " 'line may be crossed with another variety to produce a first generation '\n",
      " 'progeny plant. The first generation progeny plant may then be backcrossed to '\n",
      " 'one of its parent varieties to create a BC1 or BC2. Progeny are selfed and '\n",
      " 'selected so that the newly developed variety has many of the attributes of '\n",
      " 'the recurrent parent and yet several of the desired attributes of the '\n",
      " 'nonrecurrent parent. This approach leverages the value and strengths of the '\n",
      " 'recurrent parent for use in new watermelon varieties.</p>\\n'\n",
      " '<p id=\"p-0054\" num=\"0095\">The single-seed descent procedure in the strict '\n",
      " 'sense refers to planting a segregating population, harvesting a sample of '\n",
      " 'one seed per plant, and using the one-seed sample to plant the next '\n",
      " 'generation. When the population has been advanced from the F<sub>2 </sub>to '\n",
      " 'the desired level of inbreeding, the plants from which lines are derived '\n",
      " 'will each trace to different F<sub>2 </sub>individuals. The number of plants '\n",
      " 'in a population declines each generation due to failure of some seeds to '\n",
      " 'germinate or some plants to produce at least one seed. As a result, not all '\n",
      " 'of the F<sub>2 </sub>plants originally sampled in the population will be '\n",
      " 'represented by a progeny when generation advance is completed.</p>\\n'\n",
      " '<p id=\"p-0055\" num=\"0096\">In addition to phenotypic observations, the '\n",
      " 'genotype of a plant can also be examined. There are many laboratory-based '\n",
      " 'techniques known in the art that are available for the analysis, comparison '\n",
      " 'and characterization of plant genotype. Such techniques include, without '\n",
      " 'limitation, High Resolution Melting (HRM), DNA- or RNA-sequencing, CAPS '\n",
      " 'Markers, ELISA, Western blot, microarrays, Single Nucleotide Polymorphisms '\n",
      " '(SNPs), Isozyme Electrophoresis, Restriction Fragment Length Polymorphisms '\n",
      " '(RFLPs), Randomly Amplified Polymorphic DNAs (RAPDs), Arbitrarily Primed '\n",
      " 'Polymerase Chain Reaction (AP-PCR), Differential Display Polymerase Chain '\n",
      " 'Reaction (DD-PCR), Quantitative Real-Time Polymerase Chain Reaction '\n",
      " '(qRT-PCR), DNA Amplification Fingerprinting (DAF), Sequence Characterized '\n",
      " 'Amplified Regions (SCARs), Amplified Fragment Length Polymorphisms (AFLPs), '\n",
      " 'and Simple Sequence Repeats (SSRs, which are also referred to as '\n",
      " 'Microsatellites).</p>\\n'\n",
      " '<p id=\"p-0056\" num=\"0097\">Molecular markers can also be used during the '\n",
      " 'breeding process for the selection of qualitative traits. For example, '\n",
      " 'markers closely linked to alleles or markers containing sequences within the '\n",
      " 'actual alleles of interest can be used to select plants that contain the '\n",
      " 'alleles of interest during a backcrossing breeding program. The markers can '\n",
      " 'also be used to select toward the genome of the recurrent parent and against '\n",
      " 'the markers of the donor parent. This procedure attempts to minimize the '\n",
      " 'amount of genome from the donor parent that remains in the selected plants. '\n",
      " 'It can also be used to reduce the number of crosses back to the recurrent '\n",
      " 'parent needed in a backcrossing program. The use of molecular markers in the '\n",
      " 'selection process is often called genetic marker enhanced selection or '\n",
      " 'marker-assisted selection. Molecular markers may also be used to identify '\n",
      " 'and exclude certain sources of germplasm as parental varieties or ancestors '\n",
      " 'of a plant by providing a means of tracking genetic profiles through '\n",
      " 'crosses.</p>\\n'\n",
      " '<p id=\"p-0057\" num=\"0098\">Mutation breeding may also be used to introduce '\n",
      " 'new traits into watermelon varieties. Mutations that occur spontaneously or '\n",
      " 'are artificially induced can be useful sources of variability for a plant '\n",
      " 'breeder. The goal of artificial mutagenesis is to increase the rate of '\n",
      " 'mutation for a desired characteristic. Mutation rates can be increased by '\n",
      " 'many different means including temperature, long-term seed storage, tissue '\n",
      " 'culture conditions, radiation (such as X-rays, Gamma rays, neutrons, Beta '\n",
      " 'radiation, or ultraviolet radiation), chemical mutagens (such as base '\n",
      " 'analogs like 5-bromo-uracil), antibiotics, alkylating agents (such as sulfur '\n",
      " 'mustards, nitrogen mustards, epoxides, ethyleneamines, sulfates, sulfonates, '\n",
      " 'sulfones, or lactones), azide, hydroxylamine, nitrous acid, or acridines. '\n",
      " 'Once a desired trait is observed through mutagenesis the trait may then be '\n",
      " 'incorporated into existing germplasm by traditional breeding techniques. '\n",
      " 'Details of mutation breeding can be found in Principles of Cultivar '\n",
      " 'Development by Fehr, Macmillan Publishing Company, 1993.</p>\\n'\n",
      " '<p id=\"p-0058\" num=\"0099\">The production of double haploids can also be used '\n",
      " 'for the development of homozygous varieties in a breeding program. Double '\n",
      " 'haploids are produced by the doubling of a set of chromosomes from a '\n",
      " 'heterozygous plant to produce a completely homozygous individual. For '\n",
      " 'example, see Wan et al., <i>Theor. Appl. Genet., </i>77:889-892, 1989.</p>\\n'\n",
      " '<p id=\"p-0059\" num=\"0100\">Additional non-limiting examples of breeding '\n",
      " 'methods that may be used include, without limitation, those found in '\n",
      " '<i>Principles of Plant Breeding</i>, John Wiley and Son, pp. 115-161, 1960; '\n",
      " 'Allard, 1960; Simmonds, 1979; Sneep et al., 1979; Fehr, 1987; '\n",
      " '&#x201c;Carrots and Related Vegetable Umbelliferae&#x201d;, Rubatzky, V. E., '\n",
      " 'et al., 1999.</p>\\n'\n",
      " '<heading id=\"h-0009\" level=\"1\">Definitions</heading>\\n'\n",
      " '<p id=\"p-0060\" num=\"0101\">In the description that follows, a number of terms '\n",
      " 'are used. In order to provide a clear and consistent understanding of the '\n",
      " 'specification and claims, including the scope to be given such terms, the '\n",
      " 'following definitions are provided:</p>\\n'\n",
      " '<p id=\"p-0061\" num=\"0102\">Allele. The allele is any of one or more '\n",
      " 'alternative forms of a gene, all of which relate to one trait or '\n",
      " 'characteristic. In a diploid cell or organism, the two alleles of a given '\n",
      " 'gene occupy corresponding loci on a pair of homologous chromosomes.</p>\\n'\n",
      " '<p id=\"p-0062\" num=\"0103\">Androecious plant. A plant having staminate '\n",
      " 'flowers only.</p>\\n'\n",
      " '<p id=\"p-0063\" num=\"0104\">Backcrossing. Backcrossing is a process in which a '\n",
      " 'breeder repeatedly crosses hybrid progeny back to one of the parents, for '\n",
      " 'example, a first generation hybrid F<sub>1 </sub>with one of the parental '\n",
      " 'genotype of the F<sub>1 </sub>hybrid.</p>\\n'\n",
      " '<p id=\"p-0064\" num=\"0105\">Blossom end. The blossom end is the distal end of '\n",
      " 'the fruit (the &#x201c;far&#x201d; end as measured from the base of the '\n",
      " 'plant) where the flower blossom is located. The other end of a fruit is the '\n",
      " 'stem end.</p>\\n'\n",
      " '<p id=\"p-0065\" num=\"0106\">Blossom scar. The blossom scar is the small mark '\n",
      " 'left on the distal end of the fruit after the flower falls off.</p>\\n'\n",
      " '<p id=\"p-0066\" num=\"0107\">Covered cultivation. Any type of cultivation where '\n",
      " 'the plants are not exposed to direct sunlight. The covering includes but is '\n",
      " 'not limited to greenhouses, glasshouses, nethouses, plastic houses, and '\n",
      " 'tunnels.</p>\\n'\n",
      " '<p id=\"p-0067\" num=\"0108\">Essentially all the physiological and '\n",
      " 'morphological characteristics. A plant having essentially all the '\n",
      " 'physiological and morphological characteristics means a plant having the '\n",
      " 'physiological and morphological characteristics of the recurrent parent, '\n",
      " 'except for the characteristics derived from the converted gene.</p>\\n'\n",
      " '<p id=\"p-0068\" num=\"0109\">Gene. As used herein, &#x201c;gene&#x201d; refers '\n",
      " 'to a segment of nucleic acid. A gene can be introduced into a genome of a '\n",
      " 'species, whether from a different species or from the same species, using '\n",
      " 'transformation or various breeding methods.</p>\\n'\n",
      " '<p id=\"p-0069\" num=\"0110\">Gynoecious plant. A plant having pistillate '\n",
      " 'flowers only.</p>\\n'\n",
      " '<p id=\"p-0070\" num=\"0111\">Monoecious plant. A plant having separate '\n",
      " 'staminate and pistillate flowers on the same plant.</p>\\n'\n",
      " '<p id=\"p-0071\" num=\"0112\">Percent Identity. Percent identity as used herein '\n",
      " 'refers to the comparison of the homozygous alleles of two watermelon lines, '\n",
      " 'hybrids, or varieties. Percent identity is determined by comparing a '\n",
      " 'statistically significant number of the homozygous alleles of two developed '\n",
      " 'varieties, lines or hybrids. For example, a percent identity of 90% between '\n",
      " 'watermelon plant 1 and watermelon plant 2 means that the two plants have the '\n",
      " 'same allele at 90% of their loci.</p>\\n'\n",
      " '<p id=\"p-0072\" num=\"0113\">Percent Similarity. Percent similarity as used '\n",
      " 'herein refers to the comparison of the homozygous alleles of a watermelon '\n",
      " 'plant with another plant. For example, if a homozygous allele of hybrid '\n",
      " 'watermelon &#x2018;E26S.00171&#x2019; matches at least one of the alleles '\n",
      " 'from the other plant then they are scored as similar. Percent similarity is '\n",
      " 'determined by comparing a statistically significant number of loci and '\n",
      " 'recording the number of loci with similar alleles as a percentage. A percent '\n",
      " 'similarity of 90% between hybrid watermelon &#x2018;E26S.00171&#x2019; and '\n",
      " 'another plant means that hybrid watermelon &#x2018;E26S.00171&#x2019; '\n",
      " 'matches at least one of the alleles of the other plant at 90% of the '\n",
      " 'loci.</p>\\n'\n",
      " '<p id=\"p-0073\" num=\"0114\">Pips. Immature white ovules present in fruit '\n",
      " 'produced by triploid watermelon plants.</p>\\n'\n",
      " '<p id=\"p-0074\" num=\"0115\">Propagate. To &#x201c;propagate&#x201d; a plant '\n",
      " 'means to reproduce the plant by means including, but not limited to, seeds, '\n",
      " 'cuttings, divisions, tissue culture, embryo culture, or other in vitro '\n",
      " 'method.</p>\\n'\n",
      " '<p id=\"p-0075\" num=\"0116\">Quantitative Trait Loci (QTL). Quantitative trait '\n",
      " 'loci refer to genetic loci that control to some degree numerically '\n",
      " 'representable traits that are usually continuously distributed.</p>\\n'\n",
      " '<p id=\"p-0076\" num=\"0117\">Regeneration. Regeneration refers to the '\n",
      " 'development of a plant from tissue culture.</p>\\n'\n",
      " '<p id=\"p-0077\" num=\"0118\">Seedless. Seedless plants refer to triploid '\n",
      " 'watermelon plants that produce fruit without viable seeds. These fruit may '\n",
      " 'be devoid of seeds or seed-like structures, or may contain immature white '\n",
      " 'ovules.</p>\\n'\n",
      " '<p id=\"p-0078\" num=\"0119\">Single gene converted. Single gene converted or '\n",
      " 'conversion plant refers to plants which are developed by a plant breeding '\n",
      " 'technique called backcrossing wherein essentially all of the desired '\n",
      " 'morphological and physiological characteristics of an inbred are recovered '\n",
      " 'in addition to the single gene transferred into the inbred via the '\n",
      " 'backcrossing technique, genetic engineering, or mutation.</p>\\n'\n",
      " '<p id=\"p-0079\" num=\"0120\">Transgene. A &#x201c;transgene&#x201d; is a gene '\n",
      " 'taken or copied from one organism and inserted into another organism. A '\n",
      " 'transgene may be a gene that is foreign to the receiving organism or it may '\n",
      " 'be a modified version of a native, or endogenous, gene.</p>\\n'\n",
      " '<p id=\"p-0080\" num=\"0000\">Overview of the hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;</p>\\n'\n",
      " '<p id=\"p-0081\" num=\"0121\">Watermelon hybrid &#x2018;E26S.00171&#x2019; is a '\n",
      " 'triploid hybrid that produces fruit with a spotted rind pattern and vivid '\n",
      " 'red flesh color. <figref idref=\"DRAWINGS\">FIG. <b>1</b>A</figref> shows '\n",
      " 'whole fruit of watermelon hybrid &#x2018;E26S.00171&#x2019;, <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>B</figref> shows longitudinal cross-sections '\n",
      " 'of fruit of watermelon hybrid &#x2018;E26S.00171&#x2019;, <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>C</figref> shows the upper side of a leaf with '\n",
      " 'spots of watermelon hybrid &#x2018;E26S.00171&#x2019;, <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>D</figref> shows the lower side of a leaf with '\n",
      " 'spots of watermelon hybrid &#x2018;E26S.00171&#x2019;, <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>E</figref> shows a female flower with a spot '\n",
      " 'of watermelon hybrid &#x2018;E26S.00171&#x2019;, <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>1</b>F</figref> shows a female flower of watermelon '\n",
      " 'hybrid &#x2018;E26S.00171&#x2019;, <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>1</b>G</figref> shows a male flower of watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;, <figref idref=\"DRAWINGS\">FIG. <b>1</b>H</figref> '\n",
      " 'shows an immature fruit with a spot of watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;, and <figref idref=\"DRAWINGS\">FIG. '\n",
      " '<b>1</b>I</figref> shows a plant with spots of watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;. Watermelon hybrid &#x2018;E26S.00171&#x2019; is '\n",
      " 'the result of numerous generations of plant selections chosen for its rind '\n",
      " 'appearance; vivid red color and firmness of its flesh; and very low '\n",
      " 'incidence of hollow heart.</p>\\n'\n",
      " '<p id=\"p-0082\" num=\"0122\">The hybrid has shown uniformity and stability for '\n",
      " 'the traits, within the limits of environmental influence for the traits. No '\n",
      " 'variant traits have been observed or are expected in '\n",
      " '&#x2018;E26S.00171&#x2019;.</p>\\n'\n",
      " '<heading id=\"h-0010\" level=\"2\">Objective Description of the Hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;</heading>\\n'\n",
      " '<p id=\"p-0083\" num=\"0123\">Watermelon hybrid &#x2018;E26S.00171&#x2019; has '\n",
      " 'the following morphologic and other characteristics:</p>\\n'\n",
      " '<p id=\"p-0084\" num=\"0124\">General fruit type: Round-Oval</p>\\n'\n",
      " '<p id=\"p-0085\" num=\"0125\">Type of culture: In the open</p>\\n'\n",
      " '<p id=\"p-0086\" num=\"0126\">Ploidy: Triploid</p>\\n'\n",
      " '<p id=\"p-0087\" num=\"0127\">Plant:\\n'\n",
      " '<ul id=\"ul0005\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0005-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0006\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0006-0001\" num=\"0128\">Color: Green</li>\\n'\n",
      " '        <li id=\"ul0006-0002\" num=\"0129\">Pattern: Spots</li>\\n'\n",
      " '        <li id=\"ul0006-0003\" num=\"0130\">Color of pattern: Yellow</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0088\" num=\"0131\">Immature fruit:\\n'\n",
      " '<ul id=\"ul0007\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0007-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0008\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0008-0001\" num=\"0132\">Skin color: Lighter green</li>\\n'\n",
      " '        <li id=\"ul0008-0002\" num=\"0133\">Skin pattern: Spots sometimes show '\n",
      " 'at immature stage, and sometimes are not visible until the fruit is more '\n",
      " 'mature</li>\\n'\n",
      " '        <li id=\"ul0008-0003\" num=\"0134\">Color of skin pattern: Yellow</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0089\" num=\"0135\">Mature fruit:\\n'\n",
      " '<ul id=\"ul0009\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0009-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0010\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0010-0001\" num=\"0136\">Shape: Round</li>\\n'\n",
      " '        <li id=\"ul0010-0002\" num=\"0137\">Length: 27 cm</li>\\n'\n",
      " '        <li id=\"ul0010-0003\" num=\"0138\">Width: 25 cm</li>\\n'\n",
      " '        <li id=\"ul0010-0004\" num=\"0139\">Length/width ratio: 1.08</li>\\n'\n",
      " '        <li id=\"ul0010-0005\" num=\"0140\">Skin color: Peacock green</li>\\n'\n",
      " '        <li id=\"ul0010-0006\" num=\"0141\">Skin pattern: Spots</li>\\n'\n",
      " '        <li id=\"ul0010-0007\" num=\"0142\">Skin pattern color: Yellow</li>\\n'\n",
      " '        <li id=\"ul0010-0008\" num=\"0143\">Spot size: Majority are &#x3c;2 cm '\n",
      " 'in diameter; range from 0.2 cm to 15 cm in diameter</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0090\" num=\"0144\">Rind:\\n'\n",
      " '<ul id=\"ul0011\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0011-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0012\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0012-0001\" num=\"0145\">Thickness of sides: 1.2 cm</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0091\" num=\"0146\">Flesh:\\n'\n",
      " '<ul id=\"ul0013\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0013-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0014\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0014-0001\" num=\"0147\">Texture: Firm-crisp</li>\\n'\n",
      " '        <li id=\"ul0014-0002\" num=\"0148\">Color: RHS 46B (Vivid red)</li>\\n'\n",
      " '        <li id=\"ul0014-0003\" num=\"0149\">Color intensity: 7 (1-9 scale, where '\n",
      " '1 is less intense and 9 is most intense)</li>\\n'\n",
      " '        <li id=\"ul0014-0004\" num=\"0150\">Taste: 6 (1-9 scale, where 1 is less '\n",
      " 'sweet and 9 is most sweet)</li>\\n'\n",
      " '        <li id=\"ul0014-0005\" num=\"0151\">Hollow heart: Low</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0092\" num=\"0152\">Pips:\\n'\n",
      " '<ul id=\"ul0015\" list-style=\"none\">\\n'\n",
      " '    <li id=\"ul0015-0001\" num=\"0000\">\\n'\n",
      " '    <ul id=\"ul0016\" list-style=\"none\">\\n'\n",
      " '        <li id=\"ul0016-0001\" num=\"0153\">Size: Medium-large</li>\\n'\n",
      " '    </ul>\\n'\n",
      " '    </li>\\n'\n",
      " '</ul>\\n'\n",
      " '</p>\\n'\n",
      " '<heading id=\"h-0011\" level=\"2\">Comparisons to Other Watermelon '\n",
      " 'Varieties</heading>\\n'\n",
      " '<p id=\"p-0093\" num=\"0154\">Table 1 below compares some of the characteristics '\n",
      " 'of the triploid hybrid &#x2018;E26S.00171&#x2019; with the triploid hybrid '\n",
      " '&#x2018;Harvest Moon&#x2019; (U.S. Pat. No. 9,545,065). Column 1 lists the '\n",
      " 'characteristic, column 2 shows the characteristics for triploid hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;, and column 3 shows the characteristics for '\n",
      " 'comparison triploid hybrid &#x2018;Harvest Moon&#x2019;. Both '\n",
      " '&#x2018;E26S.00171&#x2019; and &#x2018;Harvest Moon&#x2019; have '\n",
      " 'medium-large pip size. Additional distinguishing characteristics can be '\n",
      " 'determined from the comparison of <figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>1</b>A-<b>1</b>I</figref> (&#x2018;E26S.00171&#x2019;) and <figref '\n",
      " 'idref=\"DRAWINGS\">FIG. <b>2</b></figref> (&#x2018;Harvest Moon&#x2019;).</p>\\n'\n",
      " '<p id=\"p-0094\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00001\" num=\"00001\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 1</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Comparison of &#x2018;E26S.00171&#x2019; and &#x2018;Harvest '\n",
      " 'Moon&#x2019;.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"3\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"91pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"56pt\" align=\"center\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"70pt\" align=\"center\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Characteristic</entry>\\n'\n",
      " '<entry>&#x2018;E26S.00171&#x2019;</entry>\\n'\n",
      " '<entry>&#x2018;Harvest Moon&#x2019;</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Fruit length</entry>\\n'\n",
      " '<entry>27 cm</entry>\\n'\n",
      " '<entry>33 cm</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Fruit width</entry>\\n'\n",
      " '<entry>25 cm</entry>\\n'\n",
      " '<entry>28 cm</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Length/width ratio</entry>\\n'\n",
      " '<entry>1.08</entry>\\n'\n",
      " '<entry>1.17</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Rind thickness</entry>\\n'\n",
      " '<entry>1.2 cm</entry>\\n'\n",
      " '<entry>1.3 cm</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Fruit flesh color</entry>\\n'\n",
      " '<entry>RHS 46B (Vivid</entry>\\n'\n",
      " '<entry>RHS 53C (Strong</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>red)</entry>\\n'\n",
      " '<entry>red)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Fruit flesh color intensity<sup>1</sup></entry>\\n'\n",
      " '<entry>7</entry>\\n'\n",
      " '<entry>5</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Fruit flesh taste<sup>2</sup></entry>\\n'\n",
      " '<entry>6</entry>\\n'\n",
      " '<entry>5</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Fruit flesh firmness</entry>\\n'\n",
      " '<entry>Firm-crisp</entry>\\n'\n",
      " '<entry>Crisp</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Hollow heart</entry>\\n'\n",
      " '<entry>Low</entry>\\n'\n",
      " '<entry>Medium</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"3\" align=\"left\" id=\"FOO-00001\"><sup>1</sup>1-9 '\n",
      " 'scale, where 1 is less intense and 9 is most intense</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"3\" align=\"left\" id=\"FOO-00002\"><sup>2</sup>1-9 '\n",
      " 'scale where 1 is less sweet and 9 is most sweet</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<heading id=\"h-0012\" level=\"1\">Further Embodiments</heading>\\n'\n",
      " '<p id=\"p-0095\" num=\"0155\">In one aspect, the present invention is directed '\n",
      " 'to methods of producing a homozygous-spotted diploid watermelon line (HSDL), '\n",
      " 'including: (a) crossing a first diploid watermelon plant with a spotted '\n",
      " 'trait with a second diploid watermelon plant without the spotted trait to '\n",
      " 'produce progeny watermelon seed, wherein the presence of the spotted trait '\n",
      " 'is determined by visually phenotyping for the spotted trait or by genotyping '\n",
      " 'to identify a presence of a marker linked to a spotted allele, wherein the '\n",
      " 'spotted allele is located between position 134886 and position 86340 on '\n",
      " 'Chromosome 4 of a &#x2018;Charleston Gray&#x2019; genome; (b) crossing a '\n",
      " 'watermelon plant grown from the progeny watermelon seed with itself or a '\n",
      " 'second watermelon plant to produce additional progeny watermelon seed; (c) '\n",
      " 'planting the additional progeny watermelon seed of step (b) to produce '\n",
      " 'additional watermelon plants; (d) selecting watermelon plants with the '\n",
      " 'spotted trait from the additional watermelon plants of step (c); (e) '\n",
      " 'repeating the crossing of step (b), the planting of step (c), and the '\n",
      " 'selecting of step (d) for an additional 3-10 generations to produce a '\n",
      " 'homozygous-spotted diploid watermelon line (HSDL), wherein the HSDL is 100% '\n",
      " 'spotted (i.e., homozygous for the spotted allele). In one embodiment, the '\n",
      " 'first diploid plant of step (a) is homozygous for the spotted allele or '\n",
      " 'heterozygous for the spotted allele. Another embodiment, which may be '\n",
      " 'combined with any of the preceding embodiments, further includes selecting '\n",
      " 'one or more additional traits of interest at step (c), wherein the one or '\n",
      " 'more additional traits of interest are selected from the group of a fruit '\n",
      " 'flesh color trait, a fruit flesh firmness trait, or a fruit taste trait. In '\n",
      " 'a further embodiment, visual phenotyping includes looking at the leaves '\n",
      " 'and/or fruit of the plant and scoring for presence or absence of spots. In '\n",
      " 'another embodiment, the marker includes a first marker including a T at '\n",
      " 'position 101 of SEQ ID NO: 1 and/or a second marker including a G at '\n",
      " 'position 101 of SEQ ID NO: 6. In a further embodiment, genotyping of the '\n",
      " 'first marker includes the use of a forward primer including SEQ ID NO: 3, a '\n",
      " 'reverse primer including SEQ ID NO: 4, and/or a probe including SEQ ID NO: '\n",
      " '5, and genotyping of the second marker includes the use of a forward primer '\n",
      " 'including SEQ ID NO: 8, a reverse primer including SEQ ID NO: 9, and/or a '\n",
      " 'probe including SEQ ID NO: 10. The marker may be used to determine the '\n",
      " 'percentage of the spotted allele, as described in Example 1.</p>\\n'\n",
      " '<p id=\"p-0096\" num=\"0156\">Some embodiments of the present invention are '\n",
      " 'directed to the HSDL plant produced by the method of any of the preceding '\n",
      " 'embodiments. The HSDL plant of the present invention may have the additional '\n",
      " 'traits of fruit flesh color (e.g., a dark red fruit flesh color), fruit '\n",
      " 'flesh firmness (e.g., medium-firm fruit flesh), and fruit taste (e.g., a '\n",
      " 'sweet taste). Watermelon plant parts include watermelon leaves, ovules, '\n",
      " 'pollen, seeds, watermelon fruits, parts of watermelon fruits, flowers, '\n",
      " 'cells, and the like. In another embodiment, the present invention is further '\n",
      " 'directed to watermelon leaves, ovules, pollen, seeds, watermelon fruits, '\n",
      " 'parts of watermelon fruits, and/or flowers isolated from HSDL watermelon '\n",
      " 'plants. In another embodiment, the present invention is further directed to '\n",
      " 'protoplasts produced from HSDL watermelon plants. In another embodiment, the '\n",
      " 'present invention is further directed to tissue culture of HSDL watermelon '\n",
      " 'plants, and to watermelon plants regenerated from the tissue culture, where '\n",
      " 'the plant has all of the morphological and physiological characteristics of '\n",
      " 'HSDL watermelon. In certain embodiments, tissue culture of HSDL watermelon '\n",
      " 'plants is produced from a plant part selected from leaf, anther, pistil, '\n",
      " 'stem, petiole, root, root tip, fruit, seed, flower, cotyledon, hypocotyl, '\n",
      " 'embryo, and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0097\" num=\"0157\">Further embodiments of the present invention are '\n",
      " 'directed to methods of producing a homozygous-spotted tetraploid watermelon '\n",
      " 'line (HSTL), including: (a) doubling a chromosome number of the HSDL plant '\n",
      " 'of claim <b>5</b>, thereby producing a homozygous-spotted tetraploid '\n",
      " 'watermelon line (HSTL), wherein the HSTL is 100% spotted (i.e., homozygous '\n",
      " 'for the spotted allele). The HSTL plant of the present invention may have '\n",
      " 'the additional traits of fruit flesh color (e.g., a dark red fruit flesh '\n",
      " 'color), fruit flesh firmness (e.g., medium-firm fruit flesh), and fruit '\n",
      " 'taste (e.g., a sweet taste). Watermelon plant parts include watermelon '\n",
      " 'leaves, ovules, pollen, seeds, watermelon fruits, parts of watermelon '\n",
      " 'fruits, flowers, cells, and the like. In another embodiment, the present '\n",
      " 'invention is further directed to watermelon leaves, ovules, pollen, seeds, '\n",
      " 'watermelon fruits, parts of watermelon fruits, and/or flowers isolated from '\n",
      " 'HSTL watermelon plants. In another embodiment, the present invention is '\n",
      " 'further directed to protoplasts produced from HSTL watermelon plants. In '\n",
      " 'another embodiment, the present invention is further directed to tissue '\n",
      " 'culture of HSTL watermelon plants, and to watermelon plants regenerated from '\n",
      " 'the tissue culture, where the plant has all of the morphological and '\n",
      " 'physiological characteristics of HSTL watermelon. In certain embodiments, '\n",
      " 'tissue culture of HSTL watermelon plants is produced from a plant part '\n",
      " 'selected from leaf, anther, pistil, stem, petiole, root, root tip, fruit, '\n",
      " 'seed, flower, cotyledon, hypocotyl, embryo, and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0098\" num=\"0158\">Additional embodiments of the present invention '\n",
      " 'are directed to methods of producing a 33% spotted triploid hybrid '\n",
      " 'watermelon plant, including: (a) crossing the HSDL plant of claim <b>5</b> '\n",
      " 'onto a 0% spotted tetraploid inbred plant, wherein the HSDL plant is a male '\n",
      " 'parent and the tetraploid inbred plant is a female parent, to produce a 33% '\n",
      " 'spotted triploid hybrid watermelon plant. Some embodiments of the present '\n",
      " 'invention are directed to the 33% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of the preceding embodiment. The 33% spotted triploid '\n",
      " 'hybrid watermelon plant of the present invention may have the additional '\n",
      " 'traits of fruit flesh color (e.g., a dark red fruit flesh color), fruit '\n",
      " 'flesh firmness (e.g., medium-firm fruit flesh), and fruit taste (e.g., a '\n",
      " 'sweet taste). Watermelon plant parts include watermelon leaves, ovules, '\n",
      " 'pollen, seeds, watermelon fruits, parts of watermelon fruits, flowers, '\n",
      " 'cells, and the like. In another embodiment, the present invention is further '\n",
      " 'directed to watermelon leaves, ovules, pollen, seeds, watermelon fruits, '\n",
      " 'parts of watermelon fruits, and/or flowers isolated from 33% spotted '\n",
      " 'triploid hybrid watermelon plants. In another embodiment, the present '\n",
      " 'invention is further directed to protoplasts produced from 33% spotted '\n",
      " 'triploid hybrid watermelon plants. In another embodiment, the present '\n",
      " 'invention is further directed to tissue culture of 33% spotted triploid '\n",
      " 'hybrid watermelon plants, and to watermelon plants regenerated from the '\n",
      " 'tissue culture, where the plant has all of the morphological and '\n",
      " 'physiological characteristics of 33% spotted triploid hybrid watermelon '\n",
      " 'plants. In certain embodiments, tissue culture of 33% spotted triploid '\n",
      " 'hybrid watermelon plants is produced from a plant part selected from leaf, '\n",
      " 'anther, pistil, stem, petiole, root, root tip, fruit, seed, flower, '\n",
      " 'cotyledon, hypocotyl, embryo, and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0099\" num=\"0159\">Yet further embodiments of the present invention '\n",
      " 'are directed to methods of producing a 50% or 100% spotted diploid hybrid '\n",
      " 'watermelon plant, including: (a) crossing the HSDL plant of claim <b>5</b> '\n",
      " 'onto a 0% spotted diploid inbred plant or onto a second HSDL plant of claim '\n",
      " '<b>5</b>, thereby producing a 50% or 100% spotted diploid hybrid watermelon '\n",
      " 'plant. Some embodiments of the present invention are directed to the 50% or '\n",
      " '100% spotted diploid hybrid watermelon plant produced by the method of the '\n",
      " 'preceding embodiment. The 50% or 100% spotted diploid hybrid watermelon '\n",
      " 'plant of the present invention may have the additional traits of fruit flesh '\n",
      " 'color (e.g., a dark red fruit flesh color), fruit flesh firmness (e.g., '\n",
      " 'medium-firm fruit flesh), and fruit taste (e.g., a sweet taste). Watermelon '\n",
      " 'plant parts include watermelon leaves, ovules, pollen, seeds, watermelon '\n",
      " 'fruits, parts of watermelon fruits, flowers, cells, and the like. In another '\n",
      " 'embodiment, the present invention is further directed to watermelon leaves, '\n",
      " 'ovules, pollen, seeds, watermelon fruits, parts of watermelon fruits, and/or '\n",
      " 'flowers isolated from 50% or 100% spotted diploid hybrid watermelon plants. '\n",
      " 'In another embodiment, the present invention is further directed to '\n",
      " 'protoplasts produced from 50% or 100% spotted diploid hybrid watermelon '\n",
      " 'plants. In another embodiment, the present invention is further directed to '\n",
      " 'tissue culture of 50% or 100% spotted diploid hybrid watermelon plants, and '\n",
      " 'to watermelon plants regenerated from the tissue culture, where the plant '\n",
      " 'has all of the morphological and physiological characteristics of 50% or '\n",
      " '100% spotted diploid hybrid watermelon plants. In certain embodiments, '\n",
      " 'tissue culture of 50% or 100% spotted diploid hybrid watermelon plants is '\n",
      " 'produced from a plant part selected from leaf, anther, pistil, stem, '\n",
      " 'petiole, root, root tip, fruit, seed, flower, cotyledon, hypocotyl, embryo, '\n",
      " 'and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0100\" num=\"0160\">In another aspect, the present invention is '\n",
      " 'directed to methods of producing a homozygous-spotted tetraploid watermelon '\n",
      " 'line (HSTL), including: (a) providing a diploid watermelon plant with a '\n",
      " 'spotted trait, wherein the presence of the spotted trait is determined by '\n",
      " 'visually phenotyping for the spotted trait or by genotyping to identify a '\n",
      " 'presence of a marker linked to a spotted allele, and wherein the spotted '\n",
      " 'allele is located between position 134886 and position 86340 on Chromosome 4 '\n",
      " 'of a &#x2018;Charleston Gray&#x2019; genome; (b) doubling a chromosome '\n",
      " 'number of the diploid watermelon plant to produce a tetraploid watermelon '\n",
      " 'plant with the spotted trait; (c) crossing the tetraploid watermelon plant '\n",
      " 'with a non-spotted watermelon plant to produce a F1 generation of watermelon '\n",
      " 'plants; (d) self-pollinating or open pollinating the F1 generation of '\n",
      " 'watermelon plants to produce a F2 generation of watermelon plants; (e) '\n",
      " 'phenotyping progeny watermelon plants from the F2 generation to identify the '\n",
      " 'presence of the spotted trait and/or genotyping progeny watermelon plants '\n",
      " 'from the F2 generation to identify the presence of the marker linked to the '\n",
      " 'spotted allele and using the marker to determine a dosage of the spotted '\n",
      " 'allele; (f) selecting progeny watermelon plants with the highest dosage of '\n",
      " 'the spotted allele; (g) repeating the self-pollinating of step (d), the '\n",
      " 'genotyping of step (e), and the selecting of step (f) for an additional 3-10 '\n",
      " 'generations to produce a homozygous-spotted tetraploid watermelon line '\n",
      " '(HSTL), wherein the HSTL is 100% spotted (i.e., homozygous for the spotted '\n",
      " 'allele). In one embodiment, the tetraploid plant of step (b) is homozygous '\n",
      " 'for the spotted allele or heterozygous for the spotted allele. In a further '\n",
      " 'embodiment, visual phenotyping or phenotyping includes looking at the leaves '\n",
      " 'and/or fruit of the plant and scoring for presence or absence of spots. In '\n",
      " 'another embodiment, the marker includes a first marker including a T at '\n",
      " 'position 101 of SEQ ID NO: 1 and/or a second marker including a G at '\n",
      " 'position 101 of SEQ ID NO: 6. In a further embodiment, genotyping of the '\n",
      " 'first marker includes the use of a forward primer including SEQ ID NO: 3, a '\n",
      " 'reverse primer including SEQ ID NO: 4, and/or a probe including SEQ ID NO: '\n",
      " '5, and wherein genotyping of the second marker includes the use of a forward '\n",
      " 'primer including SEQ ID NO: 8, a reverse primer including SEQ ID NO: 9, '\n",
      " 'and/or a probe including SEQ ID NO: 10. The marker may be used to determine '\n",
      " 'the percentage of the spotted allele, as described in Example 1.</p>\\n'\n",
      " '<p id=\"p-0101\" num=\"0161\">Some embodiments of the present invention are '\n",
      " 'directed to the HSTL plant produced by the method of any of the preceding '\n",
      " 'embodiments. The HSTL plant of the present invention may have the additional '\n",
      " 'traits of fruit flesh color (e.g., a dark red fruit flesh color), fruit '\n",
      " 'flesh firmness (e.g., medium-firm fruit flesh), and fruit taste (e.g., a '\n",
      " 'sweet taste). Watermelon plant parts include watermelon leaves, ovules, '\n",
      " 'pollen, seeds, watermelon fruits, parts of watermelon fruits, flowers, '\n",
      " 'cells, and the like. In another embodiment, the present invention is further '\n",
      " 'directed to watermelon leaves, ovules, pollen, seeds, watermelon fruits, '\n",
      " 'parts of watermelon fruits, and/or flowers isolated from HSTL watermelon '\n",
      " 'plants. In another embodiment, the present invention is further directed to '\n",
      " 'protoplasts produced from HSTL watermelon plants. In another embodiment, the '\n",
      " 'present invention is further directed to tissue culture of HSTL watermelon '\n",
      " 'plants, and to watermelon plants regenerated from the tissue culture, where '\n",
      " 'the plant has all of the morphological and physiological characteristics of '\n",
      " 'HSTL watermelon. In certain embodiments, tissue culture of HSTL watermelon '\n",
      " 'plants is produced from a plant part selected from leaf, anther, pistil, '\n",
      " 'stem, petiole, root, root tip, fruit, seed, flower, cotyledon, hypocotyl, '\n",
      " 'embryo, and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0102\" num=\"0162\">Additional embodiments of the present invention '\n",
      " 'are directed to methods of producing a 66% spotted triploid hybrid '\n",
      " 'watermelon plant, including: (a) crossing the HSTL plant of claim <b>13</b> '\n",
      " 'onto a 0% spotted diploid inbred plant, wherein the HSTL plant is a female '\n",
      " 'parent and the diploid inbred plant is a male parent, to produce a 66% '\n",
      " 'spotted triploid hybrid watermelon plant. Some embodiments of the present '\n",
      " 'invention are directed to the 66% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of the preceding embodiment. The 66% spotted triploid '\n",
      " 'hybrid watermelon plant of the present invention may have the additional '\n",
      " 'traits of fruit flesh color (e.g., a dark red fruit flesh color), fruit '\n",
      " 'flesh firmness (e.g., medium-firm fruit flesh), and fruit taste (e.g., a '\n",
      " 'sweet taste). Watermelon plant parts include watermelon leaves, ovules, '\n",
      " 'pollen, seeds, watermelon fruits, parts of watermelon fruits, flowers, '\n",
      " 'cells, and the like. In another embodiment, the present invention is further '\n",
      " 'directed to watermelon leaves, ovules, pollen, seeds, watermelon fruits, '\n",
      " 'parts of watermelon fruits, and/or flowers isolated from 66% spotted '\n",
      " 'triploid hybrid watermelon plants. In another embodiment, the present '\n",
      " 'invention is further directed to protoplasts produced from 66% spotted '\n",
      " 'triploid hybrid watermelon plants. In another embodiment, the present '\n",
      " 'invention is further directed to tissue culture of 66% spotted triploid '\n",
      " 'hybrid watermelon plants, and to watermelon plants regenerated from the '\n",
      " 'tissue culture, where the plant has all of the morphological and '\n",
      " 'physiological characteristics of 66% spotted triploid hybrid watermelon '\n",
      " 'plants. In certain embodiments, tissue culture of 66% spotted triploid '\n",
      " 'hybrid watermelon plants is produced from a plant part selected from leaf, '\n",
      " 'anther, pistil, stem, petiole, root, root tip, fruit, seed, flower, '\n",
      " 'cotyledon, hypocotyl, embryo, and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0103\" num=\"0163\">Further embodiments of the present invention are '\n",
      " 'directed to methods of producing a 100% spotted triploid hybrid watermelon '\n",
      " 'plant, including: (a) crossing the HSDL plant of claim <b>5</b> with the '\n",
      " 'HSTL plant of claim <b>13</b> to produce a 100% spotted triploid hybrid '\n",
      " 'watermelon plant. Some embodiments of the present invention are directed to '\n",
      " 'the 100% spotted triploid hybrid watermelon plant produced by the method of '\n",
      " 'the preceding embodiment. The 100% spotted triploid hybrid watermelon plant '\n",
      " 'of the present invention may have the additional traits of fruit flesh color '\n",
      " '(e.g., a dark red fruit flesh color), fruit flesh firmness (e.g., '\n",
      " 'medium-firm fruit flesh), and fruit taste (e.g., a sweet taste). Watermelon '\n",
      " 'plant parts include watermelon leaves, ovules, pollen, seeds, watermelon '\n",
      " 'fruits, parts of watermelon fruits, flowers, cells, and the like. In another '\n",
      " 'embodiment, the present invention is further directed to watermelon leaves, '\n",
      " 'ovules, pollen, seeds, watermelon fruits, parts of watermelon fruits, and/or '\n",
      " 'flowers isolated from 100% spotted triploid hybrid watermelon plants. In '\n",
      " 'another embodiment, the present invention is further directed to protoplasts '\n",
      " 'produced from 100% spotted triploid hybrid watermelon plants. In another '\n",
      " 'embodiment, the present invention is further directed to tissue culture of '\n",
      " '100% spotted triploid hybrid watermelon plants, and to watermelon plants '\n",
      " 'regenerated from the tissue culture, where the plant has all of the '\n",
      " 'morphological and physiological characteristics of 100% spotted triploid '\n",
      " 'hybrid watermelon plants. In certain embodiments, tissue culture of 100% '\n",
      " 'spotted triploid hybrid watermelon plants is produced from a plant part '\n",
      " 'selected from leaf, anther, pistil, stem, petiole, root, root tip, fruit, '\n",
      " 'seed, flower, cotyledon, hypocotyl, embryo, and meristematic cell.</p>\\n'\n",
      " '<p id=\"p-0104\" num=\"0164\">In yet another aspect, the present invention is '\n",
      " 'directed to methods of selecting a spotted watermelon plant, including: (a) '\n",
      " 'crossing a spotted watermelon plant to a watermelon plant to produce a '\n",
      " 'population of progeny watermelon plants; (b) genotyping the population of '\n",
      " 'progeny watermelon plants for a presence of a marker linked to a spotted '\n",
      " 'allele, wherein the spotted allele is located between position 134886 and '\n",
      " 'position 86340 on Chromosome 4 of a &#x2018;Charleston Gray&#x2019; genome; '\n",
      " 'and (c) selecting a progeny plant on the basis of the presence of the one or '\n",
      " 'more sequences genotyped in (b), wherein the progeny watermelon plant is '\n",
      " 'spotted. In one embodiment, the marker includes a first marker including a T '\n",
      " 'at position 101 of SEQ ID NO: 1 (or a T at position 61 of SEQ ID NO: 2) '\n",
      " 'and/or a second marker including a G at position 101 of SEQ ID NO: 6 (or a G '\n",
      " 'at position 61 of SEQ ID NO: 7), and wherein genotyping of the first marker '\n",
      " 'includes the use of a forward primer including SEQ ID NO: 3, a reverse '\n",
      " 'primer including SEQ ID NO: 4, and/or a probe including SEQ ID NO: 5, and '\n",
      " 'wherein genotyping of the second marker includes the use of a forward primer '\n",
      " 'including SEQ ID NO: 8, a reverse primer including SEQ ID NO: 9, and/or a '\n",
      " 'probe including SEQ ID NO: 10. The marker may be used to determine the '\n",
      " 'percentage of the spotted allele, as described in Example 1.</p>\\n'\n",
      " '<p id=\"p-0105\" num=\"0165\">In still another aspect, the present invention is '\n",
      " 'directed to methods of selecting a spotted watermelon plant, including: (a) '\n",
      " 'providing a test spotted watermelon plant and a control spotted watermelon '\n",
      " 'plant; (b) using a PCR-based technique to assess the presence of one or more '\n",
      " 'sequences selected from the group of a sequence including T at position 101 '\n",
      " 'of SEQ ID NO: 1 (or a T at position 61 of SEQ ID NO: 2) or a sequence '\n",
      " 'including a G at position 101 of SEQ ID NO: 6 (or a G at position 61 of SEQ '\n",
      " 'ID NO: 7); and (c) determining the percentage of the spotted allele in the '\n",
      " 'test spotted watermelon plant based on a comparison between the results of '\n",
      " 'step (b) for the test spotted watermelon plant and the control spotted '\n",
      " 'watermelon plant. In one embodiment, the PCR-based technique of step (b) '\n",
      " 'includes the use of primers and probes selected from the group of SEQ ID NO: '\n",
      " '3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 8, SEQ ID NO: 9, or SEQ ID NO: 10. '\n",
      " 'The PCR-based technique may be any known PCR-based technique. When high '\n",
      " 'resolution melting (HRM) is used, the curve obtained from the test spotted '\n",
      " 'watermelon plant is compared to the curve obtained from the control spotted '\n",
      " 'watermelon plant (e.g., a diploid F1 with 50% spotting). Further, the order '\n",
      " 'of curves shown in <figref idref=\"DRAWINGS\">FIG. <b>4</b></figref> may be '\n",
      " 'used to determine the percentage of the spotted allele in the test spotted '\n",
      " 'watermelon plant. In a further embodiment, the PCR-based technique of step '\n",
      " '(b) is replaced with a sequencing-based technique.</p>\\n'\n",
      " '<p id=\"p-0106\" num=\"0166\">Occasionally, triploid watermelon varieties '\n",
      " 'produce true and viable seed at a very low frequency. This is due to the '\n",
      " 'random segregation of chromosomes, which can result in chromosomes '\n",
      " 'occasionally ending up distributed back to normal in the ovule and pollen '\n",
      " 'such that they are balanced enough to be viable. Thus, occasional viable '\n",
      " 'pollen from a triploid variety can be &#x201c;bridged&#x201d;, or crossed, '\n",
      " 'onto a normal diploid or tetraploid line to make viable hybrid seed. '\n",
      " 'Similarly, occasional viable ovules within a triploid can be bridged or '\n",
      " 'crossed onto, using pollen from a normal tetraploid or diploid line to '\n",
      " 'create viable seed. Also occasionally, a triploid within and of itself can '\n",
      " 'make a viable seed that can make a fertile plant. This occasional fertility '\n",
      " 'of triploid varieties is well-known by watermelon breeders, and can be taken '\n",
      " 'advantage of to incorporate triploid varieties with desirable '\n",
      " 'characteristics into breeding programs. Watermelon variety '\n",
      " '&#x2018;E26S.00171&#x2019;, like other triploid varieties, occasionally '\n",
      " 'produces flowers with viable pollen or viable ovules. Similarly, watermelon '\n",
      " 'variety &#x2018;E26S.00171&#x2019; occasionally produces fruit containing '\n",
      " 'viable, fertile seed. An additional aspect of this disclosure is directed to '\n",
      " 'methods for producing offspring of watermelon variety '\n",
      " '&#x2018;E26S.00171&#x2019; (e.g., using viable pollen, viable ovules, or '\n",
      " 'viable seed occasionally produced by &#x2018;E26S.00171&#x2019;).</p>\\n'\n",
      " '<p id=\"p-0107\" num=\"0167\">As used herein, the term plant includes plant '\n",
      " 'cells, plant protoplasts, plant cell tissue cultures from which watermelon '\n",
      " 'plants can be regenerated, plant calli, plant clumps and plant cells that '\n",
      " 'are intact in plants or parts of plants, such as embryos, pollen (pollen '\n",
      " 'grains), ovules, flowers, leaves, stems, and the like.</p>\\n'\n",
      " '<heading id=\"h-0013\" level=\"2\">Genetic Marker Profile</heading>\\n'\n",
      " '<p id=\"p-0108\" num=\"0168\">In addition to phenotypic observations, a plant '\n",
      " 'can also be identified by its genotype. The genotype of a plant can be '\n",
      " 'characterized through a genetic marker profile which can identify plants of '\n",
      " 'the same or a related variety or line, or can be used to determine or '\n",
      " 'validate a pedigree. Genetic marker profiles can be obtained by techniques '\n",
      " 'such as Restriction Fragment Length Polymorphisms (RFLPs), Randomly '\n",
      " 'Amplified Polymorphic DNAs (RAPDs), Arbitrarily Primed Polymerase Chain '\n",
      " 'Reaction (AP-PCR), DNA Amplification Fingerprinting (DAF), Sequence '\n",
      " 'Characterized Amplified Regions (SCARs), Amplified Fragment Length '\n",
      " 'Polymorphisms (AFLPs), Simple Sequence Repeats (SSRs) which are also '\n",
      " 'referred to as Microsatellites, and Single Nucleotide Polymorphisms (SNPs). '\n",
      " 'For example, see Cregan et al., &#x201c;An Integrated Genetic Linkage Map of '\n",
      " 'the Soybean Genome&#x201d; Crop Science 39:1464-1490 (1999), and Berry et '\n",
      " 'al., &#x201c;Assessing Probability of Ancestry Using Simple Sequence Repeat '\n",
      " 'Profiles: Applications to Maize Inbred Lines and Soybean Varieties&#x201d; '\n",
      " 'Genetics 165:331-342 (2003).</p>\\n'\n",
      " '<p id=\"p-0109\" num=\"0169\">Particular markers used for these purposes are not '\n",
      " 'limited to any particular set of markers, but are envisioned to include any '\n",
      " 'type of marker and marker profile which provides a means of distinguishing '\n",
      " 'varieties.</p>\\n'\n",
      " '<p id=\"p-0110\" num=\"0170\">Differentially-inherited nucleic acid '\n",
      " 'polymorphisms that segregate among progeny may be used as molecular markers. '\n",
      " 'The polymorphisms can be of any origin, including insertions, deletions, '\n",
      " 'duplications, repetitive elements, point mutations, recombination events, '\n",
      " 'and transposable elements. Means of characterizing genetic marker profiles '\n",
      " 'using any of the above techniques are well known in the art.</p>\\n'\n",
      " '<p id=\"p-0111\" num=\"0171\">The present invention includes markers for a '\n",
      " 'spotted trait. Further provided by the invention is a watermelon plant '\n",
      " 'characterized by molecular and physiological data obtained from the '\n",
      " 'representative sample of said line deposited with the National Collection of '\n",
      " 'Industrial, Food and Marine Bacteria Ltd. (NCIMB Ltd.). In addition, the '\n",
      " 'present invention includes a watermelon plant formed by the combination of '\n",
      " 'the disclosed watermelon plant or plant cell with another watermelon plant '\n",
      " 'or cell and including the homozygous alleles of the line.</p>\\n'\n",
      " '<heading id=\"h-0014\" level=\"2\">Single-Gene Conversions</heading>\\n'\n",
      " '<p id=\"p-0112\" num=\"0172\">When the terms watermelon plant, cultivar, hybrid, '\n",
      " 'or watermelon line are used in the context of the present invention, this '\n",
      " 'also includes any single gene conversions of that line. The term '\n",
      " '&#x201c;single gene converted plant&#x201d; as used herein refers to those '\n",
      " 'watermelon plants which are developed by a plant breeding technique called '\n",
      " 'backcrossing wherein essentially all of the desired morphological and '\n",
      " 'physiological characteristics of a cultivar are recovered in addition to the '\n",
      " 'single gene transferred into the line via the backcrossing technique. '\n",
      " 'Backcrossing methods can be used with the present invention to improve or '\n",
      " 'introduce a characteristic into the line. The term '\n",
      " '&#x201c;backcrossing&#x201d; as used herein refers to the repeated crossing '\n",
      " 'of a hybrid progeny back to one of the parental watermelon plants for that '\n",
      " 'line, backcrossing 1, 2, 3, 4, 5, 6, 7, 8, or more times to the recurrent '\n",
      " 'parent. The parental watermelon plant which contributes the gene for the '\n",
      " 'desired characteristic is termed the nonrecurrent or donor parent. This '\n",
      " 'terminology refers to the fact that the nonrecurrent parent is used one time '\n",
      " 'in the backcross protocol and therefore does not recur. The parental '\n",
      " 'watermelon plant to which the gene or genes from the nonrecurrent parent are '\n",
      " 'transferred is known as the recurrent parent as it is used for several '\n",
      " 'rounds in the backcrossing protocol (Poehlman &#x26; Sleper, 1994; Fehr, '\n",
      " '1987). In a typical backcross protocol, the original cultivar of interest '\n",
      " '(recurrent parent) is crossed to a second line (nonrecurrent parent) that '\n",
      " 'carries the single gene of interest to be transferred. The resulting progeny '\n",
      " 'from this cross are then crossed again to the recurrent parent and the '\n",
      " 'process is repeated until a watermelon plant is obtained wherein essentially '\n",
      " 'all of the desired morphological and physiological characteristics of the '\n",
      " 'recurrent parent are recovered in the converted plant, in addition to the '\n",
      " 'single transferred gene from the nonrecurrent parent.</p>\\n'\n",
      " '<p id=\"p-0113\" num=\"0173\">The selection of a suitable recurrent parent is an '\n",
      " 'important step for a successful backcrossing procedure. The goal of a '\n",
      " 'backcross protocol is to alter or substitute a single trait or '\n",
      " 'characteristic in the original line. To accomplish this, a single gene of '\n",
      " 'the recurrent cultivar is modified or substituted with the desired gene from '\n",
      " 'the nonrecurrent parent, while retaining essentially all of the rest of the '\n",
      " 'desired genetic, and therefore the desired physiological and morphological, '\n",
      " 'constitution of the original line. The choice of the particular nonrecurrent '\n",
      " 'parent will depend on the purpose of the backcross; one of the major '\n",
      " 'purposes is to add some commercially desirable, agronomically important '\n",
      " 'trait to the plant. The exact backcrossing protocol will depend on the '\n",
      " 'characteristic or trait being altered to determine an appropriate testing '\n",
      " 'protocol. Although backcrossing methods are simplified when the '\n",
      " 'characteristic being transferred is a dominant allele, a recessive allele '\n",
      " 'may also be transferred. In this instance it may be necessary to introduce a '\n",
      " 'test of the progeny to determine if the desired characteristic has been '\n",
      " 'successfully transferred.</p>\\n'\n",
      " '<p id=\"p-0114\" num=\"0174\">Many single gene traits have been identified that '\n",
      " 'are not regularly selected for in the development of a new line but that can '\n",
      " 'be improved by backcrossing techniques. Single gene traits may or may not be '\n",
      " 'transgenic, and examples of these traits include but are not limited to, '\n",
      " 'male sterility, modified fatty acid metabolism, modified carbohydrate '\n",
      " 'metabolism, herbicide resistance, resistance for bacterial, fungal, or viral '\n",
      " 'disease, insect resistance, enhanced nutritional quality, industrial usage, '\n",
      " 'yield stability, and yield enhancement. These genes are generally inherited '\n",
      " 'through the nucleus. Several of these single gene traits are described in '\n",
      " 'U.S. Pat. Nos. 5,777,196, 5,948,957 and 5,969,212, the disclosures of which '\n",
      " 'are specifically hereby incorporated by reference.</p>\\n'\n",
      " '<heading id=\"h-0015\" level=\"2\">Tissue Culture</heading>\\n'\n",
      " '<p id=\"p-0115\" num=\"0175\">As used herein, the term &#x201c;tissue '\n",
      " 'culture&#x201d; indicates a composition including isolated cells of the same '\n",
      " 'or a different type or a collection of such cells organized into parts of a '\n",
      " 'plant. Exemplary types of tissue cultures are protoplasts, calli, '\n",
      " 'meristematic cells, and plant cells that can generate tissue culture that '\n",
      " 'are intact in plants or parts of plants, such as leaves, pollen, embryos, '\n",
      " 'roots, root tips, anthers, pistils, flowers, seeds, petioles, and the like. '\n",
      " 'Means for preparing and maintaining plant tissue culture are well known in '\n",
      " 'the art. The use of tissue culture to propagate watermelon plants is '\n",
      " 'exemplified in Adelberg, J. W., B. B. Rhodes, <i>Micropropagation from '\n",
      " 'zygotic tissue of watermelon</i>, C. E. Thomas (ed.) Proc. of the '\n",
      " 'Cucurbitaceae 89<i>: Evaluation and enhancement of cucurbit germplasm</i>, '\n",
      " 'Charleston S.C., USA; and Zhang et al., <i>Shoot regeneration from immature '\n",
      " 'cotyledon of watermelon</i>, Cucurbit Genetics Coop. 17:111-115 (1994).</p>\\n'\n",
      " '<p id=\"p-0116\" num=\"0176\">Transformation of plant protoplasts also can be '\n",
      " 'achieved using methods based on calcium phosphate precipitation, '\n",
      " 'polyethylene glycol treatment, electroporation, and combinations of these '\n",
      " 'treatments (see, e.g., Potrykus et al., 1985; Omirulleh et al., 1993; Fromm '\n",
      " 'et al., 1986; Uchimiya et al., 1986; Marcotte et al., 1988). Transformation '\n",
      " 'of plants and expression of foreign genetic elements is exemplified in Choi '\n",
      " 'et al. (1994) and Ellul et al. (2003). Direct uptake of DNA into protoplasts '\n",
      " 'using CaCl2 precipitation, polyvinyl alcohol, or poly-L-ornithine has also '\n",
      " 'been reported (see, e.g., Hain et al., 1985 and Draper et al., 1982). '\n",
      " 'Electroporation of protoplasts and whole cells and tissues have also been '\n",
      " \"described (see, e.g., Saker et al., 1998; Donn et al., 1990; D'Halluin et \"\n",
      " 'al., 1992; and Laursen et al., 1994; Chupean et al., 1989).</p>\\n'\n",
      " '<heading id=\"h-0016\" level=\"2\">Additional Breeding Methods</heading>\\n'\n",
      " '<p id=\"p-0117\" num=\"0177\">This invention also is directed to methods for '\n",
      " 'producing a diploid, triploid, and tetraploid watermelon plants with spotted '\n",
      " 'phenotypes and commercially desirable characteristics, and using these '\n",
      " 'plants in breeding. In this manner, a diverse array of watermelon patterns '\n",
      " '(e.g., fruit rind, leaf, etc.), watermelon flesh color, and watermelon taste '\n",
      " 'may be developed and selected for superior qualities including appearance '\n",
      " 'and taste. Further, watermelon varieties may be developed and selected for '\n",
      " 'superior qualities including improved firmness of flesh, improved internal '\n",
      " 'quality, increased yield, and hardiness. Breeding steps that may be used in '\n",
      " 'the watermelon plant breeding program include pedigree breeding, '\n",
      " 'backcrossing, mutation breeding, and recurrent selection. In conjunction '\n",
      " 'with these steps, techniques such as RFLP-enhanced selection, genetic marker '\n",
      " 'enhanced selection (for example SSR markers) and the making of double '\n",
      " 'haploids may be utilized.</p>\\n'\n",
      " '<p id=\"p-0118\" num=\"0178\">Several methods can be used to produce triploid '\n",
      " 'seeds from an inbred tetraploid and/or diploid line. Two commonly used '\n",
      " 'methods are the bee-pollination method and the hand-pollination method. In '\n",
      " 'the United States, the bee-pollination method is generally used to produce '\n",
      " 'triploid watermelon seed. Hand-pollination is mainly used to produce '\n",
      " 'triploid watermelon seed in areas where isolation is not available and '\n",
      " 'several triploid hybrids are produced in the same field block.</p>\\n'\n",
      " '<p id=\"p-0119\" num=\"0179\">Tetraploid watermelon lines can be developed from '\n",
      " 'diploid lines by doubling the chromosomes of diploid watermelon lines using '\n",
      " 'methods routine in the art. Chromosome doubling was first accomplished with '\n",
      " 'the alkaloid colchicine by applying colchicine to the growing point of newly '\n",
      " 'emerged watermelon seedlings. Tissue culture methods have also been '\n",
      " 'developed (Zhang, X. P., B. B. Rhodes, H. T. Skorupska, W. C. Bridges, 1995, '\n",
      " 'Generating Tetraploid Watermelon Using Colchicine in Vitro, G. Lester &#x26; '\n",
      " \"J. Dunlap et al. (eds.), Cucurbitaceae '94: 134-139). Dinitroanilines have \"\n",
      " 'been used to double chromosome numbers, and their effectiveness has '\n",
      " 'previously been compared with crops other than watermelon. Li et al. '\n",
      " 'compared in vitro chromosome doubling effectiveness using colchicine and the '\n",
      " 'dinitroanilines, ethalfluralin '\n",
      " '(N-ethyl-N-2-methyl-2-propenyl)-2,6-dinitro-4-(trifluoromethyl)benzanine), '\n",
      " 'and oryzalin (3,5-dinitro-N4,N4-dipropylsulfanilamide) and concluded that '\n",
      " 'either ethalfluralin or oryzalin was preferable to colchicine (Li, Ying, J. '\n",
      " 'F. Whitesides, B. Rhodes, 1999, In vitro generation of tetraploid watermelon '\n",
      " 'with two different dinitroanilines and colchicines, Cucurbit Genetics '\n",
      " 'Cooperative Rpt 22:38-40).</p>\\n'\n",
      " '<p id=\"p-0120\" num=\"0180\">Several treatment methods can be used to induce '\n",
      " 'tetraploids from diploids using the chemicals mentioned above. One exemplary '\n",
      " 'method is to treat the seed before sowing. The seed are soaked in clean '\n",
      " 'water for 5-6 hrs and then the seed are soaked in either colchicine solution '\n",
      " '(0.2%) or dinitroanilines (e.g. 35 &#x3bc;M oryzalin) for 24 hrs. The seed '\n",
      " 'are briefly rinsed before sowing. Dry seed can also be directly soaked in '\n",
      " 'the chemical solution without pre-soaking in the water. The treatment '\n",
      " 'usually reduces the germination and emergence. A second method is to treat '\n",
      " 'the newly emerged seedling. To illustrate, the diploid inbreds can be sown '\n",
      " 'in the greenhouse in seedling flats. The soil temperature is kept at '\n",
      " '29-31&#xb0; C. for rapid and uniform germination. One drop of colchicine '\n",
      " '(0.1%) or dinitroanilines (e.g. 35 &#x3bc;M oryzalin) solution is added to '\n",
      " 'the shoot apex between the cotyledons as soon as the seedling has emerged '\n",
      " 'from soil. The colchicine solution is applied to the growing point in the '\n",
      " 'morning or evening for three consecutive days. Good chromosome doubling is '\n",
      " 'achieved from application of oryzalin. Another illustrative method is to '\n",
      " 'treat the shoot apex of germinated seed after which the germinated seed is '\n",
      " 'planted into soil. The seeds are germinated in an incubator at 30&#xb0; C. '\n",
      " 'When the radicles are about 2 cm long, the portion above the hypocotyls of '\n",
      " 'germinated seeds is immersed upside down into colchicine (0.1%) or '\n",
      " 'dinitroaniline solution (35 &#x3bc;M oryzalin) for 10-15 hrs at 30&#xb0; C. '\n",
      " 'in an incubator. The treatment is typically conducted in a high humidity '\n",
      " 'chamber or box to assure that the radicles/roots are not desiccated. The '\n",
      " 'seeds are then washed and planted in the soil. The last two methods, '\n",
      " 'although more tedious to use, usually give better recovery of tetraploid '\n",
      " 'events as the root system is not affected by the treatment.</p>\\n'\n",
      " '<p id=\"p-0121\" num=\"0181\">The next step is to develop tetraploid lines from '\n",
      " 'individual converting events. For example, the selected tetraploid '\n",
      " 'individuals based on morphological expression can be self-pollinated and the '\n",
      " 'resulting seeds planted in the next generation as lines. These lines can '\n",
      " 'again be self-pollinated and compared for fertility and horticultural '\n",
      " 'traits. Only the desirable lines are selected if there is difference among '\n",
      " 'these lines. Desirable lines may be bulk harvested if there is no variation '\n",
      " 'within the line and among selected lines. Further seed increases may be '\n",
      " 'conducted in an isolation block. Mass selection may be conducted for this '\n",
      " 'increase in the isolation plot and thereafter. Fertility of the tetraploid '\n",
      " 'may be improved in subsequent generations.</p>\\n'\n",
      " '<p id=\"p-0122\" num=\"0182\">Crossing two different tetraploids and then going '\n",
      " 'through recombination breeding can also result in new tetraploid lines. A '\n",
      " 'longer breeding period is typically employed to develop a stable tetraploid '\n",
      " 'line using this approach because of the larger number of combinations and '\n",
      " 'the fewer seed that tetraploids produce. However, some breeders have made '\n",
      " 'good progress by taking this approach.</p>\\n'\n",
      " '<p id=\"p-0123\" num=\"0183\">Because meiosis is sometimes irregular in '\n",
      " 'autotetraploids, there can be diploids and aneuploids among the offspring. '\n",
      " 'The leaves, flowers and pollen grains of tetraploids are morphologically '\n",
      " 'distinct from diploids (Zhang, X. P., B. B. Rhodes, H. T. Skorupska, W. C. '\n",
      " 'Bridges, 1995<i>, Generating Tetraploid Watermelon Using Colchicine in '\n",
      " \"Vitro</i>, G. Lester &#x26; J. Dunlap et al. (eds.), Cucurbitaceae '94: \"\n",
      " '134-139). Tetraploids also have a different number of chloroplasts in the '\n",
      " 'guard cells (Compton, M. E., D. J. Gray and G. W. Elmstrom. 1996<i>, '\n",
      " 'Identification of tetraploid regenerants from cotyledons of diploid '\n",
      " 'watermelon cultures in vitro</i>, Euphytica 87:165-172). These morphological '\n",
      " 'traits can help the breeder eliminate the diploids and aneuploids occurring '\n",
      " 'in the tetraploid population during sexual propagation. Diploid reversions '\n",
      " 'can also be identified in situations in which a diploid derived from a '\n",
      " 'tetraploid line is desired, and such diploid reversions are also encompassed '\n",
      " 'by the present invention.</p>\\n'\n",
      " '<p id=\"p-0124\" num=\"0184\">The cultivar of the invention can also be used for '\n",
      " 'transformation where exogenous genes are introduced and expressed by the '\n",
      " 'cultivar of the invention. Genetic variants of hybrid '\n",
      " '&#x2018;E26S.00171&#x2019; created either through traditional breeding '\n",
      " 'methods or through transformation of hybrid &#x2018;E26S.00171&#x2019; by '\n",
      " 'any of a number of protocols known to those of skill in the art are intended '\n",
      " 'to be within the scope of this invention.</p>\\n'\n",
      " '<p id=\"p-0125\" num=\"0185\">Mutations for use in mutation breeding can be '\n",
      " 'induced in plants by using mutagenic chemicals such as ethyl methane '\n",
      " 'sulfonate (EMS), by irradiation of plant material with gamma rays or fast '\n",
      " 'neutrons, or by other means. The resulting nucleotide changes are random, '\n",
      " 'but in a large collection of mutagenized plants the mutations in a gene of '\n",
      " 'interest can be readily identified by using the TILLING (Targeting Induced '\n",
      " 'Local Lesions IN Genomes) method (McCallum et al. (2000) Targeted screening '\n",
      " 'for induced mutations. Nat. Biotechnol. 18, 455-457, and Henikoff et al. '\n",
      " '(2004) TILLING. Traditional mutagenesis meets functional genomics. Plant '\n",
      " 'Physiol. 135, 630-636). The principle of this method is based on the PCR '\n",
      " 'amplification of the gene of interest from genomic DNA of a large collection '\n",
      " 'of mutagenized plants in the M2 generation. By DNA sequencing or by looking '\n",
      " 'for point mutations using a single-strand specific nuclease, such as the '\n",
      " 'CEL-I nuclease (Till et al. (2004) Mismatch cleavage by single-strand '\n",
      " 'specific nucleases. Nucleic Acids Res. 32, 2632-2641), the individual plants '\n",
      " 'that have a mutation in the gene of interest are identified. By screening '\n",
      " 'many plants, a large collection of mutant alleles is obtained, each giving a '\n",
      " 'different effect on gene expression or enzyme activity. The gene expression '\n",
      " 'or protein levels can for example be tested by transcript analysis levels '\n",
      " '(e.g., by RT-PCR) or by quantification of protein levels with antibodies. '\n",
      " 'Plants with the desired reduced gene expression or reduced protein '\n",
      " 'expression are then back-crossed or crossed to other breeding lines to '\n",
      " 'transfer only the desired new allele into the background of the crop '\n",
      " 'wanted.</p>\\n'\n",
      " '<p id=\"p-0126\" num=\"0186\">Genes of interest for use in breeding may also be '\n",
      " 'edited using gene editing techniques including transcription activator-like '\n",
      " 'effector nuclease (TALEN) gene editing techniques, Clustered Regularly '\n",
      " 'Interspaced Short Palindromic Repeat (CRISPR/Cas9) gene editing techniques, '\n",
      " 'and/or zinc-finger nuclease (ZFN) gene editing techniques. For this, '\n",
      " 'transgenic plants are generated expressing one or more constructs targeting '\n",
      " 'the gene of interest. These constructs may include, without limitation, an '\n",
      " 'anti-sense construct, an optimized small-RNA construct, an inverted repeat '\n",
      " 'construct, a targeting construct, a guide RNA construct, a construct '\n",
      " 'encoding a targeting protein, and/or a combined sense-anti-sense construct, '\n",
      " 'and may work in conjunction with a nuclease, an endonuclease, and/or an '\n",
      " 'enzyme, so as to downregulate the expression of a gene of interest.</p>\\n'\n",
      " '<p id=\"p-0127\" num=\"0187\">One of ordinary skill in the art of plant breeding '\n",
      " 'would know how to evaluate the traits of two plant varieties to determine if '\n",
      " 'there is no significant difference between the two traits expressed by those '\n",
      " 'varieties. For example, see Fehr and Walt, Principles of Cultivar '\n",
      " 'Development, p. 261-286 (1987). Thus the invention includes watermelon '\n",
      " 'plants including a combination of at least two traits selected from the '\n",
      " 'combination of traits listed in the Summary of the Invention for hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;, so that said progeny watermelon plant is not '\n",
      " 'significantly different for said traits than watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;, as determined at the 5% significance level when '\n",
      " 'grown in the same environmental conditions and/or may be characterized by '\n",
      " 'percent similarity or identity to watermelon hybrid '\n",
      " '&#x2018;E26S.00171&#x2019;, as determined by SSR markers. Mean trait values '\n",
      " 'may be used to determine whether trait differences are significant, and '\n",
      " 'preferably the traits are measured on plants grown under the same '\n",
      " 'environmental conditions. Once such a variety or line is developed its value '\n",
      " 'is substantial since it is important to advance the germplasm base as a '\n",
      " 'whole in order to maintain or improve traits such as firmness of flesh, '\n",
      " 'internal quality, taste, appearance, yield, disease resistance, pest '\n",
      " 'resistance, and plant performance in extreme environmental conditions.</p>\\n'\n",
      " '<p id=\"p-0128\" num=\"0188\">As used herein, the term &#x201c;plant&#x201d; '\n",
      " 'includes plant cells, plant protoplasts, plant cell tissue cultures from '\n",
      " 'which watermelon plants can be regenerated, plant calli, plant clumps, and '\n",
      " 'plant cells that are intact in plants or parts of plants, such as leaves, '\n",
      " 'pollen, embryos, cotyledons, hypocotyl, roots, root tips, anthers, pistils, '\n",
      " 'flowers, seeds, stems, and the like.</p>\\n'\n",
      " '<p id=\"p-0129\" num=\"0189\">The use of the terms &#x201c;a,&#x201d; '\n",
      " '&#x201c;an,&#x201d; and &#x201c;the,&#x201d; and similar referents in the '\n",
      " 'context of describing the invention (especially in the context of the '\n",
      " 'following claims) are to be construed to cover both the singular and the '\n",
      " 'plural, unless otherwise indicated herein or clearly contradicted by '\n",
      " 'context. The terms &#x201c;comprising,&#x201d; &#x201c;having,&#x201d; '\n",
      " '&#x201c;including,&#x201d; and &#x201c;containing&#x201d; are to be '\n",
      " 'construed as open-ended terms (i.e., meaning &#x201c;including, but not '\n",
      " 'limited to,&#x201d;) unless otherwise noted. Recitation of ranges of values '\n",
      " 'herein are merely intended to serve as a shorthand method of referring '\n",
      " 'individually to each separate value falling within the range, unless '\n",
      " 'otherwise indicated herein, and each separate value is incorporated into the '\n",
      " 'specification as if it were individually recited herein. For example, if the '\n",
      " 'range 10-15 is disclosed, then 11, 12, 13, and 14 are also disclosed. All '\n",
      " 'methods described herein can be performed in any suitable order unless '\n",
      " 'otherwise indicated herein or otherwise clearly contradicted by context. The '\n",
      " 'use of any and all examples, or exemplary language (e.g., &#x201c;such '\n",
      " 'as&#x201d;) provided herein, is intended merely to better illuminate the '\n",
      " 'invention and does not pose a limitation on the scope of the invention '\n",
      " 'unless otherwise claimed. No language in the specification should be '\n",
      " 'construed as indicating any non-claimed element as essential to the practice '\n",
      " 'of the invention.</p>\\n'\n",
      " '<p id=\"p-0130\" num=\"0190\">While a number of exemplary aspects and '\n",
      " 'embodiments have been discussed above, those of skill in the art will '\n",
      " 'recognize certain modifications, permutations, additions, and '\n",
      " 'sub-combinations thereof. It is therefore intended that the following '\n",
      " 'appended claims and claims hereafter introduced are interpreted to include '\n",
      " 'all such modifications, permutations, additions, and sub-combinations as are '\n",
      " 'within their true spirit and scope.</p>\\n'\n",
      " '<heading id=\"h-0017\" level=\"1\">EXAMPLES</heading>\\n'\n",
      " '<p id=\"p-0131\" num=\"0191\">The presently disclosed subject matter will be '\n",
      " 'better understood by reference to the following Examples, which are provided '\n",
      " 'as exemplary of the invention, and not by way of limitation.</p>\\n'\n",
      " '<heading id=\"h-0018\" level=\"1\">Example 1: Assessment of the Spot Allele '\n",
      " 'Dosage in Spotted Watermelons</heading>\\n'\n",
      " '<p id=\"p-0132\" num=\"0192\">This example describes the identification of '\n",
      " 'markers able to identify dosage of the spot allele in spotted watermelon '\n",
      " 'lines.</p>\\n'\n",
      " '<heading id=\"h-0019\" level=\"2\">Materials and Methods</heading>\\n'\n",
      " '<heading id=\"h-0020\" level=\"2\">Samples</heading>\\n'\n",
      " '<p id=\"p-0133\" num=\"0193\">Multiple spotted watermelon populations were used '\n",
      " 'for testing. Young leaf samples were obtained from each of the following: '\n",
      " '(1) a tetraploid F2 population segregating for the spot locus, (2) triploid '\n",
      " 'variety with one spot allele (33%), (3) triploid variety with two spot '\n",
      " 'alleles (66%), (4) triploid variety with three spot alleles (100%), (5) '\n",
      " 'tetraploid line with a present spot locus, (6) tetraploid line with an '\n",
      " 'absent spot locus, (7) diploid line with a present spot locus, and (8) '\n",
      " 'diploid line with an absent spot locus.</p>\\n'\n",
      " '<heading id=\"h-0021\" level=\"2\">Marker Candidates</heading>\\n'\n",
      " '<p id=\"p-0134\" num=\"0194\">A diploid F2 population was used to map the region '\n",
      " 'associated with the spot phenotype. The results obtained from this F2 '\n",
      " 'population were compared with whole genome resequencing of two lines showing '\n",
      " 'the spot phenotype. Fine mapping and recombinant selection was then done on '\n",
      " 'two populations.</p>\\n'\n",
      " '<p id=\"p-0135\" num=\"0195\">The reference genome of &#x2018;Charleston '\n",
      " 'Gray&#x2019; (Wu et al. Plant Biotechnol J. 2019 December; 17(12):2246-2258. '\n",
      " 'doi: 10.1111/pbi.13136) was used. The mapping process identified the region '\n",
      " 'linked to the spot phenotype as being between 86340 (start) and 242510 (end) '\n",
      " 'on CG_Chr04 (&#x2018;Charleston Gray&#x2019; Chromosome 4). In this region, '\n",
      " '145 SNPs were identified.</p>\\n'\n",
      " '<heading id=\"h-0022\" level=\"2\">Phenotypic Assessment of Mapping '\n",
      " 'Population</heading>\\n'\n",
      " '<p id=\"p-0136\" num=\"0196\">The diploid F2 population used to map the region '\n",
      " 'was phenotyped for leaf appearance (visual assessment, not quantitative). A '\n",
      " 'pre-grouping of sample plants was done based on spot density.</p>\\n'\n",
      " '<heading id=\"h-0023\" level=\"2\">High Resolution Melting Analysis</heading>\\n'\n",
      " '<p id=\"p-0137\" num=\"0197\">An amplicon of 196 bp was generated by PCR on '\n",
      " 'genomic DNA following standard setup. Post amplification analysis by '\n",
      " 'high-resolution melting curve was performed following manufacturer '\n",
      " 'recommended settings for temperature ramping and fluorescence acquisition. '\n",
      " 'The melting curves were then normalized and analyzed using an internal '\n",
      " 'software.</p>\\n'\n",
      " '<heading id=\"h-0024\" level=\"2\">Results</heading>\\n'\n",
      " '<p id=\"p-0138\" num=\"0198\">The phenotypic assessment of the F2 diploid '\n",
      " 'mapping population identified some plants with higher leaf spot density '\n",
      " '(<figref idref=\"DRAWINGS\">FIGS. <b>4</b>A-<b>4</b>D</figref>), and some '\n",
      " 'plants with lower leaf spot density (<figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>4</b>E-<b>4</b>H</figref>). Plants with these distinct phenotypes were '\n",
      " 'separated into two groups. Subsequent analysis with markers (see below) '\n",
      " 'confirmed that these phenotypes corresponded to different levels of spotted '\n",
      " 'allele dose. Those plants with lower leaf spot density (<figref '\n",
      " 'idref=\"DRAWINGS\">FIGS. <b>4</b>E-<b>4</b>H</figref>) were found to be '\n",
      " 'heterozygous, i.e., have 50% spotted allele dose, while those plants with '\n",
      " 'higher leaf spot density (<figref idref=\"DRAWINGS\">FIGS. '\n",
      " '<b>4</b>A-<b>4</b>D</figref>) were found to be homozygous, i.e., have 100% '\n",
      " 'spotted allele dose. Without wishing to be bound by theory, it is thought '\n",
      " 'that the allele percentage may correlate with leaf spot density in diploid '\n",
      " 'spotted plants.</p>\\n'\n",
      " '<p id=\"p-0139\" num=\"0199\">Two SNP markers were identified that were able to '\n",
      " 'assess the number of spot alleles in watermelons with different ploidy, '\n",
      " 'which are provided in Table 2, below.</p>\\n'\n",
      " '<p id=\"p-0140\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00002\" num=\"00002\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 2</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>SNP markers.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"4\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"56pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"35pt\" align=\"center\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"70pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"4\" colwidth=\"56pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>SNP in</entry>\\n'\n",
      " '<entry/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Chromosome</entry>\\n'\n",
      " '<entry>Position</entry>\\n'\n",
      " '<entry>&#x2018;Charleston Gray&#x2019;</entry>\\n'\n",
      " '<entry>SNP in Samples</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"4\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>CG_Chr04</entry>\\n'\n",
      " '<entry>134886</entry>\\n'\n",
      " '<entry>A</entry>\\n'\n",
      " '<entry>T</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>CG_Chr04</entry>\\n'\n",
      " '<entry>&#x2002;86340</entry>\\n'\n",
      " '<entry>A</entry>\\n'\n",
      " '<entry>G</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"4\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0141\" num=\"0200\"><figref idref=\"DRAWINGS\">FIG. <b>4</b></figref> '\n",
      " 'shows the results of the high resolution melting analysis conducted using '\n",
      " 'primers and a probe directed to the SNP at position 134886. Each group of '\n",
      " 'curves with a black circle corresponded to a specific percentage of spot '\n",
      " 'allele. Table 3, below, provides the percentages, the number of alleles '\n",
      " 'present, and the corresponding ploidies. For example, 75% spotted was only '\n",
      " 'possible in tetraploid lines, which can have up to four spotted '\n",
      " 'alleles.</p>\\n'\n",
      " '<p id=\"p-0142\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00003\" num=\"00003\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 3</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>High resolution melting analysis results.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"3\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"63pt\" align=\"center\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"91pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"63pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry>% Spot</entry>\\n'\n",
      " '<entry>Number of spotted</entry>\\n'\n",
      " '<entry/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Allele</entry>\\n'\n",
      " '<entry>alleles</entry>\\n'\n",
      " '<entry>Ploidy</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>100%&#x2002;</entry>\\n'\n",
      " '<entry>Homozygous present</entry>\\n'\n",
      " '<entry>All</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>75%</entry>\\n'\n",
      " '<entry>3 alleles</entry>\\n'\n",
      " '<entry>Tetrapioid (4n)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>66%</entry>\\n'\n",
      " '<entry>2 alleles</entry>\\n'\n",
      " '<entry>Triploid (3n)_</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>50%</entry>\\n'\n",
      " '<entry>2 alleles</entry>\\n'\n",
      " '<entry>Tetrapioid (4n)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>50%</entry>\\n'\n",
      " '<entry>1 allele</entry>\\n'\n",
      " '<entry>Diploid (2n)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>33%</entry>\\n'\n",
      " '<entry>1 allele</entry>\\n'\n",
      " '<entry>Triploid (3n)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>25%</entry>\\n'\n",
      " '<entry>1 allele</entry>\\n'\n",
      " '<entry>Tetrapioid (4n)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>&#x2002;0%</entry>\\n'\n",
      " '<entry>Homozygous absent</entry>\\n'\n",
      " '<entry>All</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0143\" num=\"0201\">These results clearly showed that the identified '\n",
      " 'markers were able to distinguish the allele percentage present in spotted '\n",
      " 'watermelon lines and varieties. Thus, these markers may be used in breeding '\n",
      " 'and/or selection of spotted watermelon lines and varieties.</p>\\n'\n",
      " '<heading id=\"h-0025\" level=\"1\">Example 2: Phenotypic and Genotypic Ratios '\n",
      " 'for a Segregating Spotted Diploid Watermelon, and Breeding of a Homozygous '\n",
      " 'Spotted Diploid Line (HSDL)</heading>\\n'\n",
      " '<p id=\"p-0144\" num=\"0202\">This example describes the phenotypic and '\n",
      " 'genotypic ratios for a segregating diploid watermelon, as well as the '\n",
      " 'breeding process to obtain a new homozygous spotted diploid line '\n",
      " '(HSDL).</p>\\n'\n",
      " '<heading id=\"h-0026\" level=\"2\">Materials and Methods</heading>\\n'\n",
      " '<heading id=\"h-0027\" level=\"2\">Spotted Alleles</heading>\\n'\n",
      " '<p id=\"p-0145\" num=\"0203\">Spotted plants were identified by phenotypic '\n",
      " 'assessment. The spotted allele underlying the spotted phenotype was '\n",
      " 'designated as &#x201c;A&#x201d;, while the non-spotted allele was designated '\n",
      " 'as &#x201c;a&#x201d;.</p>\\n'\n",
      " '<heading id=\"h-0028\" level=\"2\">Breeding Process</heading>\\n'\n",
      " '<p id=\"p-0146\" num=\"0204\">The initial cross was of a spotted plant with a '\n",
      " 'non-spotted plant. The spotted plant was identified by phenotype alone. The '\n",
      " 'F1 generation produced by this cross all had the spotted phenotype. Then, '\n",
      " 'the F1 generation was self pollinated to produce a F2 generation.</p>\\n'\n",
      " '<heading id=\"h-0029\" level=\"2\">Results</heading>\\n'\n",
      " '<p id=\"p-0147\" num=\"0205\">The phenotypes observed in the F2 generation '\n",
      " 'indicated that the F1 generation had the genotype of Aa, i.e., was '\n",
      " 'heterozygous for the spotted allele. There were are four possible '\n",
      " 'combinations of the spotted genotype observed in the F2 population. Table 4A '\n",
      " 'shows the genotypic ratios, while Table 4B shows the phenotypic ratios.</p>\\n'\n",
      " '<p id=\"p-0148\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00004\" num=\"00004\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 4A</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Genotypic ratios in F2 population.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"4\">\\n'\n",
      " '<colspec colname=\"offset\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"63pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"63pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"63pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>A</entry>\\n'\n",
      " '<entry>a</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry namest=\"offset\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>A</entry>\\n'\n",
      " '<entry>AA</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>a</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>aa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry namest=\"offset\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0149\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00005\" num=\"00005\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 4B</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Phenotypic ratios in F2 population.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"4\">\\n'\n",
      " '<colspec colname=\"offset\" colwidth=\"21pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"42pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"77pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"77pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>A</entry>\\n'\n",
      " '<entry>a</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry namest=\"offset\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>A</entry>\\n'\n",
      " '<entry>Spotted (100%)</entry>\\n'\n",
      " '<entry>Spotted (50%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>a</entry>\\n'\n",
      " '<entry>Spotted (50%)</entry>\\n'\n",
      " '<entry>Not spotted (0%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry namest=\"offset\" nameend=\"3\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0150\" num=\"0206\">As can be seen from Table 4B, the phenotype of the '\n",
      " 'F2 population was 3:1 spotted, with two different percentages of spotted '\n",
      " 'allele (100% and 50%). The F2 plants with the spotted phenotype were then '\n",
      " 'self pollinated and selected for the spotted phenotype for a sufficient '\n",
      " 'number of generations to produce a new homozygous spotted diploid line '\n",
      " '(HSDL). <figref idref=\"DRAWINGS\">FIG. <b>5</b></figref> shows an exemplary '\n",
      " 'diploid watermelon line with 100% spot allele.</p>\\n'\n",
      " '<p id=\"p-0151\" num=\"0207\">The markers identified in Example 1 can be used to '\n",
      " 'facilitate the breeding process described in this Example. Specifically, the '\n",
      " 'markers for the spot allele are very helpful to use at the F2 stage in order '\n",
      " 'to select the material with the higher dosage of spotted alleles, namely '\n",
      " '100% (AA). The use of these markers will allow the production of a diploid '\n",
      " 'line homozygous for the spotted trait (HSDL) in fewer generations of '\n",
      " 'inbreeding.</p>\\n'\n",
      " '<heading id=\"h-0030\" level=\"1\">Example 3: Genotypic Ratios for a Segregating '\n",
      " 'Tetraploid Watermelon, and Breeding of a Homozygous Spotted Tetraploid Line '\n",
      " '(HSTL)</heading>\\n'\n",
      " '<p id=\"p-0152\" num=\"0208\">This example describes the phenotypic and '\n",
      " 'genotypic ratios for a segregating tetraploid watermelon, as well as the '\n",
      " 'breeding process to obtain a new homozygous spotted tetraploid line '\n",
      " '(HSTL).</p>\\n'\n",
      " '<heading id=\"h-0031\" level=\"2\">Materials and Methods</heading>\\n'\n",
      " '<heading id=\"h-0032\" level=\"2\">Spotted Alleles</heading>\\n'\n",
      " '<p id=\"p-0153\" num=\"0209\">Spotted plants were identified by phenotypic '\n",
      " 'assessment. The spotted allele was designated as &#x201c;A&#x201d;, while '\n",
      " 'the non-spotted allele was designated as &#x201c;a&#x201d;.</p>\\n'\n",
      " '<heading id=\"h-0033\" level=\"2\">Breeding Process</heading>\\n'\n",
      " '<p id=\"p-0154\" num=\"0210\">The initial cross was of a spotted plant with a '\n",
      " 'non-spotted plant. The spotted plant was identified by phenotype alone. The '\n",
      " 'F1 generation produced by this cross all had the spotted phenotype. Then, '\n",
      " 'the F1 generation was self pollinated to produce a F2 generation.</p>\\n'\n",
      " '<heading id=\"h-0034\" level=\"2\">Results</heading>\\n'\n",
      " '<p id=\"p-0155\" num=\"0211\">The phenotypes observed in the F2 generation '\n",
      " 'indicated that the F1 generation had the genotype of AAaa, i.e., was '\n",
      " 'heterozygous for the spotted allele. There were thirty-six possible '\n",
      " 'combinations of the spotted genotype observed in the F2 population. Table 5A '\n",
      " 'shows the genotypic ratios, while Table 5B shows the phenotypic ratios.</p>\\n'\n",
      " '<p id=\"p-0156\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00006\" num=\"00006\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 5A</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Genotypic ratios in F2 population.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"7\">\\n'\n",
      " '<colspec colname=\"offset\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"4\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"5\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"6\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>AA</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>aa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry namest=\"offset\" nameend=\"6\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"7\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"4\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"5\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"6\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"7\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry>AA</entry>\\n'\n",
      " '<entry>AAAA</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>AAAa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>aa</entry>\\n'\n",
      " '<entry>AAaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '<entry>Aaaa</entry>\\n'\n",
      " '<entry>aaaa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"7\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0157\" num=\"0000\">\\n'\n",
      " '<tables id=\"TABLE-US-00007\" num=\"00007\">\\n'\n",
      " '<table frame=\"none\" colsep=\"0\" rowsep=\"0\">\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"1\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"217pt\" align=\"center\"/>\\n'\n",
      " '<thead>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" rowsep=\"1\">TABLE 5B</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</thead>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"1\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Phenotypic ratios in F2 population.</entry>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"7\">\\n'\n",
      " '<colspec colname=\"offset\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"4\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"5\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"6\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>AA</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>aa</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry namest=\"offset\" nameend=\"6\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '<tgroup align=\"left\" colsep=\"0\" rowsep=\"0\" cols=\"7\">\\n'\n",
      " '<colspec colname=\"1\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"2\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"3\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"4\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"5\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"6\" colwidth=\"28pt\" align=\"left\"/>\\n'\n",
      " '<colspec colname=\"7\" colwidth=\"35pt\" align=\"left\"/>\\n'\n",
      " '<tbody valign=\"top\">\\n'\n",
      " '<row>\\n'\n",
      " '<entry>AA</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(100%)</entry>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>Aa</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(75%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry>aa</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Spotted</entry>\\n'\n",
      " '<entry>Not</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(50%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '<entry>(25%)</entry>\\n'\n",
      " '<entry>spotted</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry/>\\n'\n",
      " '<entry>(0%)</entry>\\n'\n",
      " '</row>\\n'\n",
      " '<row>\\n'\n",
      " '<entry namest=\"1\" nameend=\"7\" align=\"center\" rowsep=\"1\"/>\\n'\n",
      " '</row>\\n'\n",
      " '</tbody>\\n'\n",
      " '</tgroup>\\n'\n",
      " '</table>\\n'\n",
      " '</tables>\\n'\n",
      " '</p>\\n'\n",
      " '<p id=\"p-0158\" num=\"0212\">As can be seen from Table 5B, the phenotype of the '\n",
      " 'F2 population was 35:1 spotted, with four different percentages of spotted '\n",
      " 'allele (100%, 75%, 50%, and 25%). The F2 plants with the spotted phenotype '\n",
      " 'were then self pollinated and selected for the spotted phenotype for a '\n",
      " 'sufficient number of generations to produce a homozygous spotted tetraploid '\n",
      " 'line (HSTL). <figref idref=\"DRAWINGS\">FIG. <b>6</b></figref> shows an '\n",
      " 'exemplary tetraploid watermelon line with 100% spot allele.</p>\\n'\n",
      " '<p id=\"p-0159\" num=\"0213\">The markers identified in Example 1 can be used to '\n",
      " 'facilitate the breeding process described in this Example. Specifically, the '\n",
      " 'markers for the spot allele are very helpful to use at the F2 stage in order '\n",
      " 'to select the material with the higher dosage of spotted alleles, namely '\n",
      " '100% (AAAA). The use of the marker allows the production of a tetraploid '\n",
      " 'line homozygous for the spotted trait (HSTL) in fewer generations of '\n",
      " 'inbreeding.</p>\\n'\n",
      " '<?detailed-description description=\"Detailed Description\" end=\"tail\"?>\\n'\n",
      " '</description>\\n'\n",
      " '<us-sequence-list-doc id=\"SEQLST-1\">\\n'\n",
      " '<sequence-list file=\"US20230000035A1-20230105-S00001.XML\" '\n",
      " 'carriers=\"internal-electronic\" seq-file-type=\"ST.25\"/>\\n'\n",
      " '</us-sequence-list-doc>\\n'\n",
      " '<claims id=\"claims\">\\n'\n",
      " '<claim id=\"CLM-00001\" num=\"00001\">\\n'\n",
      " '<claim-text><b>1</b>. A method of producing a homozygous-spotted diploid '\n",
      " 'watermelon line (HSDL), comprising:\\n'\n",
      " '<claim-text>(a) crossing a first diploid watermelon plant with a spotted '\n",
      " 'trait with a second diploid watermelon plant without the spotted trait to '\n",
      " 'produce progeny watermelon seed, wherein the presence of the spotted trait '\n",
      " 'is determined by visually phenotyping for the spotted trait or by genotyping '\n",
      " 'to identify a presence of a marker linked to a spotted allele, wherein the '\n",
      " 'spotted allele is located between position 134886 and position 86340 on '\n",
      " 'Chromosome 4 of a &#x2018;Charleston Gray&#x2019; genome;</claim-text>\\n'\n",
      " '<claim-text>(b) crossing a watermelon plant grown from the progeny '\n",
      " 'watermelon seed with itself or a second watermelon plant to produce '\n",
      " 'additional progeny watermelon seed;</claim-text>\\n'\n",
      " '<claim-text>(c) planting the additional progeny watermelon seed of step (b) '\n",
      " 'to produce additional watermelon plants;</claim-text>\\n'\n",
      " '<claim-text>(d) selecting watermelon plants with the spotted trait from the '\n",
      " 'additional watermelon plants of step (c);</claim-text>\\n'\n",
      " '<claim-text>(e) repeating the crossing of step (b), the planting of step '\n",
      " '(c), and the selecting of step (d) for an additional 3-10 generations to '\n",
      " 'produce a homozygous-spotted diploid watermelon line (HSDL), wherein the '\n",
      " 'HSDL is 100% spotted.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00002\" num=\"00002\">\\n'\n",
      " '<claim-text><b>2</b>. The method of <claim-ref idref=\"CLM-00001\">claim '\n",
      " '1</claim-ref>, wherein the first diploid plant of step (a) is homozygous for '\n",
      " 'the spotted allele or heterozygous for the spotted allele.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00003\" num=\"00003\">\\n'\n",
      " '<claim-text><b>3</b>. The method of <claim-ref idref=\"CLM-00001\">claim '\n",
      " '1</claim-ref>, wherein the marker comprises a first marker comprising a T at '\n",
      " 'position 101 of SEQ ID NO: 1 and/or a second marker comprising a G at '\n",
      " 'position 101 of SEQ ID NO: 6.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00004\" num=\"00004\">\\n'\n",
      " '<claim-text><b>4</b>. The method of <claim-ref idref=\"CLM-00003\">claim '\n",
      " '3</claim-ref>, wherein genotyping of the first marker comprises the use of a '\n",
      " 'forward primer comprising SEQ ID NO: 3, a reverse primer comprising SEQ ID '\n",
      " 'NO: 4, and/or a probe comprising SEQ ID NO: 5, and wherein genotyping of the '\n",
      " 'second marker comprises the use of a forward primer comprising SEQ ID NO: 8, '\n",
      " 'a reverse primer comprising SEQ ID NO: 9, and/or a probe comprising SEQ ID '\n",
      " 'NO: 10.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00005\" num=\"00005\">\\n'\n",
      " '<claim-text><b>5</b>. The HSDL plant produced by the method of <claim-ref '\n",
      " 'idref=\"CLM-00001\">claim 1</claim-ref>.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00006\" num=\"00006\">\\n'\n",
      " '<claim-text><b>6</b>. A method of producing a homozygous-spotted tetraploid '\n",
      " 'watermelon line (HSTL), comprising: (a) doubling a chromosome number of the '\n",
      " 'HSDL plant of <claim-ref idref=\"CLM-00005\">claim 5</claim-ref>, thereby '\n",
      " 'producing a homozygous-spotted tetraploid watermelon line (HSTL), wherein '\n",
      " 'the HSTL is 100% spotted.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00007\" num=\"00007\">\\n'\n",
      " '<claim-text><b>7</b>. A method of producing a 33% spotted triploid hybrid '\n",
      " 'watermelon plant, comprising:\\n'\n",
      " '<claim-text>(a) crossing the HSDL plant of <claim-ref '\n",
      " 'idref=\"CLM-00005\">claim 5</claim-ref> onto a 0% spotted tetraploid inbred '\n",
      " 'plant, wherein the HSDL plant is a male parent and the tetraploid inbred '\n",
      " 'plant is a female parent, to produce a 33% spotted triploid hybrid '\n",
      " 'watermelon plant.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00008\" num=\"00008\">\\n'\n",
      " '<claim-text><b>8</b>. The 33% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of <claim-ref idref=\"CLM-00006\">claim '\n",
      " '6</claim-ref>.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00009\" num=\"00009\">\\n'\n",
      " '<claim-text><b>9</b>. A method of producing a 50% or 100% spotted diploid '\n",
      " 'hybrid watermelon plant, comprising:\\n'\n",
      " '<claim-text>(a) crossing the HSDL plant of <claim-ref '\n",
      " 'idref=\"CLM-00005\">claim 5</claim-ref> onto a 0% spotted diploid inbred plant '\n",
      " 'or onto a second HSDL plant of <claim-ref idref=\"CLM-00005\">claim '\n",
      " '5</claim-ref>, thereby producing a 50% or 100% spotted diploid hybrid '\n",
      " 'watermelon plant.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00010\" num=\"00010\">\\n'\n",
      " '<claim-text><b>10</b>. The 50% or 100% spotted diploid hybrid watermelon '\n",
      " 'plant produced by the method of <claim-ref idref=\"CLM-00009\">claim '\n",
      " '9</claim-ref>.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00011\" num=\"00011\">\\n'\n",
      " '<claim-text><b>11</b>. A method of producing a homozygous-spotted tetraploid '\n",
      " 'watermelon line (HSTL), comprising:\\n'\n",
      " '<claim-text>(a) providing a diploid watermelon plant with a spotted trait, '\n",
      " 'wherein the presence of the spotted trait is determined by visually '\n",
      " 'phenotyping for the spotted trait or by genotyping to identify a presence of '\n",
      " 'a marker linked to a spotted allele, and wherein the spotted allele is '\n",
      " 'located between position 134886 and position 86340 on Chromosome 4 of a '\n",
      " '&#x2018;Charleston Gray&#x2019; genome;</claim-text>\\n'\n",
      " '<claim-text>(b) doubling a chromosome number of the diploid watermelon plant '\n",
      " 'to produce a tetraploid watermelon plant with the spotted '\n",
      " 'trait;</claim-text>\\n'\n",
      " '<claim-text>(c) crossing the tetraploid watermelon plant with a non-spotted '\n",
      " 'watermelon plant to produce a F1 generation of watermelon '\n",
      " 'plants;</claim-text>\\n'\n",
      " '<claim-text>(d) self-pollinating or open pollinating the F1 generation of '\n",
      " 'watermelon plants to produce a F2 generation of watermelon '\n",
      " 'plants;</claim-text>\\n'\n",
      " '<claim-text>(e) phenotyping progeny watermelon plants from the F2 generation '\n",
      " 'to identify the presence of the spotted trait and/or genotyping progeny '\n",
      " 'watermelon plants from the F2 generation to identify the presence of the '\n",
      " 'marker linked to the spotted allele and using the marker to determine a '\n",
      " 'dosage of the spotted allele;</claim-text>\\n'\n",
      " '<claim-text>(f) selecting progeny watermelon plants with the highest dosage '\n",
      " 'of the spotted allele;</claim-text>\\n'\n",
      " '<claim-text>(g) repeating the self-pollinating of step (d), the genotyping '\n",
      " 'of step (e), and the selecting of step (f) for an additional 3-10 '\n",
      " 'generations to produce a homozygous-spotted tetraploid watermelon line '\n",
      " '(HSTL), wherein the HSTL is 100% spotted.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00012\" num=\"00012\">\\n'\n",
      " '<claim-text><b>12</b>. The method of <claim-ref idref=\"CLM-00011\">claim '\n",
      " '11</claim-ref>, wherein the tetraploid plant of step (b) is homozygous for '\n",
      " 'the spotted allele or heterozygous for the spotted allele.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00013\" num=\"00013\">\\n'\n",
      " '<claim-text><b>13</b>. The method of <claim-ref idref=\"CLM-00011\">claim '\n",
      " '11</claim-ref>, wherein the marker comprises a first marker comprising a T '\n",
      " 'at position 101 of SEQ ID NO: 1 and/or a second marker comprising a G at '\n",
      " 'position 101 of SEQ ID NO: 6.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00014\" num=\"00014\">\\n'\n",
      " '<claim-text><b>14</b>. The method of <claim-ref idref=\"CLM-00013\">claim '\n",
      " '13</claim-ref>, wherein genotyping of the first marker comprises the use of '\n",
      " 'a forward primer comprising SEQ ID NO: 3, a reverse primer comprising SEQ ID '\n",
      " 'NO: 4, and/or a probe comprising SEQ ID NO: 5, and wherein genotyping of the '\n",
      " 'second marker comprises the use of a forward primer comprising SEQ ID NO: 8, '\n",
      " 'a reverse primer comprising SEQ ID NO: 9, and/or a probe comprising SEQ ID '\n",
      " 'NO: 10.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00015\" num=\"00015\">\\n'\n",
      " '<claim-text><b>15</b>. The HSTL plant produced by the method of <claim-ref '\n",
      " 'idref=\"CLM-00011\">claim 11</claim-ref>.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00016\" num=\"00016\">\\n'\n",
      " '<claim-text><b>16</b>. A method of producing a 66% spotted triploid hybrid '\n",
      " 'watermelon plant, comprising:\\n'\n",
      " '<claim-text>(a) crossing the HSTL plant of <claim-ref '\n",
      " 'idref=\"CLM-00013\">claim 13</claim-ref> onto a 0% spotted diploid inbred '\n",
      " 'plant, wherein the HSTL plant is a female parent and the diploid inbred '\n",
      " 'plant is a male parent, to produce a 66% spotted triploid hybrid watermelon '\n",
      " 'plant.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00017\" num=\"00017\">\\n'\n",
      " '<claim-text><b>17</b>. The 66% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of <claim-ref idref=\"CLM-00016\">claim '\n",
      " '16</claim-ref>.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00018\" num=\"00018\">\\n'\n",
      " '<claim-text><b>18</b>. A method of producing a 100% spotted triploid hybrid '\n",
      " 'watermelon plant, comprising:\\n'\n",
      " '<claim-text>(a) crossing the HSDL plant of <claim-ref '\n",
      " 'idref=\"CLM-00005\">claim 5</claim-ref> with the HSTL plant of <claim-ref '\n",
      " 'idref=\"CLM-00013\">claim 13</claim-ref> to produce a 100% spotted triploid '\n",
      " 'hybrid watermelon plant.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00019\" num=\"00019\">\\n'\n",
      " '<claim-text><b>19</b>. The 100% spotted triploid hybrid watermelon plant '\n",
      " 'produced by the method of <claim-ref idref=\"CLM-00016\">claim '\n",
      " '16</claim-ref>.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00020\" num=\"00020\">\\n'\n",
      " '<claim-text><b>20</b>. A method of selecting a spotted watermelon plant, '\n",
      " 'comprising:\\n'\n",
      " '<claim-text>a) crossing a spotted watermelon plant to a watermelon plant to '\n",
      " 'produce a population of progeny watermelon plants;</claim-text>\\n'\n",
      " '<claim-text>b) genotyping the population of progeny watermelon plants for a '\n",
      " 'presence of a marker linked to a spotted allele, wherein the spotted allele '\n",
      " 'is located between position 134886 and position 86340 on Chromosome 4 of a '\n",
      " '&#x2018;Charleston Gray&#x2019; genome; and</claim-text>\\n'\n",
      " '<claim-text>c) selecting a progeny plant on the basis of the presence of the '\n",
      " 'one or more sequences genotyped in (b), wherein the progeny watermelon plant '\n",
      " 'is spotted.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00021\" num=\"00021\">\\n'\n",
      " '<claim-text><b>21</b>. The method of <claim-ref idref=\"CLM-00020\">claim '\n",
      " '20</claim-ref>, wherein the marker comprises a first marker comprising a T '\n",
      " 'at position 101 of SEQ ID NO: 1 and/or a second marker comprising a G at '\n",
      " 'position 101 of SEQ ID NO: 6, and wherein genotyping of the first marker '\n",
      " 'comprises the use of a forward primer comprising SEQ ID NO: 3, a reverse '\n",
      " 'primer comprising SEQ ID NO: 4, and/or a probe comprising SEQ ID NO: 5, and '\n",
      " 'wherein genotyping of the second marker comprises the use of a forward '\n",
      " 'primer comprising SEQ ID NO: 8, a reverse primer comprising SEQ ID NO: 9, '\n",
      " 'and/or a probe comprising SEQ ID NO: 10.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00022\" num=\"00022\">\\n'\n",
      " '<claim-text><b>22</b>. A method of assessing a percentage of a spotted '\n",
      " 'allele present in a spotted watermelon plant, comprising:\\n'\n",
      " '<claim-text>a) providing a test spotted watermelon plant and a control '\n",
      " 'spotted watermelon plant;</claim-text>\\n'\n",
      " '<claim-text>b) using a PCR-based technique to assess the presence of one or '\n",
      " 'more sequences selected from the group consisting of a sequence comprising T '\n",
      " 'at position 101 of SEQ ID NO: 1 and a sequence comprising a G at position '\n",
      " '101 of SEQ ID NO: 6; and</claim-text>\\n'\n",
      " '<claim-text>c) determining the percentage of the spotted allele in the test '\n",
      " 'spotted watermelon plant based on a comparison between the results of step '\n",
      " '(b) for the test spotted watermelon plant and the control spotted watermelon '\n",
      " 'plant.</claim-text>\\n'\n",
      " '</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '<claim id=\"CLM-00023\" num=\"00023\">\\n'\n",
      " '<claim-text><b>23</b>. The method of <claim-ref idref=\"CLM-00022\">claim '\n",
      " '22</claim-ref>, wherein the PCR-based technique of step (b) comprises the '\n",
      " 'use of primers and probes selected from the group consisting of SEQ ID NO: '\n",
      " '3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 8, SEQ ID NO: 9, and SEQ ID NO: '\n",
      " '10.</claim-text>\\n'\n",
      " '</claim>\\n'\n",
      " '</claims>\\n'\n",
      " '</us-patent-application>\\n')\n"
     ]
    }
   ],
   "source": [
    "import pprint as pp\n",
    "pp.pprint(xml_parts[34])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['This example details expression of Pro-relaxin polypeptides by E. coli.',\n",
       " 'E. coli expressed Pro-relaxin as a single chain protein composed of 88 amino acids. Upon digestion with trypsin and carboxypeptidase, a connecting peptide and leader sequence are removed. The resulting peptide is a small 6 kDa two-chain peptide member of the insulin superfamily which consists of a 24 residue A-chain and a 29 residue B-chain. The structural fold is characterized by two peptide chains which are held together by two interchain (Cys11-Cys36, and Cys24-Cys48) and one intrachain (Cys10-Cys15) disulfide bonds. The tertiary structure based on a crystal structure of human relaxin-2 revealed a compact fold comprising three helical segments and a short extended region that enclose a hydrophobic core.',\n",
       " 'Relaxin with one or more non-naturally encoded amino acid(s) provides a unique chemistry and enables a specific PEGylated recombinant variant containing a biosynthetically incorporated, chemically reactive, carbonyl group, by replacement of a natural amino acid with para-acetylphenylalanine (pAcF), providing a unique covalent site of attachment for a poly(ethylene) glycol (PEG).']"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "doc_w_exp[20230028168][0][2][\"content\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package averaged_perceptron_tagger to\n",
      "[nltk_data]     C:\\Users\\am3081\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package averaged_perceptron_tagger is already up-to-\n",
      "[nltk_data]       date!\n",
      "[nltk_data] Downloading package punkt to\n",
      "[nltk_data]     C:\\Users\\am3081\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package punkt is already up-to-date!\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'past'"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from classify_patent import check_tense_nltk\n",
    "check_tense_nltk(\"\".join(doc_w_exp[20230028168][0][0][\"content\"]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved as no dup_testing_example_exatly_in_heading.json\n"
     ]
    }
   ],
   "source": [
    "save_as_json(doc_w_exp,filename=\"no dup_testing_example_exatly_in_heading.json\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def process_document_examples(xml_parts, max_docs=100):\n",
    "    \"\"\"Process and extract examples from XML documents.\n",
    "    \n",
    "    Args:\n",
    "        xml_parts (list): List of XML document parts\n",
    "        max_docs (int): Maximum number of documents to process\n",
    "        \n",
    "    Returns:\n",
    "        dict: Document numbers mapped to extracted examples\n",
    "    \"\"\"\n",
    "    doc_with_examples = {}\n",
    "    \n",
    "    for xml in xml_parts:\n",
    "        example_headings = extract_experiments_w_heading(xml)\n",
    "        if not example_headings:\n",
    "            continue\n",
    "            \n",
    "        extracted_examples = []\n",
    "        \n",
    "        try:\n",
    "            if len(example_headings) > 1:\n",
    "                # Process multiple headings\n",
    "                for heading in example_headings:\n",
    "                    siblings = heading.find_next_siblings()\n",
    "                    examples = extract_examples_start_w_word(siblings)\n",
    "                    \n",
    "                    if examples:\n",
    "                        extracted_examples.append(examples)\n",
    "                    else:\n",
    "                        processed = process_siblings(siblings)\n",
    "                        if processed:\n",
    "                            extracted_examples.append(processed)\n",
    "                            \n",
    "            else:\n",
    "                # Process single heading\n",
    "                siblings = example_headings[0].find_next_siblings()\n",
    "                examples = extract_examples_start_w_word(siblings)\n",
    "                \n",
    "                if examples and examples[0][\"content\"]:\n",
    "                    extracted_examples.append(examples)\n",
    "                else:\n",
    "                    processed = process_siblings(siblings)\n",
    "                    if processed:\n",
    "                        extracted_examples.append(processed)\n",
    "            \n",
    "            # Fall back to direct XML processing if needed\n",
    "            if not extracted_examples:\n",
    "                examples = extract_examples_w_word(xml)\n",
    "                if examples:\n",
    "                    extracted_examples.append(examples)\n",
    "            \n",
    "            # Store results if examples found\n",
    "            if extracted_examples:\n",
    "                doc_number = int(find_doc_number(xml)[0])\n",
    "                doc_with_examples[doc_number] = extracted_examples\n",
    "                \n",
    "                if len(doc_with_examples) >= max_docs:\n",
    "                    break\n",
    "                    \n",
    "        except Exception as e:\n",
    "            print(f\"Error processing document: {str(e)}\")\n",
    "            continue\n",
    "            \n",
    "    return doc_with_examples\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "cuda",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
